---
title: The red panda and Cserhati
date: 2023-06-22 12:00:00 -08:00
author: Gerdien de Jong
---

<figure><img src="/uploads/2023/blah.jpeg" alt="red panda image"/><figcaption>Red Panda</figcaption></figure>


In december 2022, I wrote [a blogpost on Panda’s Thumb](https://pandasthumb.org/archives/2022/12/a-tale-of-two-papers.html) about two articles on the classification of the red panda <em>Ailurus fulgens</em> by Matthew/Matyas Cserhati.
One of these articles was in a scientific journal:

A tail of two pandas – whole genome k-mer signature analysis of the red panda (_Ailurus fulgens_)
and the Giant panda (_Ailuropoda melanoleuca_), [BMC Genomics 22 (2021) 228](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-07531-3)

A very similar article was in a creationist journal:

Classification of the Enigmatic Red Panda (_Ailurus fulgens_) Based on Molecular Baraminology-Based
Analysis. [Creation Research Society Quarterly 58 (2021) 76--84](https://www.creationresearch.org/classification-of-the-enigmatic-red-panda-ailurus-fulgens-based-on-molecular-baraminology-based-analysis)

My post on Panda’s Thumb provoked a reaction from Cserhati on the Dutch creationist website
‘Oorsprong’: [_Evolutionists on Panda’s Thumb Attack Well-Respected Science Journal_](https://oorsprong.info/evolutionists-on-pandas-thumb-attack-well-respected-science-journal/).
As a consequence, I felt it might be opportune to specify the trouble with Cserhati’s articles in detail.
<!--more-->

I’ve done so [in 18 posts](https://creationismeweersproken.blogspot.com/2023/01/de-rode-panda-en-cserhati-1-de.html) on my Dutch website ‘Creationisme Weersproken’. The translation of these posts is now available [at my Github site](https://github.com/Gerdien898/pandas) and here [at the Panda's Thumb site](/uploads/2023/redpandacserhati.pdf). At my Github site a file with all the web adresses concerning “the red panda and Cserhati” can be found too.

One file ‘The red panda and Cserhati collection’ presents the total of my posts; the individual posts are as follows (number is page number in the ‘collection’ post):

| post | title | page number which is also a link |
| ---- | ----- | -------------------------------- |
| 1 | The scientific situation | [2](/uploads/2023/redpandacserhati.pdf#page=2) |
| 2 | Literature survey | [10](/uploads/2023/redpandacserhati.pdf#page=10) |
| 3 | A phylogeny is a mobile | [14](/uploads/2023/redpandacserhati.pdf#page=14) |
| 4 | Two papers | [17](/uploads/2023/redpandacserhati.pdf#page=17) |
| 5 | The introduction of the BMC GENOMICS article | [19](/uploads/2023/redpandacserhati.pdf#page=19) |
| 6 | WHOLE GENOME K-MER SIGNATURE | [30](/uploads/2023/redpandacserhati.pdf#page=30) |
| 7 | WGKS IS not suitable for phylogeny | [40](/uploads/2023/redpandacserhati.pdf#page=40) |
| 8 | Clustering | [46](/uploads/2023/redpandacserhati.pdf#page=46) |
| 9 | Clusters on WGKS data | [49](/uploads/2023/redpandacserhati.pdf#page=49) |
| 10 | Procyonidae WGKS? | [55](/uploads/2023/redpandacserhati.pdf#page=55) |
| 11 | Clustering mtDNA | [58](/uploads/2023/redpandacserhati.pdf#page=58) |
| 12 | Phylogenetic trees based on mtDNA | [64](/uploads/2023/redpandacserhati.pdf#page=64) |
| 13 | The BMC GENOMICS article | [73](/uploads/2023/redpandacserhati.pdf#page=73) |
| 14 | The introduction of the CRSQ artcle | [76](/uploads/2023/redpandacserhati.pdf#page=76) |
| 15 | Data and results in the CRSQ article | [80](/uploads/2023/redpandacserhati.pdf#page=80) |
| 16 | Conclusions and discussion in the CRSQ article | [97](/uploads/2023/redpandacserhati.pdf#page=97) |
| 17 | Baramin | [105](/uploads/2023/redpandacserhati.pdf#page=105) |
| 18 | Cserhati tries to answer | [112](/uploads/2023/redpandacserhati.pdf#page=112) |

Creationism is considered in posts 14, 17 and in parts of post 18. Posts 15 and 16 are about clustering, and therefore technical and not about creationism, although the article discussed is in CRSQ.

A survey of scientific work and the present consensus on the red panda was necessary as Cserhati did not cite recent literature. The cited articles on phylogeny were on average at least 10 years old, and important ones are missing. (post 1, 2).  The third post, on how to read a phylogenetic tree, was unfortunately necessary. Cserhati repeatedly confuses the layout of a phylogenetic tree with relatedness as indicated by a phylogenetic tree. The same errors occur in the introduction to the CRSQ paper, and again in his table on the scientific literature in the ‘Oorsprong’ post (post 3, 5 , pp 115-121 in post 18). More errors in citing the literature occur. Perhaps the worst error is citing Flynn et al (2000) as if Flynn et al consider the red panda to be a mustelid, whereas Flynn et al consider the red panda to be a musteloid (pg 24 post 5). This is either basic ignorance or utter sloppiness.

Cserhati’s inability to read a phylogenetic tree might explain why he makes a mess of the NJ and ML phylogenetic trees on mtDNA. Taking the bears as outgroup, the mtDNA data yield a perfectly possible phylogenetic tree. (post 12) Cserhati prides himself on WGKS as it seems the use of this method in phylogeny was his own idea. Post 6 details weaknesses of WGKS, and post 7 shows that WGKS is not suitable for phylogeny as clear by Cserhati’s own data.

Creationists prefer to use clustering. I set out some problems with clustering for classification in general (post 8), whereas the problems with clustering that besets creationism is detailed in post 18 pp 131 -135 Cserhati ignores his own clustering results on WGKS and mtDNA. On WGKS, he gives the correct number of clusters, three clusters, in the CRSQ paper; but veers to four clusters in the BMC Genomics paper. Three WGKS clusters, Felidae, Ursidae and Musteloidea, follow from the supplementary figures. On mtDNA, the optimal number of clusters equals two, Ursidae and Musteloidea, as is apparent from the supplementary figures in both papers. Cserhati reports 5 clusters, the five families, Ursidae, Mephitidae, Ailuridae, Procyonidae and Mustelidae without giving any justification other than eye-balling a heatmap. (posts 9, 11, 15).

In BMC Genomics Cserhati writes:

> The main conclusion that we can draw from this study is that on a whole genome level Ailurus
fulgens possibly belongs to the mustelid clade”.

In CRSQ Cserhati writes:

>Based on these results, it is likely that the red panda belongs to the mustelid holobaramin, ..

In both cases such a main conclusion is only based on reporting four clusters in WGKS rather than three. A non-optimal result based on part of the data.  Evidence that the red panda belongs to the family Mustelidae is totally lacking.

