<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Dembski and Ewert wish upon a star</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Dembski and Ewert wish upon a star" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2024/11/dembskiewert2.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Joe Felsenstein" />
  <meta property="article:published_time" content="2024-11-10T13:00:00-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2024/11/dembskiewert2.html">Dembski and Ewert wish upon a star</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Joe Felsenstein</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2024-11-10T13:00:00-07:00">November 10, 2024 13:00 MST</time></span>
	  </div>
   </header>
  <div align="center"><figure><img width="600px" src="/uploads/2023/Six_Grandfathers.jpg" alt="[Six grandfathers image]" />
<figcaption><div align="center">The mountain called <em>Six Grandfathers</em> by the 
Lakota people before it was renamed Mount Rushmore.<br /> An image of Mount Rushmore adorns the cover of Dembski and 
  Ewert's book, as an example of a<br /> design inference we make. Does the Lakota name reflect 
a design inference? <em>Wikimedia, public domain</em></div></figcaption></figure></div>

<p>&nbsp;</p>

<p>In my <a href="https://pandasthumb.org/archives/2024/06/dembskiewert1.html">post of 12 June</a>, I commented on the first part of William 
Dembski and Winston Ewert’s new book, the second edition of Dembski’s 
1998 book <em>The Design Inference: Eliminating Chance 
Through Small Probabilities</em>.   I noted that to make the argument 
for a Design Inference they had set aside Dembski’s previous criterion 
of Complex Specified Information, and instead replaced it by 
Algorithmic Specified Complexity (ASC).</p>

<p>This is a measure of the difference between the length of a bitstring and the length of a bitstring that describes it. 
It originates from the mathematical work on “randomness deficiciency”, where the bitstring is a binary number, and the smaller bistring is a computer program that computes it.  In that field, numbers that are considered truly random are those generated by programs that are almost as long as the number.  A number that can be generated by a short program is not considered random.</p>

<p>Dembski and Ewert intend to apply this to biology.  The longer number somehow specifies a phenotype (or maybe a 
genotype), while the shorter number “describes” it.  In my previous review, I had reached Chapter 7, “Evolutionary Biology”.  Here, let’s consider how they argue the ASC criterion can be applied to biological adaptations.  (Spoiler – we’re going to be disappointed).</p>

<a class="anchor" id="more"></a>

<p>I was full of expectations when I started reading Chapter 7.  I had a number of questions in mind, ones I have 
asked before in posts and comments about their ASC criterion for design inference (for example, <a href="https://pandasthumb.org/archives/2019/12/Is-Algorithmic-Specified-Complexity-Useless-for-Analyzing-Evolution.html">here</a>, <a href="https://pandasthumb.org/archives/2020/12/Any-new-arguments.html">here</a>, and <a href="https://pandasthumb.org/archives/2021/12/worst-of-2021.html">here</a>).</p>

<ol>
  <li>What did the bitstring represent? A genotype?  A phenotype?</li>
  <li>Was the “description” string a program for computing the larger string?  Was it perhaps some genetic encoding of developmental instructions?</li>
  <li>When the difference between the lengths of the two is large enough, this is supposed to be astronomically improbable.  Under what distribution?</li>
  <li>Is the probability somehow able to take into account the possibility of natural selection building up such
large difference step by step?</li>
</ol>

<p>Chapter 7 has a number of sections.  Let me describe briefly what each covers, and what it accomplishes.  I will summarize their arguments.  I will add comments of my own, in parentheses, or otherwise explicitly identified:</p>

<h4 id="insulating-evolution-against-small-probabilities-pages-321-326">Insulating Evolution against Small Probabilities (pages 321-326)</h4>

<p>Dembski and Ewert describe the success and wide acceptance of their design inference in evolutionary biology.  They then say “And now back to the real world”.  The success and acceptance, they accept, has not happened in reality.  This is because evolutionary biologists have been resistant to accepting that evolution of biological systems involves 
probabilities that small.  The evolutionary biologists have done so by invoking mechanisms of cumulative selection.  As an example Dembski and Ewert describe Richard Dawkins’s teaching example of a simple form of simulated natural selection starting with a random 28-letter sequence of letters and spaces, and reaching the target phrase “METHINKS IT IS LIKE A WEASEL” in hundreds or thousands of steps, instead of the 10<sup>40</sup> steps it would take if one simply drew new 28-letter sequences at random.</p>

<h4 id="variations-on-the-weasel-pages-326-330">Variations on the WEASEL (pages 326-330)</h4>

<p>Evolutionary biologists are obsessed by the Weasel example (I would say it is creationists and advocates of Design that 
are actually the ones obsessed with it). “Perhaps design theorists should shift focus to the origin of life”.  (The infamous OTOOL diversion – Off To the Origin Of Life).  There cumulative selection cannot play a role, they argue.  Simulations like the Weasel are declared to be “suggestive but misleading, so removed from biological reality that they cannot decide the matter”.</p>

<h4 id="resetting-darwinian-evolutions-bayesian-prior-pages-331-336">Resetting Darwinian Evolution’s Bayesian Prior (pages 331-336)</h4>

<p>Considering the choice between design and natural selection as a Bayesian Inference, it is important what prior probabilities one assigns these two hypotheses.  If one gives design a low enough prior probability, it cannot have any reasonably high posterior probability.  A design inference can be used to “sweep the field clear of all relevant chance hypotheses”. (Actually they could only do that if they could show that the probability of the data, given natural selection, was zero, but Demsbki and Ewert don’t quite mean that, they mean showing that the data is astronomically improbable).  Biologists argue that there may be many other “chance hypotheses”, but this is an argument but mere possibility as they they do not produce these.</p>

<h4 id="who-is-arguing-from-ignorance-pages-337-339">Who Is Arguing from Ignorance? (pages 337-339)</h4>

<p>The biologists are, since they essentially never produce the exact sequence of events needed to evolve these biological systems.  However, the biologists accuse the design theorists of arguing from ignorance.</p>

<h4 id="john-stuart-mills-method-of-difference-pages-340-345">John Stuart Mill’s Method of Difference (pages 340-345)</h4>

<p>Mill argued that if two situations differ in only one circumstance and a phenomenon occurs in one and not in the other, this circumstance is the effect, or cause.   Biologists such as Kenneth Miller say that all that is needed for natural selection to work is the presence of selection, replication, and mutation.  But these do not automatically lead to “complexity” or something “interesting”.  The evolution experiments of Sol Spiegelman and of Richard Lenski do not show those, but have genomes that rapidly decay.</p>

<h4 id="the-challenge-of-multiple-simultaneous-mutations-pages-345-352">The Challenge of Multiple Simultaneous Mutations (pages 345-352)</h4>

<p>This is basically a discussion of Michael Behe’s argument that major innovations require multiple mutations, each of which is individually disadvantageous, with an advantage only when they get together.  It is extremely improbable that these can be brought together by natural selection.</p>

<h4 id="what-to-make-of-bad-design-pages-353-to-368">What to Make of Bad Design? (pages 353 to 368)</h4>

<p>Bad design arguments are not plausible for a number of reasons.  Quotes from “some naturalistic thinkers” who do not think there is actual design but are impressed with the design of the  cell.  These include James Shapiro.  Discussion of whether there is junk DNA, giving great credit to the 2012 (supposed) refutation of junk DNA by the ENCODE Consortium.  Dembski and Ewert cite an embarrassing quote by Richard Dawkins in 2012 accepting the ENCODE result and arguing that it is compatible with his views.</p>

<h4 id="doing-the-calculation-pages-368-378">Doing the Calculation (pages 368-378)</h4>

<p>(At last, are we to be shown how?  Well, partly.)  Calculates the probability of a “biological system”, using protein folds as one example, invoking Douglas Axe’s experiments which that only one in 10<sup>-77</sup> randomly constructed proteins would have enough of the proper structure and sequence 
to make a minimally functional β-lactamase enzyme.  Axe’s paper was in <em>Journal of Molecular Biology</em>, volume 341, issue 5, page 1295 in 2004. The argument is based on the view that to get started evolving a β-lactamase enzyme one must have random assembly of such a protein, before natural selection can act. The example uses the protein sequence, with a 20-letter alphabet, as the string, equivalent to about a 648-bit string.  Dembski and 
Ewert do not try to quantitate the length of the “description” needed.  They acknowledge that evolutionary biologistss would argue that there might be other paths to the same functionality, without using these specific structures.</p>

<h4 id="the-breakdown-of-evolvability-pages-378-381">The Breakdown of Evolvability (pages 378-381)</h4>

<p>“whence the confidence that Darwinian evolution should allow untrammeled interconnectivity from any one living form to any other, whether by direct evolution or by indirect evolution from a common evolutionary precursor?”  They talk about protein alphabets, then uses a word game of changing a letter in a word, or deleting it, or adding a letter to change to another English word.  Argues that one cannot go from any word to any other this way.  Therefore although we have not proven that such interconnectedness in biological organisms cannot exist, we have shown it may not exist.  Therefore “there are indeed good reasons to think that a gradualist Darwinian form of evolution is very limited in what it can accomplish in biology, and that some features of biological systems not only resist Darwinian explanations but also invite design inferences”.  (This last part, the “some features” part, does not follow).</p>

<h4 id="where-to-look-for-small-probabilities-in-biology-pages-382-394">Where to Look for Small Probabilities in Biology? (pages 382-394)</h4>

<p>This, the last section in Chapter 7, talks about Michael Behe’s arguments about Irreducible Complexity.  Lots of discussion of how the bacterial flagellum could or could not have arisen.  Evolutionary biologists never seem to come up with a detailed series of steps.</p>

<h4 id="what-have-dembski-and-ewert-accomplished">What have Dembski and Ewert accomplished?</h4>

<p>Of the questions I asked above, they have not answered most of them.   The one example that they give that involves a sequence of digital symbol 
asks about the probability of a particular protein sequence arising from random sequences of amino acids.  What they give is really not an Algorithmically 
Specified Complexity agument, but a much older argument.   Usually that argument takes the lengths of modern proteins as a necessary requirement for protein function, and does not take into account that many random protein sequences (and random RNA sequences) show a variety of weak enzymatic function.  And small changes in those sequences often lead to similar amounts of function.  It is therefore not impossible for evolution to follow steps uphill on a fitness surface.</p>

<p>The difficulty with Dembski and Ewert’s definition of Algorithmically Specified Complexity is that when the quantity is large, it means that there is a long sequence (of something) which is “described” by a short sequence.  And there is simply no point in their argument that explains why (a) natural selection cannot achieve that, step by step, or (b) why achieving that is needed to result in any adaptation.  The matter is left without discussion.</p>

<p>There is also a large difference between the new argument and the previous specified complexity arguments.  As I noted in the first section of this review (12 June), the new concept is not a Functional Information argument.  The “probaability” is not the probability that a random sequence has this much <em>or more</em> of the desired trait.  It is just the probability of this sequence.  There is simply no discussion of how to compute a tail probability in some relevant distribution.  Or even in some nonrelevant distribution.</p>

<p>So we are left with the questions I asked, and re-asked, above.  How does this argument work?  What are the relevant distributions?  What is the “sequence”?  How does the “description” relate to some relevant probability?  Is it a tail probability under some relevant null distribution?  Is it a probability that ordinary evolutionary processes could achieve something?</p>

<h4 id="a-burden-of-proof">A burden of proof</h4>

<p>By not explaining how their arguments work, in spite of 400+ pages of explanation, Dembski and Ewert are engaged in wish-fulfillment.  They have “wished upon a star”.  But it is not true that, as the song claims, that if you do, “your dreams come true”.  Dembski and Ewert do not fill in the blanks, not even close.  They do go off sideways into irreducible complexity arguments, and they do repeatedly claim that evolutionary biologists argue that natural selection will always succeed in meeting challenges.  As the passenger pigeon about that, or the Great Auk, or the Dodo, or the Carolina Paroquet, or the mammoth.  Or the trilobites, or the seed ferns. or the ammonites.</p>

<p>It is still possible for them to answer my objections by filling in the blanks.  Saying that this is unfair, because biologists have not outlined every last step of the evolution of a complicated biological adaptation is no excuse.   I’ll be happy to hear from them how their arguments actually work.  We’re waiting …</p>


</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2024/11/dembskiewert2.html';
this.page.identifier = '/archives/2024/11/-dembskiewert2';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


