<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Icons of ID: Neutrality</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Icons of ID: Neutrality" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2004/07/icons-of-id-neu.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="PvM" />
  <meta property="article:published_time" content="2004-07-27T22:14:42-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2004/07/icons-of-id-neu.html">Icons of ID: Neutrality</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By PvM</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2004-07-27T22:14:42-07:00">July 27, 2004 22:14 MST</time></span>
	  </div>
   </header>
  <p>In this posting I will start exploring the importance of neutrality on evolution. In fact as the evidence will show, neutrality is not only a requirement for evolvability but it also can be selected for itself. In other words, evolutionary principles can lead to neutrality and evolution will tend towards areas with many neutral neighbours</p>

<p>Neutrality is a fascinating concept which has been shown of importance in understanding RNA and protein evolution.  Additionally the vaste neutral networks may help understand ‘convergent evolution’. Perhaps the lack of sequence similarity, but strong phenotype similarity can be understood by neutral evolution and drift. In other words, due to the vasteness of neutral networks, the sequence may have drifted while the phenotype remained basically the same. Thus what may appear to be convergent evolution may very well have been divergent evolution after all. And homologies, which fail to be detected at the sequence level may still exist at the phenotype level.</p>

<p><a href="http://www.tiem.utk.edu/~gavrila/PAPS/sf.pdf">Evolution and speciation in a hyperspace: the roles of neutrality, selection, mutation and random drift</a> Sergey Gavrilets</p>

<blockquote>
  <p>There is an important consequence of this observation. Because of the redundancy in the genotype-fitness map, different genotypes are bound to have very similar (identical from any practical point of view) fitnesses. Unless there is a strongly “non-random” assignment of fitnesses (say all well-fit genotypes are put together in a single “corner” of the genotype space), a possibility exists that well-fit genotypes might form connected clusters (or networks) that might extend to some degree throughout the genotype space. If this were so, populations might evolve along these clusters by single substitutions and diverge genetically without going through any adaptive valleys. Another consequence of the extremely high dimensionality of the genotype space is the increased importance of chance and contingency in evolutionary dynamics. Because a) mutation is random (which gene will be altered to which allele is unpredictable), b) each specific mutation has a very small probability, and c) the number of genes subject to mutation is very large, the genotypes present will be significantly affected by the random order in which mutations occur. Thus, mutational order represents a major source of stochasticity in evolution in the genotype hyperspace82;91;92. One should expect that even with identical initial conditions and environmental factors different populations will diverge genetically.</p>
</blockquote>

<h2 id="neutral-networks">Neutral networks</h2>

<p>A few words about the terminology. In what follows a neutral network is a contiguous set of sequences possessing the same fitness.</p>

<blockquote>
  <p>The existence of chains of well-fit genotypes that connect reproductively isolated genotypes was postulated by Dobzhansky and other earlier workers. In contrast, the models just described show it to be inevitable under broad conditions. The existence of percolating nearly-neutral networks of well-fit genotypes which allow for “nearly-neutral” divergence appears to be a general property of adaptive landscapes with a very large number of dimensions. Do existing experimental data substantiate this theoretical claim?</p>
</blockquote>

<h2 id="gene-duplication">Gene duplication</h2>

<blockquote>
  <p>Conrad 29 puts forward an idea of an “extra-dimensional bypass” on adaptive landscapes. According to Conrad an increase in the dimensionality of an adaptive landscape is expected to transform isolated peaks into saddle points that can be easily escaped resulting in continuing evolution.</p>
</blockquote>

<p>29 <a href="http://www.minet.uni-jena.de/~zauner/kp/mc/">Conrad</a>, M. “The Geometry of Evolution.” BioSystems 24 (1990): 61-81.</p>

<p>Such an increase in dimensionality is quite relevant since some ID proponents have argued that evolution cannot increase the <a href="http://www.iscid.org/boards/ubb-get_topic-f-6-t-000128.html">dimension of its search space</a>. Of course nature does not seem to be restricted by the imagination of ID proponents.</p>

<p>See also <a href="http://nn.cs.utexas.edu/downloads/papers/stanley.aaaiss03.pdf">Achieving High-Level Functionality through Complexification</a></p>

<blockquote>
  <p>Gene duplication allows nature to add new dimensions to the genetic space. Natural evolution can thus begin searching in a simple space even if more advanced phenotypes cannot be found in that space. Because major biological shifts in body-plan complexity have resulted from adding new genes, EC should be able to utilize this kind of mutation as well. However, adding new genes requires variable-length genomes, which can be difficult to implement, as the next section discusses.</p>
</blockquote>

<p><a href="http://www.neuroinformatik.ruhr-uni-bochum.de/PEOPLE/igel/NaSA.pdf">Neutrality and Self-Adaptation</a> Christian Igel</p>

<blockquote>
  <p>In the previous section, we gave examples of positive aspects of neutrality, e.g., in evolution strategies. However, one might argue that neutral encodings are in general disadvantageous, because they enlarge the search space (Radcliffe, 1991). In this section, we want to quantify this effect. Therefore, we derive the average time to  detect a globally optimal, solution in NFL scenarios depending on the cardinality of the search space. This enables us to compare general properties of search algorithms on search spaces of different sizes.</p>

  <p>…</p>

  <p>This means that in the considered NFL scenario enlarging the genotype space by adding redundancy without a bias does not considerably increase the average number of iterations needed to find a desirable solution if initially <strong>m</strong> is large enough.</p>

  <p>…</p>

  <p>In the worst case, when initially only one element encodes a desirable solution, still the deterioration of the average search performance is bounded by a factor of two.</p>
</blockquote>

<p>Note: <strong>m</strong> is the number of genotypes mapping to a an optimal solution</p>

<p>Finally Michael Conrad</p>

<blockquote>
  <p>The Geometry of Evolution</p>

  <p>Abstract. Some structures are more suitable for self-organization through the Darwin-Wallace mechanism of variation and selection than others. Such evolutionary adaptability (or evolvability) can itself evolve through variation and selection, either by virtue of being associated with reliability and stability or by hitchhiking along with the advantageous traits whose appearance it facilitates. In order for a structure to evolve there must be a reasonable probability that genetic variation carries it from one adaptive peak to another; at the same time the structure should not be overly unstable to phenotypic perturbations, as this is incompatible with occupying a peak. Organizations that are complex in terms of numbers of components and interactions are more likely to meet the former condition, but less likely to meet the latter. Biological structures that are characterized by a high degree of component redundancy and multiple weak interactions satisfy these conflicting pressures.</p>
</blockquote>

<p>The vast extension of the network of neutral paths suggests that extensive <em>neutral networks</em> pf sequences folding into the same structure percolate the entire sequence space.  the existence of extensive neutral networks meets a claim raised ny Maynard-Smith for protein space that are suitable for efficient evolution. The evolutionary implications of neutral networks are explored in detail in [30].  Empirical evidence for a large degree of functionaly neutrality in protein space was presented recently by Wain-Hobson and co-workers [34].</p>

<p>[34] Martinez MA, Pezo V, Marliere P, Wain-Hobson S. Exploring the functional robustness of an enzyme by in vitro evolution. EMBO J. 1996 Mar 15;15(6):1203-10.</p>

<blockquote>
  <p>The evolution of natural proteins is thought to have occurred by successive fixation of individual mutations. In vitro protein evolution seeks to accelerate this process. RNA hypermutagenesis, cDNA synthesis in the presence of biased dNTP concentrations, delivers elevated mutant and mutation frequencies. Here lineages of active enzymes descended from the homotetrameric 78 residue dihydrofolate reductase (DHFR) encoded by the Escherichia coli R67 plasmid were generated by iterative RNA hypermutagenesis, resulting in &gt; 20% amino acid replacement. The 22 residue N-terminus could be deleted yielding a minimum functional entity refractory to further changes, designating it as a determinant of R67 robustness. Complete substitution of the segment still allowed fixation of mutations. By the facile introduction of multiple mutations, RNA hypermutagenesis allows the generation of active proteins derived from extant genes through a mode unexplored by natural selection.</p>
</blockquote>

<p><img alt="ridges,jpg.JPG" src="/uploads/2005/ridges,jpg.JPG" width="366" height="296" border="0" /></p>

<p>The effect of neutrality on the shape of the fitness landscape. From <a href="http://www.cogs.susx.ac.uk/users/toms/Papers/Smith_EvolutionaryComputation2002.pdf">Fitness Landscapes and Evolvability</a></p>

<p>Neutrality is but one of the many amazing aspects of protein and RNA networks. Additionally there is the scale free nature which combined with neutrality can help understand many aspects of evolution such as robustness, evolvability, modularity, and degeneracy. And the fascinating part of it all is that scale free networks can be explained by simple models of gene duplication and divergence.</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2004/07/icons-of-id-neu.html';
this.page.identifier = '/archives/2004/07/icons-of-id-neu';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


