<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Steve Pinker's hair and the muscles of worms</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Steve Pinker's hair and the muscles of worms" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2011/12/steve-pinkers-h.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="PZ Myers" />
  <meta property="article:published_time" content="2011-12-19T12:32:11-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2011/12/steve-pinkers-h.html">Steve Pinker's hair and the muscles of worms</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By PZ Myers</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2011-12-19T12:32:11-07:00">December 19, 2011 12:32 MST</time></span>
	  </div>
   </header>
  <p class="lead">I've been guilty of teaching bean-bag genetics this semester. Bean-bag genetics treats individuals as a bag of irrelevant shape containing a collection of alleles (the "beans") that are sorted and disseminated by the rules of Mendel, and at its worst, assigns one trait to one allele; it's highly unrealistic. In my defense, it was necessary &mdash; first-year students struggle enough with the basic logic of elementary transmission genetics without adding great complications &mdash; and of course, in some contexts, such as population genetics, it is a <a href="http://ije.oxfordjournals.org/content/37/3/435.full">useful</a> <a href="http://www.nature.com/nature/journal/v409/n6822/full/409771a0.html">simplification</a>. It's just anathema to anyone more interested in the physiological and developmental side of genetics.</p>

<p>The heart of the problem is that it ignores the issue of translating genotype into phenotype. If you've ever had a basic genetics course, it's quite common to have been taught only one concept about the phenotype problem: that an allele is either dominant, in which case it is expressed as the phenotype, or it's recessive, in which case it is completely ignored unless it's the only allele present. This idea is so 19th century &mdash; it's an approximation made in the complete absence of any knowledge of the nature of genes.</p>



<a class="anchor" id="more"></a>

<p>And the "one gene, one trait" model violates everything we do know about the phenotype and genotype. <i>Every</i> gene is pleiotropic &mdash; it influences multiple traits to varying degrees. <i>Every</i> trait is multigenic &mdash; multiple genes contribute to the expression of every phenotypic detail. The bean-bag model is totally inadequate for describing the relationship of genes to physiology and morphology. Instead of a bean-bag, I prefer to think of the genome as comparable to <a href="http://scienceblogs.com/pharyngula/2006/03/high_geekology.php">a power spectrum, an expression of the organism in a completely different domain</a>. But I wrote about that previously, and I'll make this explanation a little simpler.</p>

<p>Here's the problem: you can't always reliably predict the phenotype from the genotype. We have a skewed perspective on the problem, because historically, genetics has first searched for strong phenotypes, and then gone looking for the genetic cause. We've been effectively blind to many subtle phenotypic effects, simply because we don't know how to find them. When we go the other way, and start by mutating known genes and then looking for changes in the phenotype, we're often surprised to discover no detectable change. One of the classic examples is the work of Elkins (1990), who found that mutating a neural cell adhesion gene, Fasciclin I, did not generate any gross defects. Mutating another gene, a signal transduction gene called Abelson tyrosine kinase, similarly had no visible effects. Mutating the two together, though &mdash; and this is a major clue to how these strange absences of effect could work &mdash; did produce gross and obvious effects on nervous system development.</p>

<p>Providing another great example, <a href="http://scienceblogs.com/pharyngula/2009/01/soon_well_all_have_steve_pinke.php">Steve Pinker examined his own genome</a>, and discovered that his genes said he was predisposed to be <a href="http://www.nytimes.com/2009/01/11/magazine/11Genome-t.html?_r=2&pagewanted=all">red-haired and at high risk for baldness</a>. If you've seen Steve Pinker, you know he's neither.</p>

<p>How can this be? As any geneticist will tell you, the background &mdash; the other alleles present in the organism &mdash; are important in defining the pattern of expression of a specific gene of interest. One simple possibility is that the genome contains redundancy: that a trait such as adhesion of axons in the nervous system or the amount of hair on the head can be the product of multiple genes, each doing pretty much the same thing, so knocking out one doesn't have a strong effect, because there is a backup present.</p>

<div style="text-align: center; font-size: 12px"><img src="http://scienceblogs.com/pharyngula/upload/2011/12/pairwise.jpeg" width="500" height="320" alt="pairwise.jpeg"/><br />Genetic interactions provide a general model for incomplete penetrance. Representation of a negative (synergistic) genetic interaction between two genes A and B. </div>

<p>So Steve Pinker could have seen that he has a defective Gene A, which is important in regulating hair, but maybe there's another Gene B lurking in the system that we haven't characterized yet, and which can compensate for a missing Gene A, and he has a particularly strong form of it. One explanation for a variable association between an allele and the phenotype, then, is that we simply don't have all the information about the multigenic cause of the phenotype, and there are other genes that can contribute.</p>

<p>This doesn't explain all of the observed phenomena, however. Identical twins who share the same complement of alleles also exhibit variability in the phenotype; we also have isogenic animal lines, where every individual has the same genetic complement, and they also show variability in phenotype. This is the problem of penetrance; penetrance is a genetics term that refers to the likelihood that an individual carrying an allele will actually express the phenotype associated with that allele&hellip;and it's not always 100%. </p>

<p>Again, the explanation lies in the other genes present in the organism. No gene functions all by itself; its expression is dependent on a cloud of other proteins &mdash; transcription factors, enhancers, chaperones &mdash; all of which modulate the gene of interest. We also have to deal with statistical variation in the degree of expression of all those modulatory factors, which vary by chance from cell to cell, and so the actual degree of activation of a gene may follow a kind of bell curve distribution. In the cartoon below, the little diamonds represent these partners; sometimes, just by chance, they'll be present in sufficiently high numbers to boost Gene B's output enough to fully compensate for a defective Gene A; in other cases, just by chance, they're too low in concentration to adequately compensate for the absence.</p>

<div style="text-align: center; font-size: 12px"><img src="http://scienceblogs.com/pharyngula/upload/2011/12/penetrance.jpeg" width="500" height="328" alt="penetrance.jpeg"/><br />Genetic interactions provide a general model for incomplete penetrance. A model for incomplete penetrance based on variation in the activity of genetic interaction partners.</div>

<p>What the above cartoon illustrates is the concept of <i>developmental noise</i>, the idea that the cumulative total of statistical variation in gene expression during development can produce significant phenotypic variation in the absence of any differences in the genotype. Developmental noise is a phrase bruited about quite a bit, and there's good reason to think it's valid: we can see quantitative variation in gene expression with molecular techniques, for instance. But at the same time we have other concepts, like redundancy and canalization, that work to buffer variation and produce reliable outputs from developmental processes, so we don't have many good examples where we can directly correlate subtle variation at the molecular level with clear morphological differences.</p>

<p>To test that, we have to go to simple animal models (it turns out that Steve Pinker is a rather intractable experimental animal). And here we have a very nice example in the nematode worm, <i>C. elegans</i>. In these experiments, the investigators were dealing with an isogenic strain &mdash; the genetic background was identical in all of the animals &mdash; raised in a uniform environment. They were looking at a mutant in the gene <i>tbf9</i>, which causes defects in muscle formation, but only 50% penetrance; that is, half the time, the mutants appeared completely normal, and the other half of the time they had grossly abnormal muscle development.</p>

<div style="text-align: center; font-size: 12px"><img src="http://scienceblogs.com/pharyngula/upload/2011/12/devnoise.jpeg" width="500" height="383" alt="devnoise.jpeg"/><br />Genetic interactions provide a general model for incomplete penetrance. Inactivation of the gene tbx-9 in C. elegans results in an incompletely penetrant defect, with approximately half of embryos hatching with abnormal morphology (small arrow).</div>

<p>See the big red question mark? That's the big question: can we trace the abnormal phenotype all the way back to random fluctuations in the expression of other genes in the animal? Yes, they can, otherwise it would never have been published in <i>Nature</i> and I wouldn't be writing about it now.</p>

<p>In this case, they have a situation analogous to the Gene A/Gene B cartoons above. Gene B is <i>tbx-9</i>; Gene B is a related gene, a duplicate called <i>tbx-8</i> which acts as a redundant copy. In the experiments below, they knock out <i>tbx-9</i> with a mutation, and then measure the quantity of other genes in the system using a very precise technique of quantitative fluorescence. Below, I've reproduced the entirety of their summary figure, because it is awesome &mdash; I just love the idea of being able to count the number of molecules expressed in a developing system. In order to avoid overwhelming everyone, though, I'll just describe a couple of the panels to give you the gist of the work.</p>

<p>First, just look at the top left panel, a. It's a plot of the level of expression of the <i>tbx-8</i> gene over time, where each line in the plot is a different animal. The lines in black are in the wild type animal, with fully functional copies of bothe <i>tbx-8</i> and <i>tbx-9</i>, and you should be able to see that there's a fair amount of variation in expression, about two-fold, in different individuals. The lines in green are from animals mutant for <i>tbx-9</i>; it's messy, but statistically what happens when <i>tbx-9</i> is knocked out, more <i>tbx-8</i> gene product is produced.</p>

<p>Panel e, just below it, shows the complementary experiment: the expression of <i>tbx-9</i> is shown for both wild type (black) and animals with <i>tbx-8</i> knocked out. Here, the difference is very clear: <i>tbx-9</i> levels are greatly elevated in the absence of <i>tbx-8</i>. This shows that <i>tbx-8</i> and <i>tbx-9</i> are actually tied together in a regulatory relationship where  levels of one rise in response to reduced levels of the other, and vice versa.</p>


<div style="text-align: center; font-size: 12px"><a href="http://scienceblogs.com/pharyngula/upload/2011/12/quanttbx.php" onclick="window.open('http://scienceblogs.com/pharyngula/upload/2011/12/quanttbx.php', 'popup', 'width=1250,height=1032,scrollbars=no,resizable=no,toolbar=no,directories=no,location=no,menubar=no,status=no,left=0,top=0' ); return false"><img src="http://scienceblogs.com/pharyngula/upload/2011/12/quanttbx-thumb-500x412-71413.jpeg" width="500" height="412" alt="quanttbx.jpeg"/><br />(Click for larger image)</a><br />Early inter-individual variation in the induction of ancestral gene duplicates predicts the outcome of inherited mutations. a, Quantification of total green fluorescent protein (GFP) expression from a tbx-8 reporter during embryonic development in WT (black) and tbx-9(ok2473) (green) individuals. Each individual is a separate line. a.u., Arbitrary units. b, Boxplot of tbx-8 reporter expression (a) showing 1.2-fold upregulation in a tbx-9 mutant at comma stage (~290 min, P=1.6x3 10<sup>-3</sup>, Wilcoxon rank test). c, Expression of tbx-8 reporter in a tbx-9(ok2473) background for embryos that hatch with (red) or without (blue, WT) a morphological defect. d, Boxplot of c showing tbx-8 expression is higher in tbx-9 embryos that develop a WT phenotype (blue) compared with those that develop an abnormal (red) phenotype at comma stage (P= 6.1x10<sup>-3</sup>). e, Expression of a ptbx-9::GFP reporter in WT (black) and tbx-8(ok656) mutant (green). f, Boxplot of tbx-9 reporter showing 4.3-fold upregulation at comma stage (~375 min, P=3.6x10<sup>-16</sup>). g, Expression of tbx-9 reporter in a tbx-8(ok656) mutant background, colour code as in
c. h, Boxplot of g showing tbx-9 expression is higher in tbx-8 embryos that develop a WT phenotype (P=0.033). i, Expression of a pflh-2::GFP reporter in WT (black) and flh-1(bc374) mutant (green). j, Boxplot of flh-2 reporter expression (i) showing 1.8-fold upregulation in a flh-1 mutant at comma stage (~180 min, P=2.2x10<sup>-16</sup>). k, Bright-field and fluorescence image of an approximate 100-cell flh-1; pflh-2::GFP embryo. Red arrow indicates the local expression of flh-2 reporter quantified for flh-1 phenotypic prediction.
l, Boxplot showing higher flh-2 reporter expression at approximate 100 cells for WT (blue) compared with abnormal (red) phenotypes (P=0.014). Boxplots show the median, quartiles, maximum and minimum expression in each data set.</div>

<p>Now skip over to the right, to panel c. All of the lines in this plot are of <i>tbx-8</i> expression in <i>tbx-9</i> mutants, and again you see a wide variation in levels of gene expression. In addition, the lines are color-coded by whether the worm developed normally (blue), or had the mutant phenotype (red). The answer: worms with low <i>tbx-8</i> levels were more likely to have the abnormal phenotype than those with high levels.</p>

<p>Panel g, just below it, is the complementary analysis of <i>tbx-9</i> levels in <i>tbx-8</i> mutants, and it gives the same answer.</p>

<p>Obviously, though, there is still a lot of variability unaccounted for; having relatively high levels of one or the other of the <i>tbx</i> genes didn't automatically mean the worm developed a wild-type phenotype. There's got to be something more that is varying. Look way back to the second cartoon I showed, with the little diamonds representing the cloud of transcription factors and chaperone proteins that modulate gene expression. Could there also be correlated variation there? And yes, there is. The authors looked at a chaperone protein called <i>daf-21</i> that is associated with the <i>tbx</i> system, and found, in mutants for <i>tbx-9</i>, that elevated levels of <i>daf-21</i> were associated with wildtype morphology (in blue), while lowered levels of <i>daf-21</i> were associated with the mutant phenotype.</p>

<div style="text-align: center; font-size: 12px"><a href="http://scienceblogs.com/pharyngula/upload/2011/12/daf21.php" onclick="window.open('http://scienceblogs.com/pharyngula/upload/2011/12/daf21.php', 'popup', 'width=747,height=388,scrollbars=no,resizable=no,toolbar=no,directories=no,location=no,menubar=no,status=no,left=0,top=0' ); return false"><img src="http://scienceblogs.com/pharyngula/upload/2011/12/daf21-thumb-500x259-71416.jpeg" width="500" height="259" alt="daf21.jpeg"/><br />(Click for larger image)</a><br />Expression of daf-21 reporter in a tbx-9(ok2473) mutant background. Embryos that hatch into phenotypically WT worms (blue) have higher expression than those hatching with a morphological defect (red) at the comma stage (P=1.9x10<sup>-3</sup>).</div>

<p>I know what you're thinking: there isn't a perfect correlation between high <i>daf-21</i> levels and wildtype morphology either. But when they do double-label experiments, and take into account both <i>daf-21</i> and <i>tbx-8</i> levels in <i>tbx-9</i> mutants, they found that 92% of the animals with greater than median levels of expression of <i>both</i> <i>daf-21</i> and <i>tbx-8</i> had wildtype morphology. It's still not perfect, but it's pretty darned good, and besides, it's no surprise that there are probably other modulatory factors with statistical variation lurking in the system.</p>

<p>What should you learn from this? Developmental noise is real, and is a product of statistical variation in the degree of expression of multiple genetic components that contribute to a phenotype. We can measure that molecular variation in living, developing systems and correlate it phenotypic outcomes. None of this is surprising; we <i>expect</i> that the process of gene expression is going to be a bit noisy, especially in these transcriptional regulators that are present in low concentration in the cell, anyway. But the other cool thing we can observe here is that having multiple noisy systems that interact with each other can produce a more reliable, robust signal and contribute to the fidelity of developmental outcomes.</p>

<hr />
<p class="ref">Burga A, Casanueva MO, Lehner B (2011) 
Predicting mutation outcome from early stochastic variation in genetic interaction partners. Nature 480(7376):250-3.</p>

<p class="ref">Elkins T, Zinn K, McAllister L, Hoffmann FM, Goodman CS (1990)
Genetic analysis of a Drosophila neural cell adhesion molecule: interaction of fasciclin I and Abelson tyrosine kinase mutations. Cell 60(4):565-75.</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2011/12/steve-pinkers-h.html';
this.page.identifier = '/archives/2011/12/steve-pinkers-h';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


