<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Rumors of pseudogenes' demise greatly exaggerated, new study says</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Rumors of pseudogenes' demise greatly exaggerated, new study says" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2006/08/rumors-of-pseud.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Andrea Bottaro" />
  <meta property="article:published_time" content="2006-08-04T11:30:09-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2006/08/rumors-of-pseud.html">Rumors of pseudogenes' demise greatly exaggerated, new study says</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Andrea Bottaro</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2006-08-04T11:30:09-07:00">August 4, 2006 11:30 MST</time></span>
	  </div>
   </header>
  <p>I imagine that reading scientific journals is mostly a drag for ID advocates: all those papers highlighting evolutionary mechanisms, identifying transitional fossils, veryfing phylogenetic-based predictions must be really irritating.  There are, however, few and far between papers that set the ID advocates’ hearts all aflutter, so when one appears, they make sure to milk it for all its P.R. worth. One of the recent examples was a <a href="http://www.nature.com/nature/journal/v423/n6935/abs/nature01535.html">2003 paper by Hirotsune and colleagues</a> in Nature, which reported that alteration of the pattern of expression of a purported mouse pseudogene (i.e. an apparently inactivated, non-functional gene, part of the so-called “junk DNA”) results unexpectedly in the modification of the activity of its functional counterpart, leading to a series of dramatic developmental defects.</p>

<p>Now, one of the recurrent claims of Intelligent Design is that most if not all of the features of any organism, including its entire genome, should somehow be useful.  Thus, ID advocates enthusiastically latched on Hirotsune’s publication as supporting their claim that “junk DNA” is, in fact, not junk at all (indeed, if you Google for “Hirotsune” and “Intelligent design”, you will find over 100 hits). Alas for them, a <a href="http://www.pnas.org/cgi/content/abstract/0602216103v1">brand new paper by Todd Gray and colleagues in the Proceedings of the National Academy of Sciences</a> appears to completely refute the original findings.</p>

<p>As usual, let me give the background first.  Pseudogenes can be defined as the remnants of once-coding DNA sequences that have undergone a more or less significant loss of their ability to encode for any product.  There are two main sources of pseudogene formation (see figure 1).  The first is traditional gene duplication, followed by the loss of functionality by one of the duplicated members. The second is by a mechanism called retrotransposition - in short, the messenger RNA of a gene, instead of being used for protein synthesis, is transcribed back into a DNA sequence and inserted into the genome, forming so-called “retrogenes”.  Unlike pseudogenes arising from gene duplication, which often retain the original regulatory elements required for gene expression (promoters/enhancers), retrogenes lack regulatory sequences, and therefore their only chance of becoming expressed is when they integrate in proximity of some other gene’s promoter (rather unlikely, although not impossible) [3,4].  If they don’t become expressed and aquire some selectable function, they quickly degenerate, turning into retrotransposed or “processed” pseudogenes.  A third, more rare instance is that of functional genes losing their function in certain lineages, but not others (“unitary” pseudogenes - a classic example is the independent inactivation of the gene for vitamin C synthesis in primates and guinea pigs).
<img src="/uploads/2006/retrogenes-jpg.JPG" alt="retrogenes-jpg.JPG" width="600" height="434" />
<em><strong>Figure 1: Origins of pseudogenes:</strong> A. Retrotransposed pseudogenes: starting from the original gene (the coding sequences are in black, the non-coding introns in gray, and the promoter element is indicated by the large arrow upstream of the gene), transcription generates a primary mRNA (black and gray broken line), from which the introns are excised by RNA splicing.  This mature mRNA, which contains only exons and a poly-adenosine tail, is transcribed back into DNA by enzymes called reverse transcriptases, and the DNA is reinserted back into the genome.  Hence, the pseudogene product will lack intron and promoter sequences, and will bear characteristic repeat sequences at the insertion site, due to the integration mechanism.   B. Duplicated pseudogenes: DNA duplication generates a more-or-less faithful copy of the original gene, including introns and, in many cases, promoter and other transcriptional regulatory elements. In most cases, this duplicated gene will undergo crippling, inactivating mutations and turn into a pseudogene (in rarer cases, the duplicated copy will acquire new functions and become a new gene). (Adapted from [3].)</em></p>

<p>Thus, pseudogenes - and especially retrotransposed pseudogenes - are generally considered to be non-functional relics and, together with other sorts of repetitive and “selfish” DNA elements, as well as other unique DNA sequences, form the so-called “junk DNA”. (For a more general discussion of “junk DNA”, see <a href="/archives/2005/12/another-example.html"> Ian Musgrave’s discussion here at PT</a>.)  Indeed, when the pseudogenes can be followed over evolutionary lineages, they appear to evolve neutrally, accumulating mutations progressively and freely until they become almost unrecognizable, or disappear from the genome altogether. Note that the number of pseudogenes in the human genome (20,000 or so at the latest count, many of them crippled viral elements) is comparable to that of our functional genes - an impressive amount.</p>

<p>Despite its connotations, the phrase “junk DNA” (originated by Susumu Ohno in 1972) does not intend to convey an absolute and irreversible lack of function.  Indeed, as it is often noted, had that been the case “garbage DNA” would have been a better term.  In fact, “junk” is what accumulates in people’s basements and attics, not immediately useful but not nasty or burdensome enough to be quickly discarded – indeed, something that may occasionally be found to be of use (at least, that’s what I tell my wife).  Another problem with the term is that it is unfortunately often misused (in the lay press and especially by Creationists, although some scientists are guilty as well) to simply denote DNA that does not directly encode any protein sequence - which is absolutely wrong.  It has long been known, in some cases even before the term was coined, that DNA contains important non-coding elements involved in gene transcription (e.g. the promoter and enhancer elements mentioned above), RNA splicing and polyadenylation, chromosome dynamics, etc.  In addition, instances exist where the sequence of a particular stretch of DNA is irrelevant, but its presence may be important, as in the case of introns, certain “spacer’ regions, and so on.  Still, while it is clear that the term “junk DNA” should be used advisedly (if at all) there are good reasons to think that large swaths of the genome of most eukaryotic organisms are indeed non-functional, in part because these stretches of DNA accumulate mutations neutrally, and diverge much faster than known functional elements, and also because vast differences in DNA amount, presence of large duplications/deletions of intergenic regions, as well as gain and loss of specific pseudogenes are often observed in closely related organisms (see again <a href="/archives/2005/12/another-example.html"> Ian’s piece</a>).</p>

<p>Ironically, the term “junk DNA” was originally often used as pretty much a swipe by supporters of the “neutral theory” (which argues that much of what goes on in the genome is actually non-adaptive and not subject to selective forces) against the more strictly neo-Darwinian “pan-adaptationists”, for whom the long hand of natural selection reaches every nook and cranny of an organism’s phenotype and genotype, continually getting rid of even mildly noxious or just useless features, like some obsessive-compulsive cleaner.  With time, however, the accumulating evidence for selective neutrality of large parts of the genome convinced even the more strict Darwinians, and only Creationists and ID supporters have remained to argue the ultra-Darwinian pan-adaptationist position (although of course from different premises).</p>

<p>ID advocates in particular have often discussed junk DNA as an important issue for their “theory”. For instance, in his quasi-peer-reviewed <a href="http://www.discovery.org/scripts/viewDB/index.php?command=view&amp;id=2177">paper in the Proceedings of the Biological Society of Washington</a>, Stephen Meyer claimed that</p>

<blockquote>
  <p>Advocates of the design hypotheses on the other hand, would have predicted that non-coding regions of the genome might well reveal hidden functions, not only because design theorists do not think that new genetic information arises by a trial and error process of mutation and selection, but also because designed systems are often functionally polyvalent.
(Meyer SC, Proc Biol Soc Wash 117:213-239. 2004)</p>
</blockquote>

<p>Jonathan Wells wrote</p>

<blockquote>
  <p>From an ID perspective, however, it is extremely unlikely that an organism would expend its resources on preserving and transmitting so much “junk”. 
(<a href="http://www.iscid.org/papers/Wells_TOPS_051304.pdf">Wells J, PCID, Vol 3.1, 2004</a>)
<em>[Note here the similarity to the ultra-Darwinian argument: carrying “junk DNA” looks like a waste of resources, hence maladaptive for the pan-adaptationist, or “bad engineering” for the ID advocate. AB]</em></p>
</blockquote>

<p>which prompted the Discovery Institute’s Casey Luskin <a href="http://www.discovery.org/scripts/viewDB/filesDB-download.php?command=download&amp;id=546">to boldly venture (while lawyerly hedging his bets)</a> that ID predicts that</p>

<blockquote>
  <p>Much so-called “junk DNA” will turn out to perform valuable functions.</p>
</blockquote>

<p>ID connoisseurs would perceptively note here that these strong statements about “junk DNA”’s function seem to contrast with the shyness of ID advocates regarding the attributes, goal and identity of the Designer (why couldn’t She be a compulsive junk collector, as far as Her critters’ genomes go, after all?). But ID advocates have of course at least two good reasons to support this view: first, the admission of a careless or incompetent Designer would constitute a bad P.R. move with the movement’s religious supporters, and second, if uniquely identifiable junk DNA elements, like pseudogenes, are indeed non-functional, then their observed transmission along evolutionary lines (see for instance <a href="http://www.pubmedcentral.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16469101">this paper</a>) is powerful evidence of common descent, with which most ID advocates have yet to make peace.</p>

<p>Given this context, Hirotsune’s paper in 2003 must have seemed like a dream come true.  The paper’s findings were actually serendipitous, but totally striking: while generating a transgenic mouse strain (i.e. a strain of mice in which an artificial gene has been added to the genome, in order to study its effects), Hirotsune realized that one of their strains presented an unusual phenotype in which various developmental defects in the kidneys and bones presented themselves at very high frequency, but only when the transgene was inherited from the father (genes that are expressed differentially when they are inherited maternally or paternally are called “imprinted”). Looking at the genomic region in which the transgene was inserted, the authors found a processed pseudogene (Makorin-1p1, or <em>Mkrn1-p1</em> in genetic notation) with similarity to an expressed, functional gene called <em>Mkrn1</em>.</p>

<p>By further characterizing the phenomenon at the molecular level, the authors claimed that the <em>Mkrn1-p1</em> pseudogene showed transcription into RNA only when inherited paternally (i.e., it was itself imprinted), that this expression was diminished by insertion of the transgene in its proximity, that the <em>Mkrn1-p1</em> RNA product could regulate expression of the Mkrn1 functional gene by affecting stability of its mRNA, and that the phenotype due to <em>Mkrn1-p1</em> suppression could be rescued by enforcing expression of either <em>Mkrn1</em> or <em>Mkrn1-p1</em>  RNA.  In short, they claimed to have demonstrated that RNA from a processed pseudogene can play a regulatory role in the expression of its ancestral, protein-coding gene counterpart.</p>

<p>Now, this is without doubt a very interesting finding, although of limited general applicability (as I mentioned, the majority of retrogenes do not give rise to any RNA), and of course ID advocates jumped on it with a vengeance.  In the wake of the Hirotsune paper, <a href="http://www.discovery.org/scripts/viewDB/index.php?command=view&amp;id=1448">Mike Behe even submitted a letter</a> to Nature, which declined publishing it.  Among other things, the letter stated:</p>

<blockquote>
  <p>The modern molecular example of poor design is pseudogenes. Why litter a genome with useless, broken copies of functional genes? It looks just like the aftermath of a blind, wasteful process. No designer would have done it that way.(2) Yet Hirotsune et al (3) show that at least one pseudogene has a function. If at least some pseudogenes have unsuspected functions, however, might not other biological features that strike us as odd also have functions we have not yet discovered? Might even the backwards wiring of the vertebrate eye serve some useful purpose?</p>
</blockquote>

<p>Cautionary Note: if you wish to read the entire text, please make sure to shut off your irony meters first: in the letter, Behe sternly warns the scientific community against the perils of purely negative argumentation, and chastises them for naively trusting their “intuition” regarding biological function - rather cheeky, for someone whose main arguments against evolutionary theory are that we don’t have a complete mutation-by-mutation model of the evolution of certain biological structures, and that</p>

<blockquote>
  <p>The strong appearance of design allows a disarmingly simple argument: if it looks, walks and quacks like a duck, then, absent compelling evidence to the contrary, we have warrant to conclude it’s a duck. (Michael Behe, <a href="http://www.discovery.org/scripts/viewDB/index.php?command=view&amp;id=2415">“Design for Living”</a>, The New York Times, 2/7/2005)</p>
</blockquote>

<p>Anyway, back to the main story.  Although ID advocates are keen to claim that the “darwinian orthodoxy” routinely suppresses or ignores inconvenient results, Hirotsune’s paper caused quite a splash, and rapidly accumulated over 100 citations in the scientific literature, most endorsing the new model of “pseudogene trans-regulation” (<em>trans</em> here is lingo for “acting on another chromosome”) proposed by the authors [e.g. 4,5].   Other scientists pursued the lead trying to investigate the conservation of the putative regulatory portion of <em>Mkrn1-p1</em>, albeit with mixed results [6, 7].   Still, there were some reasons for skepticism – for instance, the <em>Mkrn1-p1</em> retrogene, despite its purported crucial role in mice, is absent from the genome of all other mammals tested, including closely related rats.  In addition, the Makorin gene family counts at least 3 functional members in the mouse (including, of interest, a <em>bona fide</em> functional retrogene, which is transcribed, conserved in mammals, and encodes the Makorin-3 protein), as well as numerous pseudogenes, which can complicate molecular and sequence analysis. Finally, previous studies on mice with chromosomal alterations including the segment near the <em>Mkrn1-p1</em> gene suggested that no imprinted gene with deleterious effects was present in the region.</p>

<p>Enter Gray and colleagues, which in their PNAS paper systematically re-analyzed the original story and tested some of its predictions.   Their findings consistently contradicted the Nature paper’s conclusions: they found that the <em>Mkrn1-p1</em> pseudogene is not transcribed at all, and that the RNA attributed to the pseudogene by Hirotsune is actually a variant form of transcripts from the functional gene; that the pseudogene’s DNA is extensively modified by methylation, a known hallmark of transcriptional inactivity; that neither <em>Mkrn1</em> nor <em>Mkrn1-p1</em> are imprinted; and finally that inactivation of the functional <em>Mkrn1</em> gene does not bring about the changes observed in Hirotsune’s transgenic mice.  Gray therefore concludes that Hirotsune’s data were mostly artifactual, and (quite generously) propose some alternative mechanisms of how those findings came to be originated.</p>

<p>Where does this leave us with regard to pseudogenes?  Actually, pretty much where we were before the Gray paper came out. If you take away the hype and ignore the wishful thinking of ID supporters, the evidence still overwhelmingly supports the notion that many, likely most pseudogenes are functionless, and it does so regardless of the validity of Hirotsune’s findings.  Indeed, if one assumes that evolutionary conservation of DNA sequences is a strong hallmark of potential function, then <a href="http://compbiol.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371/journal.pcbi.0020046">a recent study by a Swedish group</a> shows that at best a few dozens of the thousands of pseudogenes in the human and mouse genomes are under sufficient selective pressure to be highly conserved between the two lineages, suggesting they may be functional [8]. Still, there is ample room for potential interesting mechanisms by which pseudogenes can on occasion be recruited into regulatory and structural functions.</p>

<p>There is, of course, also an important lesson about science here: Hirotsune’s provocative, out-of-the-mainstream findings were not rejected on principle, but were given wide exposure, embraced by some as explanatory of certain processes, put to the test by others, and invalidated.  Of course, this will apply to Gray’s data as well – it is now up to Hirotsune and his supporters to test the new findings and explain them away, or accept them.  Stay tuned.</p>

<p><strong>Acknowledgements</strong>
Thanks to Ian, Nick, Douglas, Reed, Dunk Erik and the rest of the PT crew for useful comments and suggestions.</p>

<p><strong>REFERENCES</strong></p>
<ol>
  <li>
    <p>Hirotsune S, Yoshida N, Chen A, Garrett L, Sugiyama F, Takahashi S, Yagami K, Wynshaw-Boris A, Yoshiki A. An expressed pseudogene regulates the messenger-RNA stability of its homologous coding gene. Nature. 2003 423:91-6.</p>
  </li>
  <li>
    <p>Gray TA, Wilson A, Fortin PJ, Nicholls RD. The putatively functional Mkrn1-p1 pseudogene is neither expressed nor imprinted, nor does it regulate its source gene in trans. Proc Natl Acad Sci USA. 2006 Aug 1; [Epub ahead of print]</p>
  </li>
  <li>
    <p>D’Errico I, Gadaleta G, Saccone C. Pseudogenes in metazoa: origin and features. Brief Funct Genomic Proteomic. 2004 3:157-67.</p>
  </li>
  <li>
    <p>Balakirev ES, Ayala FJ. Pseudogenes: are they “junk” or functional DNA? Annu Rev Genet. 2003;37:123-51.</p>
  </li>
  <li>
    <p>Zhang Z, Gerstein M. Large-scale analysis of pseudogenes in the human genome. Curr Opin Genet Dev. 2004 14:328-35.</p>
  </li>
  <li>
    <p>Podlaha O, Zhang J. Nonneutral evolution of the transcribed pseudogene Makorin1-p1 in mice. Mol Biol Evol. 2004 21:2202-9.</p>
  </li>
  <li>
    <p>Kaneko S, Aki I, Tsuda K, Mekada K, Moriwaki K, Takahata N, Satta Y. Origin and evolution of processed pseudogenes that stabilize functional Makorin1 mRNAs in mice, primates and other mammals. Genetics. 2006 172:2421-9.</p>
  </li>
  <li>
    <p>Svensson O, Arvestad L, Lagergren J. Genome-wide survey for biologically functional pseudogenes. PLoS Comput Biol. 2006 2:e46.</p>
  </li>
</ol>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2006/08/rumors-of-pseud.html';
this.page.identifier = '/archives/2006/08/rumors-of-pseud';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


