<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Meyer v Universal Genetic Code: Common Descent</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Meyer v Universal Genetic Code: Common Descent" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2004/08/meyer-v-univers.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="PvM" />
  <meta property="article:published_time" content="2004-08-30T20:52:47-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2004/08/meyer-v-univers.html">Meyer v Universal Genetic Code: Common Descent</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By PvM</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2004-08-30T20:52:47-07:00">August 30, 2004 20:52 MST</time></span>
	  </div>
   </header>
  <p>After PBS aired the very successful series ‘<a href="http://www.pbs.org/wgbh/evolution/">Evolution</a>’, Meyer, as directory of the Discovery Institute’s Center for <del>the Renewal of</del> Science and Culture, wrote the following somewhat puzzling <a href="http://www.arn.org/docs/meyer/sm_meyerwhitworth.htm">letter to the editor</a></p>

<p>Meyer is objecting to the claim by PBS that there exists a Universal Genetic Code.</p>

<blockquote>
  <p>Secondly, and more importantly, the existence of these variant codes is not consistent with a key prediction derived from Darwin’s theory of universal common ancestry. To see why, imagine typing on a keyboard in which the assignment between the keys and the letters that appear on your screen have been secretly changed. When you hit a specific letter such as an “n,” a different letter such as “t” appears. Or, imagine that every time you hit, say, an “o,” a period and a double space appears on your screen. Now envision submitting such a paper to a professor (without any information about the special new code that your computer used). Will your paper make sense? Will you get a good grade? I doubt it.</p>
</blockquote>

<p>What is this fascination of Meyer to use linguistic strawmen arguments [1] when trying to discuss evolutionary concepts? But as others have shown, the genetic code shows, not surprisingly,  support for (neo)Darwinian theory. The variants are found as small twigs within the evolutionary tree of life. In fact, since Darwin argued for one or more common ancestors, I find Meyer’s claim not only incorrect but also exhibiting what is more commonly known as ‘a strawman’ argument.</p>

<blockquote>
  <p>In a similar way, changes in the genetic code will inevitably result in the production of some amino acid sequences that will not fold into functional (i.e., biologically meaningful) proteins–much to the detriment of the organism. Indeed, many of the variant codes in nature either insert a “stop” (the equivalent of a period
) where, in the standard code, a specific amino acid would have been, or they continue to produce amino acids where previously a “stop” would have been. Both these kinds of changes are hardly trivial from a functional point of view.</p>
</blockquote>

<p>Note how Meyer claims that changes in the genetic code will <strong>inevitably</strong> result in the production of some amino acid sequences that do not fold into functional proteins and that this is detrimental to the organism. But he provides no supporting evidence for his claims and in fact various of the extended codes involve additional amino acids. Nobody is arguing that these changes are trivial, after all the genetic code has been mostly frozen for a significant period of time. What is argued is that the evidence of variant codes is well explained by common descent. 
Now science from <a href="http://www.evolvingcode.net/uni_code.php">Evolvingcode.net</a></p>

<p><a href="http://www.evolvingcode.net/avery/NSC.html">Chart of variants in the genetic code</a> Must see (requires Shockwave plugin)!!!</p>

<p>The variations in the genetic code are mostly minor but there is more variation than originally expected.</p>

<ul>
  <li>non standard codes are now known within organisms from all 3 domains of life (Archea, Bacteria and Eukaryotes)</li>
  <li>it seems likely that many more non-standard codes await discovery (the pattern of non-standard codes strongly hint at observer bias, in that the most thoroughly studied lineages contain the most non-standard codes).</li>
  <li>the code has even changed in the large, nuclear genomes of some eukaryote nuclear genomes, including yeast (Fungi) and many ciliates (Protozoa)</li>
  <li>recent findings have shown that deviations from the standard code extend to include organisms that use amino acids over and above the 20 amino acids of the Standard Genetic Code.</li>
</ul>

<p><a href="http://www.evolvingcode.net/code_red.php">Link</a></p>

<blockquote>
  <p><strong>Do non-standard codes weaken the claim for a common ancestor for all life on earth?</strong></p>

  <p>In a word, “no”: the existence of non-standard codes offers no support for a ‘multiple origins’ view of life on earth. Lineages that exhibit non-standard codes are clearly and unambiguously related to organisms that use the standard code: they are distributed as small ‘twigs’ within the evolutionary tree of life. To believe otherwise, one would have necessarily infer that, for example, certain groups of ciliates evolved entirely separately from the rest of life, including other types of ciliates!</p>
</blockquote>

<p>Knight observes quite appropriately</p>

<blockquote>
  <p>Though certain pseudo-scientific groups seeke to use the existence of non-standard codes to criticize evolutionary theory (notably the ‘intelligent design’ Discovery Institute of Seattle , their claims conveniently and completely ignore this big picture of molecular similarities in order to suit their loaded agenda.</p>
</blockquote>

<p>There are various theoretical explanations how the genetic code can evolved, these include: Codon capture, Codon ambiguity, and Genome streamlining.</p>

<p>But perhaps Meyer can enlighten us as to his hypothesis of the evolution of the genetic code?</p>

<p>Thought so…</p>

<p>In “<a href="http://www.evolvingcode.net/documents/rewiring.pdf">REWIRING THE KEYBOARD: EVOLVABILITY OF THE GENETIC CODE</a>” by Robin D. Knight, Stephen J. Freeland and Laura F. Landweber the authors observe</p>

<blockquote>
  <p>Curiously, many of the same codons are reassigned in independent lineages, frequently between the same two meanings6, indicating that there may be an underlying predisposition towards certain reassignments. <strong>At least one of these changes seems to confer a direct selective advantage7</strong>, showing that the code is evolvable in the formal sense that the mapping between genotype and phenotype allows adaptive changes8.</p>
</blockquote>

<ol>
  <li>Santos, M. A., Cheesman, C., Costa, V., Moradas-Ferreira, P. &amp; Tuite, M. F. <a href="http://www.bio.ua.pt/genomica/Lab/1999MolMicro.pdf">Selective advantages created by codon ambiguity allowed for the evolution of an alternative genetic code in Candida spp. </a> Mol. Microbiol. 31, 937-947 (1999).</li>
</ol>

<p>in <a href="http://www.bio.ua.pt/genomica/Lab/2004TIGS.pdf">Driving change: the evolution of alternative genetic codes</a> Manuel A.S. Santos et al present a review in “TRENDS in Genetics Vol.20 No.2 February 2004” of plausible mechanisms and explanations.</p>

<p>and finally in <a href="http://bayes.colorado.edu/Papers/jme01.pdf">How Mitochondria Redefine the Code</a>  Robin D. Knight, Laura F. Landweber, and Michael Yarus show how the genetic code in mitochondria evolved.</p>

<hr />

<h2 id="references">References</h2>

<p><a href="http://www.biomedcentral.com/news/20030815/03">Extending the genetic code</a> A novel strain of yeast with an expanded genetic code of 21 amino acids   By David Secko</p>

<p>Chin JW, Cropp TA, Anderson JC, Mukherji M, Zhang Z, Schultz PG., <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12920298&amp;dopt=Abstract"><strong>An expanded eukaryotic genetic code.</strong></a>, Science. 2003 Aug 15;301(5635):964-7.</p>

<blockquote>
  <p>We describe a general and rapid route for the addition of unnatural amino acids to the genetic code of Saccharomyces cerevisiae. Five amino acids have been incorporated into proteins efficiently and with high fidelity in response to the nonsense codon TAG. The side chains of these amino acids contain a keto group, which can be uniquely modified in vitro and in vivo with a wide range of chemical probes and reagents; a heavy atom-containing amino acid for structural studies; and photocrosslinkers for cellular studies of protein interactions. This methodology not only removes the constraints imposed by the genetic code on our ability to manipulate protein structure and function in yeast, it provides a gateway to the systematic expansion of the genetic codes of multicellular eukaryotes.</p>
</blockquote>

<p>Osawa S, Jukes TH., <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2499683&amp;dopt=Abstract"><strong>Codon reassignment (codon capture) in evolution</strong></a>.  J Mol Evol. 1989 Apr;28(4):271-8.</p>

<blockquote>
  <p>The genetic code, once thought to be “frozen,” shows variations from the universal code. Variations are found in mitochondria, Mycoplasma, and ciliated protozoa. The variations result from reassignment of codons, especially stop codons. The reassignments take place by disappearance of a codon from coding sequences, followed by its reappearance in a new role. Simultaneously, a changed anticodon must appear. We discuss the role of directional mutation pressure in the events, and we also describe the possibility that such events have taken place during early evolution of the genetic code and can occur during its present evolution.</p>
</blockquote>

<p>Yokobori S, Suzuki T, Watanabe K., <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11675591&amp;dopt=Abstract"><strong>Genetic code variations in mitochondria: tRNA as a major determinant of genetic code plasticity</strong></a> J Mol Evol. 2001 Oct-Nov;53(4-5):314-26.</p>

<blockquote>
  <p>Characteristic features of tRNA such as the anticodon sequence and modified nucleotides in the anticodon loop are thought to be crucial effectors for promoting or restricting codon reassignment. Our recent findings on basepairing rules between anticodon and codon in various metazoan mitochondria suggest that the complete loss of a codon is not necessarily essential for codon reassignment to take place. We postulate that a possible competition between two tRNAs with cognate anticodon sequences towards the relevant codon to be varied has a potential role in codon reassignment. Our proposition can be viewed as an expanded version of the codon capture theory proposed by Osawa and Jukes (J Mol Evol 28: 271-278, 1989).</p>
</blockquote>

<p>Castresana J, Feldmaier-Fuchs G, Paabo S., <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9520430&amp;dopt=Abstract"><strong>Codon reassignment and amino acid composition in hemichordate mitochondria</strong></a>,  Proc Natl Acad Sci U S A. 1998 Mar 31;95(7):3703-7.</p>

<blockquote>
  <p>In the mitochondrial genome of the hemichordate Balanoglossus carnosus, the codon AAA, which is assigned to lysine in most metazoans but to asparagine in echinoderms, is absent. Furthermore, the lysine tRNA gene carries an anticodon substitution that renders its gene product unable to decode AAA codons, whereas the asparagine tRNA gene has not changed to encode a tRNA with the ability to recognize AAA codons. Thus, the hemichordate mitochondrial genome can be regarded as an intermediate in the process of reassignment of mitochondrial AAA codons, where most metazoans represent the ancestral situation and the echinoderms the derived situation. This lends support to the codon capture hypothesis. We also show that the reassignment of the AAA codon is associated with a reduction in the relative abundance of lysine residues in mitochondrial proteins.</p>
</blockquote>

<p>Anderson JC, Wu N, Santoro SW, Lakshman V, King DS, Schultz PG. An expanded genetic code with a functional quadruplet codon. Proc Natl Acad Sci U S A. 2004 May 18;101(20):7566-71. Epub 2004 May 11.</p>

<blockquote>
  <p>With few exceptions the genetic codes of all known organisms encode the same 20 amino acids, yet all that is required to add a new building block are a unique tRNA/aminoacyl-tRNA synthetase pair, a source of the amino acid, and a unique codon that specifies the amino acid. For example, the amber nonsense codon, TAG, together with orthogonal Methanococcus jannaschii or Escherichia coli tRNA/synthetase pairs have been used to genetically encode a variety of unnatural amino acids in E. coli and yeast, respectively. However, the availability of noncoding triplet codons ultimately limits the number of amino acids encoded by any organism. Here, we report the design and generation of an orthogonal synthetase/tRNA pair derived from archaeal tRNA(Lys) sequences that efficiently and selectively incorporates an unnatural amino acid into proteins in response to the quadruplet codon, AGGA. Frameshift suppression with L-homoglutamine (hGln) does not significantly affect protein yields or cell growth rates and is mutually orthogonal with amber suppression, permitting the simultaneous incorporation of two unnatural amino acids, hGln and O-methyl-L-tyrosine, at distinct positions within myoglobin. This work suggests that neither the number of available triplet codons nor the translational machinery itself represents a significant barrier to further expansion of the genetic code.</p>
</blockquote>

<hr />

<h2 id="further-readings">Further Readings</h2>

<p>[1] Meyer, Stephen C. 2004. The origin of biological information and the higher taxonomic categories. Proceedings of the Biological Society of Washington 117(2):213-239.</p>

<p>Atkins, J.F., and R. Gesteland. 2002. The 22nd amino acid. Science 296(May 24):1409-1410. Summary available at http://www.sciencemag.org/cgi/content/summary/296/5572/1409.</p>

<p>Gorman, J. 2003. Amending the genetic code: Yeast adds new amino acids to its proteins. Science News 164(Aug. 16):102. Available to subscribers at http://www.sciencenews.org/articles/20030816/fob7.asp.</p>

<p><strong>__</strong>. 2003. Unnatural biochemistry: Bacteria make and use an alien amino acid. Science News 163(Jan. 25):53. Available to subscribers at http://www.sciencenews.org/articles/20030125/fob6.asp.</p>

<p>Hesman, T. 2000. Code breakers. Science News 157(June 3):360. Available at http://www.sciencenews.org/articles/20000603/bob9.asp.</p>

<p>Kwon, I., et al. 2003. Breaking the degeneracy of the genetic code. Journal of the American Chemical Society 125:7512-7513.</p>

<p>Michael Eisenstein,<a href="http://www.nature.com/cgi-taf/dynapage.taf?file=/nmeth/journal/v1/n1/full/nmeth017.html">Gained in Translation: Expanding the Genetic Code</a>, Microbiology</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2004/08/meyer-v-univers.html';
this.page.identifier = '/archives/2004/08/meyer-v-univers';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


