<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Expanding the genetic alphabet</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Expanding the genetic alphabet" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2014/05/expanding-the-g.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Matt Young" />
  <meta property="article:published_time" content="2014-05-15T11:09:54-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2014/05/expanding-the-g.html">Expanding the genetic alphabet</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Matt Young</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2014-05-15T11:09:54-07:00">May 15, 2014 11:09 MST</time></span>
	  </div>
   </header>
  <p>By <a href="http://wasdarwinwrong.com">Gert Korthof</a>.</p>

<p>“It is a very, very hard problem. Getting all the pieces to work means re-engineering 3.5 billion years of evolution. It’s so ambitious.” These are the words of a scientist 14 years ago [1]. He was talking about the project of inventing new bases to include in DNA and let the new DNA function in a living cell. Now an important milestone has been reached towards that goal. Scientists have succeeded in adding a new base pair, d5SICS (call it X) and dNaM (call it Y), to the DNA of the bacterium <em>E. coli</em> and let it replicate its DNA [2]. Furthermore, the growth rate of the bacterium was not significantly slowed down, and the DNA-repair system did not remove the new bases. These results give us hope for realistic answers to the so far speculative questions as to why evolution settled on only two base pairs in DNA. Could there be superior bases and superior DNA?</p>

<figure>
<img src="/uploads/2014/Plasmid_modified_from_wiki.jpg" alt="Plasmid_modified_from_wiki.jpg" width="600" height="282" />
<figcaption>
<a href="http://en.wikipedia.org/wiki/Plasmid">Plasmid</a>, modified from Wikipedia. The yellow dot is the new base pair.

</figcaption>
</figure>

<p>The researchers included the new base pair in a plasmid and introduced it in to the bacterium <em>E. coli</em> (the yellow dot in the illustration is the new base pair). The plasmid replicated including the new base pair. This is a significant result, because there are many requirements for bases in DNA in general.</p>

<p>For example, any base pair must enable DNA to form a regular double helix with fixed dimensions over long distances; a base must attach properly to the DNA-backbone; any base in a pair must pair exclusively to its partner and not to bases of other pairs; any base must not pair with itself; the base pairing must be strong enough to be stable, but not too strong to make the process of strand separation (required for replication) too difficult; etc.</p>

<p>Once new bases are developed, many new possibilities are opened. For example, consider the effect on the coding potential of the genetic code. How many new amino acids could the bacterium produce? Let’s see. We are not replacing any of the existing bases. So, the DNA of the modified bacterium has now six different bases: A,T,C,G,X,Y and three base pairs: AT, CG, XY. Any position of its DNA can be a A,T,C,G,X,Y. Three bases in a row code an amino acid. There are 20 natural amino acids. Instead of the standard 4 × 4 × 4 = 64, there are now 6 × 6 × 6 = 216 different triplets or codons. If we subtract the 3 triplets of the standard 3 stop codons - which do not code for an amino acid - we are left with 61 in the old and no less than 213 different triplets available for coding amino acids in the new bacterium.</p>

<p>That is, theoretically. Life on earth uses 61 triplets to code for 20 amino acids. That is a redundant coding system. On average there are about 3 codons per amino acid. Theoretically, such a coding system could code for 61 amino acids. But evolution decided otherwise. If we leave intact the redundant coding system for the standard 20 amino acids, the modified bacterium theoretically could code for 216 – 64 (including stop codons) = 152 extra amino acids. A huge gain! Proteins can be imagined with properties not possible with the standard code.</p>

<p>At the moment it is too early to say anything sensible about the performance of the new bases. First, the bases need to code for an amino acid, and the transcription and translation system must be working. And it would be nice to observe the new bases in the bacterial chromosomal DNA.</p>

<p>It would also be nice to observe the behavior of the new bases in eukaryotic chromosomes (plants and animals). Even if it turned out that the overall performance of alternative bases is inferior to the natural bases in DNA, that would be valuable information. The question whether the 4 universal bases in DNA are uniquely fit for coding the genetic information of life on earth isn’t pure speculation any more. It can now be attacked experimentally.</p>

<p>Science has also a short News feature about the research [4].</p>

<p>Notes</p>
<ol>
  <li>
    <p>Robert F. Service (2000), <a href="http://www.sciencemag.org/content/289/5477/232">Creation’s Seventh Day</a>, Science, Volume 289, issue of 14 Jul 2000, pp. 232-235.</p>
  </li>
  <li>
    <p>Ross Thyer, Jared Ellefson (2014), <a href="http://www.nature.com/nature/journal/v509/n7500/full/nature13335.html">Synthetic biology: New letters for life’s alphabet</a>, Nature, News and Views, 15 May 2014 (online 7 May 2014) (not free).</p>
  </li>
  <li>
    <p>Denis A. Malyshev et al. (2014), <a href="http://www.nature.com/nature/journal/v509/n7500/abs/nature13314.html">A semi-synthetic organism with an expanded genetic alphabet</a>, Nature, 15 May 2014 (online 7 May 2014) (this is the original research article).</p>
  </li>
  <li>
    <p>Robert F. Service (2014) <a href="http://news.sciencemag.org/biology/2014/05/designer-microbes-expand-lifes-genetic-alphabet">Designer Microbes Expand Life’s Genetic Alphabet</a>, Science, News &amp; Analysis, 9 May 2014 (free).</p>
  </li>
</ol>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2014/05/expanding-the-g.html';
this.page.identifier = '/archives/2014/05/expanding-the-g';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


