<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Breakthrough for Intelligent Design? (Part 2)</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Breakthrough for Intelligent Design? (Part 2)" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2022/11/erkell2.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Lars Johan Erkell" />
  <meta property="article:published_time" content="2022-11-16T13:00:00-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2022/11/erkell2.html">Breakthrough for Intelligent Design? (Part 2)</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Lars Johan Erkell</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2022-11-16T13:00:00-07:00">November 16, 2022 13:00 MST</time></span>
	  </div>
   </header>
  <figure><img src="/uploads/2022/Biologg_Banner_900.jpg" alt="Biolog(g) banner" />
<figcaption>The text is a pun in Swedish: "Logg" means something like a ship´s log or a logbook. A blog is "blogg". 
So "Biolog(g)" is a biologist´s log or blog.</figcaption>
</figure>

<p>&nbsp;</p>
<p><em>This is part 2 of a series of 7 posts by Lars Johan Erkell, with comments on each by Ola Hössjer and a reply by Erkell.  <a href="https://pandasthumb.org/archives/2022/11/erkell8.html">An eighth post by Erkell</a> is near this one, put here because he refers to it. Part 1 will be found <a href="http://pandasthumb.org/archives/2022/11/erkell1.html">here</a>.</em></p>

<h2 id="breakthrough-for-intelligent-design-part-2">Breakthrough for Intelligent design? (Part 2)</h2>
<p>November 13, 2020</p>

<p>by Lars Johan Erkell</p>

<p><strong>How do you calculate design?</strong></p>

<p>What stands out about Thorvaldsen and Hössjer’s work, and what makes me write these blog posts, is that they raise the question of how one could demonstrate fine-tuning and design using scientific methods. In any case, how one could imagine doing so; it doesn’t get more concrete than that.</p>

<p>Thorvaldsen and Hössjer carefully discuss the statistical methods they use, or rather believe that one can use. The main line is – as always with ID – that you try to support ID by primarily criticizing the theory of evolution. But you cannot prove theory A by showing that theory B is bad. The authors are also aware of this. They write in section 6.3:</p>

<blockquote>
  <p>However, one may argue that the most suitable approach of science is to compare the best explanations found so far within two competing worldviews. This naturally leads to model selection. … It is possible for some problems to suggest a design model M1 that competes with the currently most promising naturalistic model M2, in terms of which model explains the data the best.
The authors therefore want to compare an evolutionary model with a design model by using statistical methods to determine which of them is more probable. And it is clear that the intention is to make the comparison based on concrete data. But from which models, and from which data?</p>
</blockquote>

<p>In their critique of evolutionary theory, a number of well-known themes are addressed, primarily “irreducible complexity” on several levels; in individual proteins, in protein complexes and in entire networks. Also the “Cambrian explosion” and the so-called the waiting time problem is addressed. What these have in common is that the authors believe that the structures of life are so complex and thus so improbable that they could never arise naturally. However, this is not a view shared by established science – here the arguments are seen as contrived.</p>

<a class="anchor" id="more"></a>

<p>The question now is, in practical terms, these problems could be calculated. A question that has been dealt with extensively - and which is probably the easiest to calculate - concerns the probability that a functioning protein will be able to arise through naturalistic mechanisms. Thorvaldsen and Hössjer mention in section 4.2 two strategies for doing this:</p>

<p>The first is based on the fact that protein consists of one or more chains of amino acids that must be in a certain sequence. Since there are 20 different amino acids to choose from, there are countless combinations. In a small protein consisting of 150 amino acids, the different amino acids can be combined in 10<sup>50</sup> to 10<sup>74</sup> different ways. These are astronomically large numbers; the probability that the “right” combination would occur by chance is vanishingly small. The big problem with this strategy, however, is that no sane biologist believes that proteins form in this random way.</p>

<p>The evolutionary biological view is instead that the first enzymes consisted of short peptide chains with a small number of amino acids – it is known that such chains can have enzymatic activity.<sup>1</sup> That such peptide chains could be formed by chance is not unreasonable. They may not have been as efficient as today’s enzymes, but if they had sufficient enzyme activity, they would have been selected and evolved into larger and more efficient molecules. So no one believes that proteins are formed in the way that Thorvaldsen and Hössjer are suggesting. It is a little strange that they mention this bizarre idea at all. It is completely irrelevant.</p>

<p>The other strategy is to set up a hypothetical evolutionary tree for how a certain protein might have evolved, and then calculate the probability of that process. But there is a lot of knowledge that is missing in order to be able to make credible calculations of this kind, not least because we have to reckon with an evolution that extends over billions of years. We would need detailed knowledge of which organisms are involved, mutation frequencies, population sizes, selection pressure and much else that is impossible to know in retrospect. This is not a realistic path. Nor, as far as I know, has anyone attempted to set up any detailed scenario of this kind.</p>

<p>The conclusion is that it is not practically possible to statistically calculate the probability that a protein could evolve via natural mechanisms. The difficulties are so enormous that it will probably never be possible. And yet this was the simplest of the cases discussed.</p>

<div align="center">***</div>
<p>Thorvaldsen and Hössjer’s strategy for demonstrating fine-tuning is based on a statistical comparison between a naturalistic model M2 and a design model M1. We then need to calculate the probability for both models in order to compare them. We have already seen that it is practically impossible to calculate the probability that a complex biological structure can arise according to a naturalistic model. But how to calculate the likelihood of a design model? It is something the authors wisely do not go into. Here we are in big trouble.</p>

<p>In order to be able to calculate the probability of how a complex biological structure can arise, we must have a credible and detailed model of the process. This also applies if there is a conscious design behind it. We must therefore have knowledge of the designer in order to assess the probability that he/she/it wanted the design exactly the way we see it. This requires detailed knowledge – we need to understand exactly how the designer thinks, what the intentions, plans and means are. In order to make statistical calculations, we must have actual data, numbers.</p>

<p>Design proponents, however, are careful not to say anything about who the designer might be, or what qualities the designer might have. This makes it fundamentally impossible to calculate the statistical probability of a design process. So it’s not just practically impossible, as with the naturalistic model, it’s fundamentally impossible.</p>

<p>The statistical model Thorvaldsen and Hössjer have put forward to demonstrate design is misconceived and unworkable – it is based on the unreasonable assumption that it is possible to make a statistical estimate of the probability that an unknown designer would have acted in a certain way. It is a mystery that they chose to publish it.</p>

<p>Notes:</p>

<ol>
  <li>Short peptides can spontaneously form stable complexes that have enzymatic activity. See: Jason Greenwald et. al.: Peptide Amyloids in the Origin of Life. J Mol Biol (2018) vol. 430, pp. 3735–3750 <a href="https://pubmed.ncbi.nlm.nih.gov/29890117/">here</a></li>
</ol>

<p>&nbsp;</p>

<p><strong>Comment by Ola Hössjer</strong>
June 29, 2021 3:00 a.m</p>

<p><strong>Part 2. About counting on design</strong></p>

<p>In order to be able to estimate the probability that a biological system should have arisen by chance (that is, step (a) of the fine-tuning argument) a naturalistic model M2 is required that describes how the system came about or evolved. The fact that the model is naturalistic means that it only involves natural explanatory mechanisms. When it comes to biological evolution, i.e. the development of complicated life forms from simple single-celled organisms, the five naturalistic mechanisms are usually involved (i) mutations, (ii) genetic drift, i.e. random variation in reproductivity when parents pass on their DNA, (iii) natural selection, (iv) recombinations of genetic material from a pair of parents, and v) the mixing of gene pools caused by geographic migration.</p>

<p>Erkell claims that it is practically impossible to give a reasonable estimate of the probability that, for example, a protein could have evolved in accordance with a naturalistic model M2. I agree that this is extremely difficult, since such a model must necessarily be complicated, with more or less unknown parameters. Nevertheless, I believe that it is possible to demonstrate the difficulty of naturalistic processes to produce complex structures. If one only focuses on certain properties of the protein, that according to M2 must have evolved, a conservative upper bound can be found for the probability that the protein came about through evolutionary processes. In addition, it is possible to account for the fact that several of the model parameters (such as selection pressure, mutation rate, population sizes, generation times, etc.) are difficult to estimate from data with good accuracy, by performing a so-called sensitivity analysis in order to calculate how the upper bound of the probability for the protein to evolve, changes when the constituent parameters change. If this upper bound of the probability is small for all biologically reasonable values of the model parameters, an argument has been made for the difficulty of the proposed naturalistic model M2 to explain the origin of the protein of interest. I therefore argue, contrary to Erkell, that the first step (a) of our method for detecting fine-tuning is highly feasible in practice.</p>

<p>The second part (b) of our fine-tuning argument consists in finding a measure f(x) of how specified a biological system x is. As mentioned in the introduction, f(x) focuses on some characteristic that the observer recognizes, which in many cases brings to mind that an intelligent designer is behind the origin of the system. In TH2020 we give several examples of how the system x and its specification f(x) can be chosen. For instance, x may correspond to a protein or a molecular machine, with a specification that tells how many parts that must be in place for x to function. Another example is that f(x) indicates the biological fitness of an organism x, that is, its reproductive capacity. In connection with the latter example, we argue in TH2020 that evolution will eventually break down the genetic material of a species because most mutations are harmful or neutral. The consequence is that the fitness of individuals of the species decreases over time. This in turn means that the probability in (a) becomes very low for an evolutionary model M2, based on the aforementioned mechanisms (i)-(v), to be able to explain that the organism in question arose during millions of years of development, because its genetic material during such a long time would be depleted.</p>

<p>But if a naturalistic model M2 has difficulties explaining the emergence of a biological system that contains some form of specification, the question remains whether there is any other better explanatory model. At the end of TH2020, we give examples of design- or creationist-inspired models M1 that we believe have greater possibilities than M2 to explain the emergence of the fine-tuned system. To make such reasoning more rigorous, it is necessary to show that the probability of such a system to emerge is much higher for the design model M1 than for the naturalistic model M2. In the example above, where f(x) corresponds to the biological fitness of an organism x, we refer to various articles where a design model is described, which, in addition to the aforementioned (micro)evolutionary mechanisms (i)-(v) adds a sixth mechanism (vi) in the form of genetic variation in a first created pair, from which all individuals in the species descend. According to such a design model M1, the emergence of species can be explained by a combination of created variation (mechanism (vi)) and short-term, small-scale microevolution (mechanisms (i)-(v)). The problem of genetic depletion is then greatly reduced, since created variation makes it possible to assume that the species arose relatively recently, so that the genetic erosion, caused by the harmful mutations, acted over a much shorter time.sup&gt;2&lt;/sup&gt;.</p>

<p>Erkell is strongly critical of us introducing a design model to explain the emergence of a biological system. He writes:</p>

<blockquote>
  <p>In order to be able to calculate the probability of how a complex biological structure can arise, we must have a credible and detailed model of the process. This also applies if there is a conscious design behind it. We must therefore have knowledge of the designer in order to assess the probability that he/she/it wanted the design exactly the way we see it. This requires detailed knowledge – we need to understand exactly how the designer thinks, what the intentions, plans and means are. In order to make statistical calculations, we must have actual data, numbers.</p>

  <p>Design proponents, however, are careful not to say anything about who the designer might be, or what qualities the designer might have. This makes it fundamentally impossible to calculate the statistical probability of a design process. So it’s not just practically impossible, as with the naturalistic model, it’s fundamentally impossible.</p>
</blockquote>

<p>I completely agree with Erkell that it is difficult to determine exactly what plans or intentions a designer has. But this is not necessary to know. It is enough to let design-inspired thinking generate concrete falsifiable hypotheses.  It is therefore possible to combine a design-inspired model with the hypothetico-deductive method, which forms the basis of empirical science, according to the following three steps:</p>

<p>1) Propose a falsifiable hypothesis, which may involve both natural mechanisms and design-inspired parts.</p>

<ol>
  <li>Evaluate (through deduction and induction) and check if the hypothesis in (1) is consistent with the data.</li>
</ol>

<p>3) If the agreement in (2) is not good, go back to (1) and propose a better hypothesis.</p>

<p>Erkell makes very far-reaching, not to say unreasonable, demands on the design model, because he does not think that falsifiable hypotheses are enough. He also wants to include characteristics of the designer into the model, that often can be very difficult to test scientifically. Although it is interesting to have a separate conversation about the intentions of the designer, this has rather to do with the interpretation of and the reason for the emergence of the testable hypotheses, which is not required for the hypothetico-deductive method. One can thus use more or less intuitive design arguments to formulate testable hypotheses, without having to test the reasoning that led to the formulation of these hypotheses.</p>

<p>Erkell’s reasoning also backfires like a boomerang on himself. Shouldn’t he make the same rigorous demands on naturalistic models M2, where the interpretation of these models is included? How, for example, should randomness of an evolutionary model be explained? Is it epistemic chance (as a way of describing lack of knowledge), ontological chance (genuine non-determinism) or should quantum mechanics be used to explain the emergence of chance? And in the latter case, which interpretation of quantum mechanics should be used?</p>

<ol>
  <li>This is usually called Haldane’s dilemma. A historical overview of Fisher’s Fundamental Theorem of Natural selection (which is closely related to Haldane’s Dilemma) is given in Basener W., Cordova S., Hössjer O. and Sanford J. (2021). Dynamical Systems and Fitness Maximization in Evolutionary Biology. In: Sriraman B. (eds) Handbook of the Mathematics of the Arts and Sciences. Running, Cham.</li>
</ol>

<p>&nbsp;</p>

<p><strong>Reply by Lars Johan Erkell</strong>
September 8, 2021 6:12 am</p>

<p>In this section, we get into the core of the work, your statistical model to detect fine-tuning/design. It assumes that you can calculate the probability that a biological structure can evolve via natural mechanisms, an extremely difficult task. You write that you can make meaningful estimates of the probability of a protein being able to evolve by doing a sensitivity analysis of an upper estimate of this probability. If the variation of this estimate is small for all biologically plausible values of the involved parameters, one would have an argument for the difficulty of the proposed naturalistic model.</p>

<p>But how do you get “all biologically reasonable values”? It is not enough to count on a single gene – genes (and proteins) interact in complex networks, and selection pressure acts on the entire network. The interaction between genes is called epistasis, and it must be included in your model. Ecology is also crucial because the selection pressure is entirely dependent on the organism’s environment. The problem is that there are astronomically many possibilities for epistatic (and ecological) interactions, and you have no clues as to which might be the relevant ones. You cannot count on a certain protein “all else being equal”; because the genome is constantly changing, “all else is not equal”. All of this means that you are nowhere near a realistic model of an evolution that has taken place in countless steps over millions or billions of years.</p>

<p>Apart from all this, there is also a fundamental flaw in the approach, namely that it assumes the probability of a process with a definite goal in the form of a definite structure (at least that’s how ID advocates tend to calculate). But evolutionary processes have no goal. As far as proteins and protein complexes are concerned, there are a huge number of structures that have the same or similar function, and it is one of these that will be selected. It is thus the probability that the function - not the exact structure - will evolve that is important.</p>

<p>In any case, you mean that “our method of detecting fine-tuning is highly feasible”. If so, why haven’t you done so? Why don’t you at least have a discussion of which factors would be included in such a calculation, how they should be identified, how they should be estimated, how they should be weighed against each other, etc.? Then—and only then—would this article be interesting. It would be even more interesting if you could succeed with the second part of your method, namely calculating the probability of design. If you are to test the probability that a certain structure is designed, you must know the characteristics of the designer - design is an act of will, and it is the probability of that act of will that you must calculate. And that is of course impossible without in-depth knowledge of the designer. You don’t write anything about how this would work, and that doesn’t surprise me; I suspect you have no idea. At least I don’t have it. But this is necessary if you want to be able to compare M1 and M2. That comparison is the very core of your method. But you comment on it like this:</p>

<p>“Erkell makes very far-reaching, not to say unreasonable, demands on the design model, because he is not satisfied with falsifiable hypotheses. Instead, he also wants to include characteristics of the designer that can often be difficult to test scientifically.”</p>

<p>There’s no getting away from this: the probability of a design hypothesis includes the probability that a designer really wants to accomplish a certain thing. Without intention no design. If we imagine an omnipotent designer (which you probably do) it is only an act of will; no practical complications exist. But I get curious; you write “characteristics of the designer that can often be difficult to test scientifically”, which must mean that there are actually characteristics of the designer that can be scientifically tested. What are they? How can you test them? Here we get into really central questions about scientific hypotheses. That discussion is so important and extensive that it hardly fits in the comments section. I have therefore written <a href="https://github.com/PandasThumb/pandasthumb.github.io/edit/source/_posts/2022/2022-11-09-erkell8.md">a separate post</a> on the matter, “Science, hypotheses and Intelligent Design”.</p>

<p>In your comment you also touch on completely different things: you talk about “genetic erosion” and refer in a footnote to “Haldane’s dilemma”. Here you are confusing two things. Haldane’s dilemma is not about some kind of erosion 
but about how fast evolution can go, more specifically whether there was enough time for the gene substitutions that must have taken place from the time of the common ancestor of humans and chimpanzees until today. The genetic erosion you talk about is John Sanford’s genetic entropy, a completely different matter. It would consist of a slow and inevitable genetic decay that would lead to our demise. However, neither Sanford nor anyone else has been able to show that this decay actually occur. Sanford builds on an earlier, now dismissed, idea that a genetic “catastrophe” could occur if enough harmful mutations accumulate in a genome. However, it has never been shown that such catastrophes actually take place, but rather that they do not (Springman, R. et al: Evolution at a High Imposed Mutation Rate: Adaptation Obscures the Load in Phage T7. Genetics (2010)184: 221 –232); Summers, J., Litwin, S.: Examining The Theory of Error Catastrophe. J. Virol., (2006) 80, 20–260). So forget about genetic entropy.</p>

<p>In conclusion, you suggest that I should place the same strict demands on my own research as I do on yours. But that’s exactly what I do - I make the same demands on your research that I myself have to live up to. Regarding your question about chance, I (and my colleagues) assume an ontological chance, since chemical reactions are based on the interaction between elementary particles and these are subject to the quantum laws in their Copenhagen interpretation. This is the basis of the molecular orbital theory, and thus of the chemistry and biology, I was once taught.</p>


</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2022/11/erkell2.html';
this.page.identifier = '/archives/2022/11/erkell2';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


