<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Protein folding -- a step closer?</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Protein folding -- a step closer?" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2005/10/protein-folding.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Steve Reuland" />
  <meta property="article:published_time" content="2005-10-04T02:18:58-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2005/10/protein-folding.html">Protein folding -- a step closer?</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Steve Reuland</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2005-10-04T02:18:58-07:00">October 4, 2005 02:18 MST</time></span>
	  </div>
   </header>
  <p>The Holy Grail of biochemistry is what’s known as the “<a href="http://www.ram.org/research/pfp.html">protein folding problem</a>”.  We’ve long known that most proteins’ function is based on their 3-dimentional structures (i.e. their folds), and that these structures are dependent on their linear amino acid sequences.  What we haven’t known is how exactly their amino acid sequences determine their structures.  This would be an extremely useful thing to know, because currently the only way to determine a protein’s 3D structure is through <a href="http://www.rcsb.org/pdb/experimental_methods.html#xrd">X-ray crystallography</a> or <a href="http://www.rcsb.org/pdb/experimental_methods.html#nmr">NMR</a>, both of which are expensive, time-consuming, and aren’t even guaranteed to work.  If the protein folding problem were solved, the thousands and thousands of proteins that we’ve sequenced could have their structures deciphered almost overnight, revolutionizing our understanding of how proteins function.  Of course it probably won’t happen that way; the protein folding problem will probably be slowly chipped away at rather than solved in one fell swoop, and crystallographers and NMR spectroscopists will remain gainfully employed for the foreseeable future.</p>

<p>But two papers published in a recent issue of <em>Nature</em> from the lab of <a href="http://www.hhmi.swmed.edu/Labs/rr/">Rama Ranganathan</a> knocked a chip out of the protein folding problem.  Their results show us that protein folding might be simpler than we think.</p>

<p>In the first paper (<a href="http://www.nature.com/nature/journal/v437/n7058/full/nature03991.html">Socolich <em>et al</em></a>), they attempt to figure out just how many and what kind of interactions are responsible for a protein adopting its fold.  As proteins collapse into a globular state, various amino acids interact with each other in order to hold the protein in a certain shape.  But how many interactions might this be, and how deep do they go?  Socolich <em>et al</em> used an evolution-based method called statistical coupling analysis (SCA).  This technique works by aligning a large number of homologous protein sequences and looking for those amino acids that are “statistically coupled” – that is, those which do not vary randomly with respect to each other, but rather tend to coevolve with each other.  These amino acids, they reason, must interact with each other in some important fashion, because you rarely find one changed without its partner being changed accordingly.  What’s really interesting is that SCA studies seem to indicate that a relatively small number of amino acid interactions are responsible for a protein’s fold, and that it takes only a small amount of sequence information to specify such a fold.</p>

<p>Socolich <em>et al</em> decided to put this idea to the test by generating artificial proteins using only information available from the SCA.  If the SCA results were accurately representing protein folds as “low sequence information” entities, then it should be possible to make artificial proteins that folded properly by restricting themselves to said information.  The researchers used a short protein domain known as the <a href="http://www.bork.embl-heidelberg.de/Modules/ww-gif.html">WW domain</a>, which works well because it has a large number of sequenced members.  (In natural proteins, WW domains typically bind proline-rich sequences.)  To do this, they produced two computer algorithms to generate artificial sequences.  The first (called IC, for site-independent conservation) only takes the pattern of amino acid conservation into account and shuffles the amino acids at each position according to how often they occur.  For this algorithm, there is no statistical coupling – each position is allowed to vary without regard to other positions.  The second (called CC, for coupled conservation) kept the conservation pattern but also took statistical coupling into account.</p>

<figure>
<img src="http://www.nature.com/nature/journal/v437/n7058/images/nature03991-f1.2.jpg" alt="" width="600" height="257" />
<figcaption><strong>FIGURE 1. SCA-based protein design.</strong>  <strong>a</strong>, Structure of a representative WW domain (Nedd4.3, Protein Data Bank 1I5H) in complex with a target peptide (in stick representation). The two canonical tryptophans are shown as space-filling side chains. The figure was prepared using PyMol51. <strong>b</strong>, SCA conservation scores for each position in the WW alignment in arbitrary units of statistical energy12. Position numbers (x axis) and the secondary structure diagram at the top coincide with matrix columns in <strong>câˆ’e. c</strong>, A matrix representation of statistical coupling values from perturbation analysis of five positions (rows) in the WW domain MSA. <strong>d</strong>, The matrix for an alignment of IC sequences, built by randomly selecting amino acids at each site from the observed frequency distributions in the natural alignment. <strong>e</strong>, The matrix for an alignment of CC sequences, derived from a design algorithm where both the conservation pattern and the pattern of statistical couplings in the natural alignment are preserved. Scale bar shows the SCA coevolution score, ranging from 0 (blue) to 2 (red).
</figcaption>
</figure>

<p>The researchers then generated protein libraries based on these two sets of possible proteins, along with a library of natural WW domains and randomly generated proteins as positive and negative controls, respectively.  They found that most of the natural proteins folded properly.  (There was no guarantee that this would be the case, since the proteins were being expressed in bacteria.)  Not surprisingly, none of the random proteins were observed to fold properly.  Also, none of the proteins that had only their amino acids patterns conserved, but no statistical coupling (the IC group), were able to fold either.  What is remarkable is that a large percentage (12 out of 43) of the statistically coupled proteins (the CC group) were found to be folded, and were within the range of thermal stability for native WW domains.  This is in spite of the fact that they had low average sequence identity (35%) to the native proteins.  These results show that protein folds can be specified by a fairly small amount of information – the amino acid conservation pattern along with a small number of statistically coupled residues.  High sequence identity is apparently not necessary.</p>

<p>But they didn’t stop there.  In the second paper (<a href="http://www.nature.com/nature/journal/v437/n7058/full/nature03990.html">Russ <em>et al</em></a>), the researchers decided to test their artificial proteins for function.  Just because the proteins folded properly there is no guarantee that they’re functional.  And it’s functionality, afterall, that is selected for.  Amazingly, they found that the binding function of the artificial proteins was indistinguishable from that of the natural proteins – they bound proline target sequences just as the native proteins did, and could even be assigned to the same functional classes.  So we see that not only is proper folding specified by the conservation pattern and a small number of coevolving residues, but that function is as well.</p>

<p>These results are interesting because by providing insight into what it takes for a protein to fold properly, they bring us a step closer to solving the protein folding problem.  It had long been thought that conserved residues were crucial for determining a protein’s structure.  And while this is certainly true, it was never by itself adequate to explain just what it took for a protein to fold.  For one thing, there is the odd fact that even in large proteins, only a small handful of residues are strictly conserved.  Most conserved residues will occasionally vary.  If you see a residue that’s conserved 80% of the time, that’s a good reason to think it’s important.  But why is it allowed to vary the other 20% of the time?  It has long been presumed that interactions between amino acids will allow some residues to change while requiring others to remain the same, and that this is very much context dependent, such that even closely related proteins will differ in which mutations they will or won’t tolerate.  But this raises the question: how much additional sequence information, aside from conserved residues, is necessary to determine a protein’s structure?  If the results here are any indication, not much.  It may be that a small number of coupled, coevolving residues contain all the additional sequence information that we need.  If this is the case, the whole business might be far simpler than previously thought.</p>

<p>I will add one caveat though: just because these results work well with WW domains, there is no guarantee that they’ll work with other types of proteins.  Far too often have people thought that the solution to the protein folding problem was within reach, only to see their carefully crafted program work well with the particular protein family that it was fine-tuned to work with, yet fail miserably with everything else.  We’ll need plenty of additional research to see if this one pans out.  And I should point out that <em>de novo</em> structural prediction wasn’t what the researchers were going for here.  It’s just that their research provides the type of knowledge that we’ll need if such predictions are ever going to be reliable.</p>

<p>Finally, to tie this in with the whole cre/ID business, the seeming difficulty with which proteins fold has at times been used to argue that new folds could never evolve, or that protein folds could never have arisen <em>de novo</em>.  These claims are similar to the type that William Dembski has made about <a href="http://www.pandasthumb.org/pt-archives/000764.html">protein function</a>, which don’t stand up well to scrutiny.  But such arguments are always easier to sell when applied to things we know little about, like the protein folding problem, since one can always appeal to “high information” content and whatnot to say that protein folds could never evolve.  Unless, of course, it turns out that they don’t require high information content.</p>

<p><strong>References</strong>:</p>

<p>Socolich M, Lockless SW, Russ WP, Lee H, Gardner KH, Ranganathan R.  “Evolutionary information for specifying a protein fold.”  Nature. 2005 Sep 22;437(7058):512-8.</p>

<p>Russ WP, Lowery DM, Mishra P, Yaffe MB, Ranganathan R.  “Natural-like function in artificial WW domains.”  Nature. 2005 Sep 22;437(7058):579-83.</p>

<p>See also the <em>Nature</em> News and Views commentary:</p>

<p>Jeffery W. Kelly. “<a href="http://www.nature.com/nature/journal/v437/n7058/full/437486a.html">Structural biology: Form and function instructions</a>”.  Nature 437, 486-487 (22 September 2005)</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2005/10/protein-folding.html';
this.page.identifier = '/archives/2005/10/protein-folding';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


