<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>It's just a stage. A phylotypic stage. Part III: Fish and more</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="It's just a stage. A phylotypic stage. Part III: Fish and more" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2010/12/its-just-a-stag-2.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Steve Matheson" />
  <meta property="article:published_time" content="2010-12-18T01:56:22-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2010/12/its-just-a-stag-2.html">It's just a stage. A phylotypic stage. Part III: Fish and more</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Steve Matheson</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2010-12-18T01:56:22-07:00">December 18, 2010 01:56 MST</time></span>
	  </div>
   </header>
  <p><a href="http://www.researchblogging.org/"><img src="http://www.researchblogging.org/public/citation_icons/rb2_large_gray.png" alt="ResearchBlogging.org" style="float:left;" /></a>Given that [disputes over the existence and meaning of the phylotypic stage](http://pandasthumb.org/archives/2010/12/its-just-a-stag.html) and the hourglass model have simmered in various forms for a century and a half, the remarkable correspondence between the hourglass model and gene expression divergence [discovered by Kalinka and Varga and colleagues](http://pandasthumb.org/archives/2010/12/its-just-a-stag-1.html) would be big news all by itself. But amazingly, that issue of _Nature_ included two distinct reports on the underpinnings of the phylotypic stage. The other article involved work in another venerable model system in genetics, the [zebrafish](http://en.wikipedia.org/wiki/Zebrafish).</p>

<p>The report is titled &quot;[A phylogenetically based transcriptome age index mirrors ontogenetic divergence patterns](http://www.nature.com/nature/journal/v468/n7325/abs/nature09632.html)&quot; and is co-authored by [Tomislav Domazet-Loso](http://www.irb.hr/en/home/tdomazet/) and [Diethard Tautz](http://www.evolbio.mpg.de/~tautz/). To understand how their work has shed light on the phylotypic stage and the evolution of development, we'll need to look first at an approach to the analysis of evolutionary genetics that these two scientists pioneered: phylostratigraphy.</p>

<p>The authors [first described phylostratigraphy in 2007](http://dx.doi.org/10.1016/j.tig.2007.08.014) and have since used the approach to examine genes that cause [human genetic disease](http://mbe.oxfordjournals.org/content/25/12/2699.long) and [cancer](http://www.biomedcentral.com/1741-7007/8/66). They define it as:</p>

<blockquote>
  <p>a statistical approach for reconstruction of macroevolutionary trends based on the principle of founder gene formation and punctuated emergence of protein families.</p>
</blockquote>

<p>The idea is that every gene has a birthday, a point at which it is first identifiable in evolutionary history. Some genes are ancient, having arisen before there were even complex cells, and others are relative juveniles, having arisen much more recently. Genes present today, then, can (in principle) be assigned an &quot;age.&quot; Domazet-Loso and Tautz represent the &quot;age&quot; of a gene by the evolutionary &quot;epoch&quot; in which it appeared, by analogy with the identification of the appearance of biological lineages with [stratigraphic epochs](http://evolution.berkeley.edu/evosite/history/biostratigraphy.shtml) in earth's history. So for example, some genes appear with the development of true animals (metazoa), and so these genes are assigned to that &quot;stratum&quot; of biological history. In fact, the authors call each epoch a 'phylostratum' to reinforce that metaphor.</p>

<p>So how does this work? To do phylostratigraphic analysis, you need two major sets of tools. First, you need a pretty solid [phylogeny](http://evolution.berkeley.edu/evosite/evo101/IIBPhylogenies.shtml), or family tree, of your organism(s) of interest. Second, you need complete or nearly-complete genome sequences of the organism of interest and of organisms that can represent the major branch points (or nodes) in the family tree. The procedure from there seems clear enough: using a well-known alignment program, you search through the family tree for each of the genes in your organism of interest, to see where it is first recognizable in the phylogeny. That point is the phylostratum from which that gene arises. With that data, you could look at the contributions of various phylostrata to various body parts or processes. Or conversely, you could look at the relative age of the sets of genes associated with those body parts or processes. Or you could look at the relative age of the sets of genes associated with different stages of development. And that's what Domazet-Loso and Tautz did in their _Nature_ paper on the hourglass model.</p>

<p>Specifically, the authors took their phylostratigraphic data and merged it with expression data at various stages of zebrafish development; they called the resulting parameter the transcriptome age index (TAI). Basically, they calculated a relative age of the genes that are turned on at each stage of development, corrected for the extent to which particular genes are being used at those stages. Then they mapped the TAI onto the timeline of zebrafish development. And this is what they saw.</p>

<p><img src="http://pandasthumb.org/archives/2010/12/18/PhylostratigraphicHourglassFig1a-700px.gif" alt="PhylostratigraphicHourglassFig1a-700px.gif" width="700" height="453" style="text-align: center; display: block; margin: 0 auto 20px;" class="mt-image-center" /></p>

<p>Does that look familiar? Like, say, half an hourglass? In the earliest stages of development, active genes are young-ish, as they are in the juvenile and the adult. In between, the genes that are active are older -- a lot older. And the low point, where genes are oldest? It's the end of segmentation and the beginning of the pharyngula stage. That's the stage that is considered the phylotypic stage in vertebrates. (What this has to do with [godless liberalism](http://scienceblogs.com/pharyngula/), I have no idea.) And so we see that hourglass again, this time traced out by the evolutionary age of the genes that are active during the phylotypic stage.</p>

<p>As you look at the graph, you might notice some other interesting periods in the life of a fish. There's a prominent peak of gene youthfulness at 6 hours of development; this corresponds to [gastrulation](http://www.ncbi.nlm.nih.gov/books/NBK9992/#A1688), that wonderful time in your life when you established yourself as a three-layered animal. That peak is due to the activation of a lot of animal-specific genes, namely those that date to the metazoan phylostratum. This includes genes that control cell-cell interactions, certainly a hallmark of animal-building. Those might seem like incredibly basic functions, but they're relatively young compared to even more basic cellular processes, and the genes that control those processes are the ones that predominate during the later phylotypic stage. (The authors showed, in fact, that extremely ancient genes are active uniformly throughout development, whereas the younger gene sets display the hourglass pattern: high-low-high.)</p>

<p>And notice that gene youthfulness declines during aging (after adulthood). Now why would that be? The authors propose that the most recent innovations (facilitated by relatively young genes) are likely to have resulted from adaptation, and so:</p>

<blockquote>
  <p>The fact that ageing animals revert to older transcriptomes is in line with the notion that animals beyond the reproductive age are not ‘visible’ to natural selection and can therefore not be subject to specific adaptations any more.</p>
</blockquote>

<p>There's a lot more: the study found differences between males and females (look at the dotted lines in the figure), for example. But they also extended their analysis to other animals with known genomes: fruit fly, [roundworm](http://en.wikipedia.org/wiki/Caenorhabditis_elegans) and [mosquito](http://en.wikipedia.org/wiki/Anopheles). In every case they saw the same pattern: young-old-young. Their fly graph displays a pattern strikingly similar to that in the fish, and nicely dovetails with the distinct analysis done by Pavel Tomancak's group:</p>

<p><img src="http://pandasthumb.org/archives/2010/12/18/PhylostratigraphicHourglassFig3aFly-700px.gif" alt="PhylostratigraphicHourglassFig3aFly-700px.gif" width="700" height="458" style="text-align: center; display: block; margin: 0 auto 20px;" class="mt-image-center" /></p>

<p>Look at the low point, where the genes are the oldest. It's the germband elongation stage -- the recognized phylotypic stage for insects, and the same point singled out in the fly paper. Remarkable.</p>

<p>So to summarize, the two papers, reported separately but simultaneously, strongly support the hourglass model of development, in which embryos are seen to converge on an evolutionarily-ancient form, after diverse beginnings and followed by radical divergence into the wonderful variety of animals seen today and in the past. Domazet-Loso and Tautz explain how these new results make sense of the hourglass:</p>

<blockquote>
  <p>These consistent overall patterns across phyla, as well as the detailed analysis within zebrafish, suggest that there is a link between evolutionary innovations and the emergence of novel genes. Adaptations are expected to occur primarily in response to altered ecological conditions. Juvenile and adults interact much more with ecological factors than embryos, which may even be a cause for fast postzygotic isolation. Similarly, the zygote may also react to environmental constraints, for example, via the amount of yolk provided in the egg. In contrast, mid-embryonic stages around the phylotypic phase are normally not in direct contact with the environment and are therefore less likely to be subject to ecological adaptations and evolutionary change.</p>
</blockquote>

<p>And as they note, [Darwin himself  made this connection](http://www.biomedcentral.com/1741-7007/7/42), reflecting on von Baer's earlier observations. Ideas, like genes, can have a long and productive history.</p>

<p>\[Cross-posted at [Quintessence of Dust](http://sfmatheson.blogspot.com/2010/12/its-just-stage-phylotypic-stage-part.html).\]</p>

<p>----------</p>

<p>Domazet-Lošo, T., &amp; Tautz, D. (2010). A phylogenetically based transcriptome age index mirrors ontogenetic divergence patterns. <em>Nature</em>, 468 (7325), 815-818. DOI: <a href="http://dx.doi.org/10.1038/nature09632">10.1038/nature09632</a></p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2010/12/its-just-a-stag-2.html';
this.page.identifier = '/archives/2010/12/its-just-a-stag-2';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


