<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>An evolutionary prediction: "Silent" mutations are not always silent</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="An evolutionary prediction: "Silent" mutations are not always silent" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2007/01/an-evolutionary-1.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="PvM" />
  <meta property="article:published_time" content="2007-01-02T17:56:05-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2007/01/an-evolutionary-1.html">An evolutionary prediction: "Silent" mutations are not always silent</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By PvM</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2007-01-02T17:56:05-07:00">January 2, 2007 17:56 MST</time></span>
	  </div>
   </header>
  <p>In <strong>1988</strong>, Purvis et al proposed an interesting hypothesis about silent mutations:</p>

<p>Purvis IJ, Bettany AJ, Santiago TC, Coggins JR, Duncan K, Eason R, Brown AJ. The efficiency of folding of some proteins is increased by controlled rates of translation in vivo. A hypothesis.  J Mol Biol. 1987 Jan 20;193(2):413-7.</p>

<blockquote>
  <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>We propose that the way in which some proteins fold is affected by the rates at which regions of their polypeptide chains are translated in vivo. Furthermore, we suggest that their gene sequences have evolved to control the rate of translational elongation such that the synthesis of defined portions of their polypeptide chains is separated temporally. We stress that many proteins are capable of folding efficiently into their native conformations without the help of differential translation rates. For these proteins the amino acid sequence does indeed contain all the information needed for the polypeptide chain to fold correctly (even in vitro, after denaturation). However, other proteins clearly do not fold efficiently into their native conformation in vitro. We argue that the efficiency of folding of these problematic proteins in vivo may be improved by controlled synthesis of the nascent polypeptide.
</code></pre></div>  </div>
</blockquote>

<p>Silent mutations are mutations which do not change the amino acid of the protein. Since the genetic code is degenerate, more than one triplet codes for the same amino acid, mutations in the genetic code can have no impact on the amino acid and thus the protein.</p>

<p>In other words, silent mutations should not change the function of the protein and are thus considered to be evolutionary ‘neutral’.</p>

<p>But as Wikipedia explains</p>

<blockquote>
  <p>However, many organisms are known to exhibit codon usage biases, suggesting that there is selection for the use of particular codons due to translational stability. Silent mutations may also affect splicing, or transcriptional control.</p>
</blockquote>

<p>In other words, mutations which appear to be neutral, since they still code for the same amino-acid and thus the same protein, are not always neutral.</p>

<p>The <a href="http://www.the-scientist.com/news/home/38329/">Scientist</a> reported on December 21, 2006 that <a href="http://www.nih.gov/news/pr/dec2006/nci-21.htm">researchers had found</a> that “Silent” mutations are not always silent” and that “Mutations leading to identical amino acid sequences can change protein folding and function”. While examples of silent mutations not being silent existed in ‘lower animals’, this was the first time that an example was found for mammals.</p>

<p>The study was published in Science Express</p>

<p>Kimchi-Sarfaty C, Mi Oh J, Kim I-W, Sauna ZE, Calcagno AM, Ambudkar SV, Gottesman MM. <a href="http://www.sciencemag.org/cgi/content/abstract/1135308v1?ck=nck">A “silent” polymorphism in the MDR1 gene changes substrate specificity</a>. Science Express, December 21, 2006.</p>

<p>Compare this with the Scientific Vacuity of ID: How does Intelligent Design explain, let alone predict, this? A ‘silent’ mutation which nevertheless has a significant effect on the efficiency of cancer treatments.</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2007/01/an-evolutionary-1.html';
this.page.identifier = '/archives/2007/01/an-evolutionary-1';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


