<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Ontogenetic depth</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Ontogenetic depth" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2004/03/ontogenetic-dep.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="PZ Myers" />
  <meta property="article:published_time" content="2004-03-26T13:45:37-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2004/03/ontogenetic-dep.html">Ontogenetic depth</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By PZ Myers</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2004-03-26T13:45:37-07:00">March 26, 2004 13:45 MST</time></span>
	  </div>
   </header>
  <p>One of the serious shortcomings of Intelligent Design is that it does nothing to provide any new or productive insights into the workings of biology. ID proponents seem to be at least vaguely aware of this failure, in that they do frequently claim to be thinking about working on a preliminary, tentative approach towards the beginnings of a potential research program (my paraphrase), but most of the effort has been directed towards political and legal enforcement of their ideas, rather than actually testing those ideas. One advantage of pursuing only legalisms is that they don't give scientists anything to grapple. Invariably, when ID proponents do dip their toes in the scientific waters, they end up getting eaten by the sharks that lurk there.</p>

<p>One example: Paul Nelson, of the Discovery Institute, has been peddling a peculiar idea he calls "ontogenetic depth" as a scientific concept that emerges from Intelligent Design. To his credit, he has been presenting this idea in legitimate science venues, at the Geological Society of America and Society for Developmental Biology meetings. Note that getting on the program at these meetings is not subject to peer-review, so it is not automatically a recognition of merit that this work has been presented publicly. It is a good sign that Nelson is willing to expose his work to criticism, though.</p>

<p>I'm going to give it some criticism here. "Ontogenetic depth" is a developmental idea, and I'm a developmental biologist. Today I also get to play  shark.</p>

<a class="anchor" id="more"></a>


<p>What is "Ontogenetic depth"? Here is the <a href="http://www.iscid.org/encyclopedia/Ontogenetic_Depth">definition</a>, as written by Nelson himself and published online, at ISCID:</p>
<blockquote>
Ontogenetic depth is a measure of the distance (in terms of cell division and differentiation) between a single-celled state and an adult animal (metazoan) capable of reproduction. All animals begin their existence as a single cell, the fertilized egg. From that cell, many other cells arise, and become specialized for particular functional roles&mdash;for instance, as sensory organs, skeletal parts, or reproductive structures (such as ovaries). The ontogenetic depth of any species measures the developmental distance between the starting point, the egg, and the stage at which organisms in the species can successfully reproduce themselves.
<br />
Developmental biology has mapped the ontogenetic depth of a handful of species, in the so-called "model systems" of the discipline, such as the nematode Caenorhabditis elegans or the fruit fly Drosophila melanogaster. To explain the historical origin of any animal species (and arguably the same is true for plants), one must give an account of how that species' respective ontogenetic network&mdash;i.e., its process of differentiation&mdash;was constructed.
</blockquote>

<p>Unfortunately, I don't have a lot to go on here. "Ontogenetic depth" has not been published in any peer-reviewed journal, I have not attended any of Nelson's presentations on the subject, there are no methods described that would allow me to measure this value in any organisms, nobody other than Nelson and his collaborator, Marcus Ross, do anything with the idea, and all I've got are a transcript of an <a href="http://www.iscid.org/paul-nelson-chat.php">online chat</a> and an <a href="http://www.iscid.org/nelsonchat.pdf">accompanying pdf summary</a>. It's enough, I suppose. It's clear that this is a poorly expressed and unusable idea based largely on misconceptions and faulty scholarship, with no actual research to back it up. I feel sympathy for Mr Ross, who has been suckered into an extraordinarily bad idea early in his academic career.<p>

<p>So what's wrong with "Ontogenetic depth"?</p>

<p><b>It's an unworkable idea.</b> Nelson claims that ontogenetic depth is precisely measurable in extant species, and is therefore a good metric ("The ontogenetic depth of a handful of extant animals (from the model  systems of developmental biology) is known with precision"). Unfortunately, this is simply not true. The sole example he gives is <i>Caenorhabditis elegans</i>, which has an exceptionally well understood pattern of stereotyped cell lineages that lead to the adult. This is not the case for other metazoans, and it's not simply because they have cells too numerous to count; it's also because most other animals don't have an invariant cell number. Even in <i>C. elegans</i>, Nelson has to say that the ontogenetic depth is somewhere between 7 and 9...so much for specificity (and so much for measuring developmental differentiation&mdash;the mysterious unitless number doesn't seem to describe the elegant precision of the nematode at all well).</p>

<p><b>It uses false assertions and confusing examples.</b> The basis of Nelson's claims about the relevance of ontogenetic depth to evolution are simply wrong. He tries to argue that changes to the early lineage of organisms are "wreckers", that is that they are never viable, and therefore that patterns of early division can never change. He says,</p>

<blockquote>When  I was writing my dissertation, I asked one of my advisors if he knew  of any cases of the heritable modication of early development . He  couldn't think of a single example (other than a change from left-to-right  shell coiling, or back again, in snails&mdash;the exception that proves  the rule).</blockquote>

<p>This is confusing in several ways. Nelson spends a fair amount of time discussing the highly ordered cleavages of <i>C. elegans</i>. Is he, and his advisor, really unaware of the many known mutations in that organism that change the lineages? It's a whole grand class of numerous mutations named <i>lin</i>; there are mutations that block subsets of divisions, add extra divisions, transform the pattern of one set of divisions to resemble another lineage, or change the timing of divisions.</p>

<p>And why is he even mentioning the direction of spiral cleavages in molluscs here? That's a change in the chirality of early divisions, not any change in the pattern of divisions themselves. I don't see that the orientation of divisions is even considered in his measurement of ontogenetic depth.</p>

<p>When he claims that changes in early divisions are invariably deleterious, I know that that is false from personal experience. The zebrafish embryos I work with have a canonical pattern of early divisions, but the animals can tolerate a significant amount of variation; heat shock or just spontaneous screw-ups can produce wacky cleavages, but the animals regulate and recover. Also look at what can be done with mammalian embryos: the inner cell mass can be dissociated, cells juggled around, and reassociated in part or in combination with other embryonic cells, about as thorough a scrambling of the pattern of early development as you can get, and the embryos can resume development normally.</p>

<p>It is also curious that Nelson discusses nematodes, cites Schnabel (1997), and manages to completely ignore that paper to claim that nematode lineages are absolutely fixed. Here is the abstract to that paper:</p>
<blockquote><i>C. elegans</i> is renowned for its invariant embryogenesis and functions as a major paradigm for a mode of development coupled to an invariant lineage. Recent work, however, suggests that the embryogenesis of the nematode is much more flexible than anticipated. The invariant premorphogenetic stage is formed from variable earlier stages through a sorting of cells. Cells do not act as individuals but already early in embryogenesis a regionalization of the embryo occurs. Cells are diversified by a binary specification of 'abstract' blastomere (regional) identities. The determination of tissues may thus be a very late event. It appears that <i>C. elegans</i>, although assigning cell fates in an invariant lineage pattern, uses the same strategies and mechanisms for embryogenesis as organisms with variable lineages.<br />
Schnabel R (1997) Why does a nematode have an invariant cell lineage? Semin Cell Dev Biol 8:341-349.</blockquote>
<p>It's as if he read the title, copied one of the figures, and didn't pay any attention at all to the conclusions of the paper, which contradict what he claims.</p>

<p><b>The idea is based on bad metaphors.</b> Oh, jeez, Nelson's "Marching Band" metaphor...where to start? He tries to explain development as something similar to what a marching band has to do to assemble formations on a playing field: individual bandmembers (cells) have to maneuver through a series of defined steps (development) to end up in a position that spells out something (the adult state).</p>

<p>This is <i><b>not</b></i> how development works.</p>

<p>Cells do not, in most cases, have a pre-programmed pattern of differentiation built into them, that they then follow rigidly to end up in a specific, fated state or position. Development is about flexibility, interaction, negotiation, and emergent properties. To state that a trumpet player who doesn't know his rules would lead to the college's name getting misspelled on the football field doesn't say a thing about whether changes in development are impossible. In many organisms, we can shoot the trumpet player, blindfold the clarinetists, tumble everyone around in a cement mixer, and add a gang of bagpipe-playing soccer hooligans, and the mess can still organize itself to spell out a message (probably not a <i>polite</i> message, but something coherent, at any rate). If Nelson wants to make a metaphor, it's going to have to be one that can account for the radical regulative reorganization that has been <i>observed</i> in real developmental biology.</p>

<p>The only way I can imagine Nelson would even propose such a bad model is that he is both unaware of the range of processes in development, and that he has been misled by exaggeration and mischaracterization of the processes in many of the most common model systems. One worrisome flaw in molecular developmental genetics research is that we rely on simple laboratory models&mdash;nematodes, fruit flies, zebrafish&mdash;that are not well representative. These animals all develop very rapidly and are relatively simple, which is convenient for research, but unfortunately means that they tend to be developmentally streamlined, with less redundancy and more inflexiblity than is good for us. Biologists also tend to <i>look</i> for changes with large-scale, robust effects, further biasing our knowledge of what kinds of changes are out there. Nelson has taken a sampling bias that we are aware of (and that he should be aware of, too; he cites Raff, who has discussed the problem in some detail in his book, <i>The Shape of Life</i>) and exaggerated it to a cartoonish degree. We know that our models bias our perception of development in a certain direction. Nelson has taken a bias and blown up the error into a natural law.</p>

<p><b>There is no research!</b> Ontogenetic depth is a sloppily-defined concept with no theoretical support for its validity and no apparent operational utility. What can a scientist do with it? Nelson has a plan:</p>

<blockquote>However, it should be possible, using modern analogues for fossil taxa&mdash;e.g., the extant monoplacophoran <i>Neopilina</i> for the extinct mollusc  <i>Scenella</i>&mdash;to obtain good estimates on the ontogenetic depth requirements  of many Cambrian forms. This is research we are now conducting. It is likely that reasonable  estimates of the ontogenetic networks, and depth, required to specify  such extinct organisms as Anomalocaris or Opabinia, will require  no less complexity than that of modern animals.</blockquote>

<p>To which I can only say, "what?"</p>

<p>He doesn't have a way to measure ontogenetic depth...at least, not any way that he has explained, and that can be justified.</p>

<p>There is no reason to think that this parameter even <i>describes</i> complexity better than, for instance, counting cell types at maturity, which is also a fuzzy and difficult job.</p>

<p>If you determine (in some way) that this modern mollusc is as complex as another modern mollusc, how can you then claim that that means that ancient molluscs were as complex as modern ones? Isn't that already your assumption when you claim that the modern form is an analogue of the extinct one?</p>

<p>We already <i>know</i> that the Cambrian fauna were complex. I don't quite see what we'd learn that would be new to have someone announce that they have invented a new metric (which doesn't seem very good) that shows that the Cambrian fauna were complex.</p>

<p>I don't think he is necessarily measuring complexity, so it's even worse. What do we learn from carrying out a convoluted process to determine that some animal in the Cambrian weighed 50 grams and had 10<sup>10</sup> cells, and some other animal in the present day weighs 50 grams and has 10<sup>10</sup> cells?</p>


<p>The actual goal of Nelson's 'research' is to conclude that developmental complexity is infrangible, and that if he shows Cambrian organisms to be complex, that it is therefore impossible for them to have evolved. One of his cheerleaders, <a href="http://www.idurc.org/itsm/itsm-091503.shtml">Tristan Abbey</a>, makes this explicit:</p>
<blockquote>But that can't be true! The earliest stages of development  are different among the various animal groups. They must have evolved  somehow. But how? If development can't be touched, how did  it change?
<br />
The simplest answer for IDists is that it didn't.</blockquote>


<p>Nelson's research program seems to be counting cell divisions in extant organisms, and leaping to the unsupported conclusion that evolution is impossible if he gets similar numbers in two different groups. That conclusion is what is unsupported and untested by his methods. No one other than a few uninformed Intelligent Design creationists are claiming that "development can't be touched".</p>

<p><b>Those anecdotes...</b> One last minor point. What is it with all these ID creationists and their little quotes from China? Wells does it, now Nelson does it:</p>
<blockquote>I had with me (on top of my conference files) an overhead transparency,  showing the complex regulatory sequence of an invertebrate developmental  gene.
And up walked the author of that very paper (a visiting American biologist),  from which I had borrowed the figure.<br />
Of course he spotted the diagram right away. "What do you plan  to do with that?" he asked me.<br />
"I thought I might use this in my talk," I said, "as a quick  illustration of the complexity of embryonic regulation."
The biologist smiled. Knowing that he was something of a critic of  neo-Darwinism, I asked him what historical process he thought had  assembled the complex regulatory sequence.<br />
His answer really surprised me. "I don't know," he said, "but  I do know that ordinary mutation and selection won't do it."<br />
He went on to say that he thought our (that is, the biological community's)  understanding of evolution lagged way behind its other knowledge.</blockquote>
<p>An unattributed quote from a mysterious biologist attending a Discovery Institute sponsored meeting does not impress me in the slightest. I would like to know what a non-ordinary mutation might be, and why anyone should be surprised that a modern evolutionist would suggest that there is more to evolution than selection.</p>

<p>Reading Nelson's proposed research was rather like reading a very poor preliminary exam proposal from an unpromising graduate student. It would be grounds for flunking the poor sap out of the program on the spot. It's certainly not publishable, nor does it even hint at the potential for being publishable. And it's probably the best piece of work to emerge from the Intelligent Design crowd yet, which should give you an idea of the low quality material they've got.</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2004/03/ontogenetic-dep.html';
this.page.identifier = '/archives/2004/03/ontogenetic-dep';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


