<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Full text of 'The Edge of Creationism'</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Full text of 'The Edge of Creationism'" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2007/10/full-text-of-th.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Nick Matzke" />
  <meta property="article:published_time" content="2007-10-30T00:01:42-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2007/10/full-text-of-th.html">Full text of 'The Edge of Creationism'</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Nick Matzke</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2007-10-30T00:01:42-07:00">October 30, 2007 00:01 MST</time></span>
	  </div>
   </header>
  <p>In the <a href="http://pandasthumb.org/archives/2007/10/behe-review-in-tree.html">post</a> about my review of Behe’s <em>The Edge of Evolution</em>, many complained that they couldn’t access the full text without a university subscription or paying a huge fee.  I have checked Elsevier’s policies on this.  Authors are not allowed to post the published PDF to their websites (you have to get that from Elsevier), but they can put up the unformatted, submitted preprint version of their articles, as long as they include the reference and DOI to the published version.  So here is the reference: Nicholas J. Matzke (2007). “The edge of creationism.” <em>Trends In Ecology and Evolution, <strong>In Press, Corrected Proof,</strong> Available online 24 October 2007.</em> <a href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6VJ1-4R003D5-1&amp;_user=4420&amp;_coverDate=10%2F24%2F2007&amp;_rdoc=4&amp;_fmt=full&amp;_orig=browse&amp;_srch=doc-info(%23toc%236081%239999%23999999999%2399999%23FLA%23display%23Articles)&amp;_cdi=6081&amp;_sort=d&amp;_docanchor=&amp;_ct=9&amp;_acct=C000059607&amp;_version=1&amp;_urlVersion=0&amp;_userid=4420&amp;md5=f4b2bb09a40abe1fce213b83416b73dd">ScienceDirect</a>,  <a href="http://dx.doi.org/10.1016/j.tree.2007.09.004">doi: 10.1016/j.tree.2007.09.004</a>.</p>

<p>…and the full text is below the fold.  Note that the unpublished version has a few minor differences from the published version.  For example, it has more <em>emphases</em> which were kind of my way of <em>jumping up and down</em> on the smoking ruins of Behe’s core arguments in <em>The Edge of Evolution</em>.</p>

<blockquote>
  <p><strong>The Edge of Creationism</strong></p>

  <p>Review of: Michael J. Behe (2007). <em>The Edge of Evolution: The Search for the Limits of Darwinism.</em> New York: Free Press. US$28.00, hbk (336 pages). ISBN-13: 978-0-7432-9620-5</p>

  <p>Reviewer: Nicholas J. Matzke</p>

  <p>Michael Behe is the leading advocate of “intelligent design” (ID), which has been on the ropes since the 2005 <em>Kitzmiller v. Dover</em> trial. There, Behe’s effort to show that ID is science and not creationism failed [1-3]. <em>The Edge of Evolution</em> is Behe’s rather scattered comeback attempt. The title refers to Behe’s thesis: that anything as complex as a three-protein complex is beyond the reach of random mutation aided by natural selection.</p>

  <p>Behe begins by trying to shore up his argument that “irreducibly complex” multiprotein systems, such as flagella, are unevolvable. He claims that the bacterial flagellum is now known to be even more complex than previously thought, trumpeting the extra complexity of four regulatory proteins—apparently unaware they are absent in various flagella [2]. Similarly, he claims that the unrelated eukaryotic flagellum (or cilium) represents “irreducible complexity squared” because intraflagellar transport (IFT) is required for assembly. But Behe somehow missed the fact, mentioned even in literature that he cites, that the malaria parasite <em>Plasmodium</em> assembles its flagellum without IFT. There is “[m]ore than one way to build a flagellum” [4].</p>

  <p>Ironically, Behe pays close attention to <em>Plasmodium</em> elsewhere, when he attempts to use P. falciparum’s evolution of chloroquine resistance (CQR) to establish that the origin of multiprotein complexes requires ID. Here is the flabbergasting line of argument. First, Behe admits that CQR evolves naturally, but contends that it requires a highly improbable simultaneous double mutation, occurring in only 1 in 10<sup>20</sup> parasites. Second, Behe asserts that 
protein-protein binding sites require several <em>simultaneous</em> point mutations and are therefore even less probable than the alleged double-mutant CQR. Behe’s last step is to square 10<sup>20</sup> to produce 10<sup>40</sup>, the number of organisms required to evolve <em>two</em> binding sites linking three proteins. Since fewer organisms than this are available in Earth’s history, any complex of three or more proteins is beyond the reach of mutations not guided by ID.</p>

  <p>The argument collapses at every step. Behe obtains the crucial 10<sup>20</sup> number from an offhand estimate in the literature that considered only the few CQR alleles that have been detected because they have taken over regional populations. What is needed, however, is an estimate of how often <em>any</em> weak-but-selectable CQR originates. A study conducted where CQR is actively evolving [5] shows that high-level CQR is actually <em>more</em> complex than two substitutions, but that it is preceded by CQR alleles having fewer substitutions, and Behe’s two mutations do not even always co-occur. As a result CQR is both more complex and vastly more probable than Behe thinks. This sinks Behe’s 1 in 10<sup>20</sup> estimate for CQR, as well as his notion that protein-protein binding sites are more complex and therefore less probable than CQR. Behe’s decision to <em>square</em> the probability for two binding sites depends on the assumption that two binding sites would have to evolve <em>at once</em>, but the assumption is false for the same reasons that Behe’s “irreducible complexity” argument failed in the first place [1-3]. The squaring assumption is further contradicted by any experiment that accidentally evolves two proteins binding to different sites on a target protein instead of just one (e.g. [6]).</p>

  <p>Behe buttresses his argument for the improbability of protein-protein binding sites with the fact that no new protein-protein binding sites arose during the evolution of CQR, but never explains why this contradicts evolutionary expectations. He apparently thinks that evolutionary theory says anything should evolve a new binding site in response to any arbitrary situation. Behe dismisses antibodies, where new sites easily evolve to bind almost anything, on the grounds that the immune system is designed, neatly inserting his conclusion into his premises, and ignoring once again the embarrasing mountain of evidence against him [1]. Microbial toxin evolution is waved aside with “it’s relatively easy to clog a system,” which ignores the fact that such proteins often have exquisitely specific binding. Snake venom shows that even vertebrates with small populations can evolve huge gene families that specifically bind diverse proteins, with massive evidence of duplication, mutation, and selection as the mechanisms, and with <em>intraspecific</em> variation in regulation, sequence, and specificity. Is Someone actively designing rattlesnake (<em>Sistrurus catenatus</em>) venom in the American Midwest [7] and fine-tuning the specificity of black mamba (<em>Dendroaspis polylepis</em>) toxins for subtypes of mammalian muscarinic acetylcholine receptors [8]?</p>

  <p>It is clear that Behe is driven not by a truly scientific investigation, but instead metaphysics. He is obsessed with “randomness,” which he incorrigibly associates with “Darwinism” and cosmic purposelessness. This is one of many incorrect but blindly-held assumptions common with creationists. But randomness in evolution is no more metaphysically significant than randomness in weather systems. If creationists realized this, we might finally see the edge of creationism, if not the end of it. But if Behe is any indication, that won’t be any time soon.</p>

  <p><strong>References</strong></p>

  <p><strong>1</strong> Bottaro, A. <em>et al.</em> (2006) <a href="http://www.google.com/search?q=Immunology+in+the+spotlight+at+the+Dover+%E2%80%98Intelligent+Design%E2%80%99+trial&amp;sourceid=navclient-ff&amp;ie=UTF-8&amp;rlz=1B3GGGL_enUS239US239">Immunology in the spotlight at the Dover ‘Intelligent Design’ trial</a>. <em>Nat. Immunol.</em> 7 (5), 433-435</p>

  <p><strong>2</strong> Pallen, M.J. and Matzke, N.J. (2006) <a href="http://scholar.google.com/scholar?hl=en&amp;lr=&amp;q=%22From+The+Origin+of+Species+to+the+origin+of+bacterial+flagella%22&amp;btnG=Search">From <em>The Origin of Species</em> to the origin of bacterial flagella</a>. <em>Nat. Rev. Immunol.</em> 4 (10), 784-790</p>

  <p><strong>3</strong> Scott, E.C. and Matzke, N.J. (2007) <a href="http://www.google.com/search?tab=sw&amp;sa=N&amp;hl=en&amp;lr=&amp;q=%22Biological+design+in+science+classrooms%22&amp;btnG=Search">Biological design in science classrooms</a>. <em>Proc. Natl. Acad. Sci. U. S. A.</em> 104 (suppl. 1), 8669-8676</p>

  <p><strong>4</strong> Briggs, L.J. <em>et al.</em> (2004) <a href="http://scholar.google.com/scholar?q=More+than+one+way+to+build+a+flagellum%3A+comparative+genomics+of+parasitic+protozoa&amp;hl=en&amp;lr=&amp;btnG=Search">More than one way to build a flagellum: comparative genomics of parasitic protozoa</a>. <em>Curr. Biol.</em> 14 (15), R611-612</p>

  <p><strong>5</strong> Mittra, P. <em>et al.</em> (2006) <a href="http://scholar.google.com/scholar?hl=en&amp;lr=&amp;q=%22Progressive+increase+in+point+mutations+associated+with+chloroquine+resistance+in+Plasmodium+falciparum+isolates+from+India%22&amp;btnG=Search">Progressive increase in point mutations associated with chloroquine resistance in <em>Plasmodium falciparum</em> isolates from India</a>. <em>J. Infect. Dis.</em> 193 (9), 1304-1312</p>

  <p><strong>6</strong> Petrenko, V.A. <em>et al.</em> (2002) <a href="http://scholar.google.com/scholar?hl=en&amp;lr=&amp;q=Alpha-helically+constrained+phage+display+library&amp;btnG=Search">Alpha-helically constrained phage display library</a>. <em>Protein Eng.</em> 15 (11), 943-950</p>

  <p><strong>7</strong> Sanz, L. <em>et al.</em> (2006) <a href="http://scholar.google.com/scholar?q=Venom+proteomes+of+closely+related+Sistrurus+rattlesnakes+with+divergent+diets&amp;hl=en&amp;lr=&amp;btnG=Search">Venom proteomes of closely related Sistrurus rattlesnakes with divergent diets</a>. <em>J. Proteome Res.</em> 5 (9), 2098-2112</p>

  <p><strong>8</strong> Fry, B.G. <em>et al.</em> (2003) <a href="http://scholar.google.com/scholar?hl=en&amp;lr=&amp;q=Molecular+evolution+and+phylogeny+of+elapid+snake+venom+three-finger+toxins&amp;btnG=Search">Molecular evolution and phylogeny of elapid snake venom three-finger toxins</a>. <em>J. Mol. Evol.</em> 57 (1), 110-129</p>
</blockquote>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2007/10/full-text-of-th.html';
this.page.identifier = '/archives/2007/10/full-text-of-th';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


