<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Accessible research: Natural selection reduced diversity on human Y chromosomes</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Accessible research: Natural selection reduced diversity on human Y chromosomes" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2014/01/accessible-rese-2.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="M. Wilson Sayres" />
  <meta property="article:published_time" content="2014-01-16T11:17:07-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2014/01/accessible-rese-2.html">Accessible research: Natural selection reduced diversity on human Y chromosomes</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By M. Wilson Sayres</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2014-01-16T11:17:07-07:00">January 16, 2014 11:17 MST</time></span>
	  </div>
   </header>
  <p>We just had a paper <a href="http://www.plosgenetics.org/doi/pgen.1004064">published over at PLoS Genetics</a> entitled, “Natural selection reduced diversity on human Y chromosomes.” But, as you may recall, it has been <a href="http://arxiv.org/abs/1303.5012">available on the arXiv</a> for quite some time.<br />
<br />
Others have already summarized the work (Razib Khan <a href="http://www.unz.com/gnxp/on-that-homogeneous-y/">nails the punchline</a>, and Ian Sample has a <a href="http://www.theguardian.com/science/2014/jan/09/y-chromosome-sex-shrinking">summary for the popular press</a>).<br />
<br />
Below are answers to many of the questions I received about this work.<br /></p>

<div>
  <div>
    <p><strong>How would you describe the importance of your findings?</strong></p>
  </div>

  <div>
    <table>
      <tbody>
        <tr>
          <td><a href="http://upload.wikimedia.org/wikipedia/commons/5/51/Cariotipo_del_paquetismo.gif"><img src="http://upload.wikimedia.org/wikipedia/commons/5/51/Cariotipo_del_paquetismo.gif" alt="" width="320" height="320" /></a></td>
        </tr>
        <tr>
          <td>Human chromosomes by Paquete</td>
        </tr>
      </tbody>
    </table>

    <p>The human sex chromosomes, X and Y, used to be nearly identical, but now the Y has lost 90% of the genes it once shared with the Y, and some have speculated that the Y chromosome will disappear in less than five million years. We show that human Y chromosomes are much more similar to each other than expected. Previously, variance in male reproductive success (meaning some men fathering many children, and some men fathering few or none), was thought to explain this similarity, but we show that an additional force, natural selection, is needed to reduce diversity across Y chromosomes to the levels we observe. </p>
  </div>

  <div>
    <p><br />
<strong>Could you briefly explain how exactly you showed that variation on the Y chromosome is consistent with natural selection? What does this mean?</strong><br />
<br />
We first ran statistical models that let us alter the number of males and females that contributed their genomes (the non-sex chromosomes, chromosome X, chromosome Y, and <a href="http://mathbionerd.blogspot.com/2013/08/y-and-mtdna-are-not-adam-and-eve-part-2.html">mtDNA</a>) to the next generation. We reduced the proportion of males passing on their genomes to the next generation in a series of experiments, but in doing so, the modeled estimates of variation in the genome did not match the observed levels of variation.<br />
<br /></p>

    <table>
      <tbody>
        <tr>
          <td><a href="http://1.bp.blogspot.com/-prxK37WcIls/UtSri4HU6LI/AAAAAAAABmc/tfbI2nD8NCo/s1600/journal.pgen.1004064.g001.png"><img src="http://1.bp.blogspot.com/-prxK37WcIls/UtSri4HU6LI/AAAAAAAABmc/tfbI2nD8NCo/s400/journal.pgen.1004064.g001.png" alt="" width="400" height="241" /></a></td>
        </tr>
        <tr>
          <td>Here none of the grey-scale bars match all of the green bars</td>
        </tr>
      </tbody>
    </table>

    <p><br />
We next developed statistical models to estimate the number of sites affected by purifying selection on a set of hypothetical Y chromosomes, then tested which models were most likely to have acted in the past, given what the variation we observe across the genome today. Purifying selection removes, or purifies out, harmful mutations. It can also affect linked neutral sites. However, everything on the Y chromosome is linked together, so selection acting anywhere on the Y-specific portion of the Y chromosome will remove variation across the entire chromosome.<br />
<br />
We found that, accounting for population-specific variation in male reproductive success (boxed results below), the number of sites predicted to be affected by purifying selection on the human Y chromosome fell in between the number of single-copy coding sites and the total number of sites in the ampliconic regions.<br /></p>

    <table>
      <tbody>
        <tr>
          <td><a href="http://1.bp.blogspot.com/-RaYByOj_EnU/UtSr-ESQ0SI/AAAAAAAABmk/kBnrQFgGYNc/s1600/journal.pgen.1004064.g002.png"><img src="http://1.bp.blogspot.com/-RaYByOj_EnU/UtSr-ESQ0SI/AAAAAAAABmk/kBnrQFgGYNc/s400/journal.pgen.1004064.g002.png" alt="" width="400" height="363" /></a></td>
        </tr>
        <tr>
          <td>Maximum likelihood estimates of chrY sites affected by purifying selection.</td>
        </tr>
      </tbody>
    </table>

    <p>This means that in addition to the single-copy coding genes on the Y chromosome, the highly repetitive, but still poorly understood, ampliconic regions are likely also affected by natural selection.<br />
<br />
Purifying selection - that removes harmful mutations - acting on many sites of the Y chromosome, resulted in a population of Y chromosomes as similar to one another as the the Y chromosomes we observed in the real human data.<br />
<br /></p>

    <div>
      <p><strong>Were you surprised to find what you uncovered?</strong></p>
    </div>

    <div>
      <p><br /></p>
    </div>

    <div>
      <p>We were surprised to see how much of the Y chromosome is affected by selection. Here we show that at least some of the the highly repetitive ampliconic regions (so called because they are amplified in copy number) are acted on by selection. These regions are necessary for sperm formation and function, but are poorly studied because of how difficult it is to sequence them. Our results suggest that we need more investment into studying the most challenging parts of our genome. </p>
    </div>

    <p><strong><br /></strong>
<strong>What can you say about the function of the 27 genes located on the Y chromosome. What do you know about the ampliconic regions?</strong><br />
<strong><br /></strong>
<br /></p>

    <table>
      <tbody>
        <tr>
          <td><a href="http://upload.wikimedia.org/wikipedia/commons/c/c9/Spermatozoa-human-3140x.jpg"><img src="http://upload.wikimedia.org/wikipedia/commons/c/c9/Spermatozoa-human-3140x.jpg" alt="" width="320" height="223" /></a></td>
        </tr>
        <tr>
          <td>Y-linked genes help sperm form and swim.</td>
        </tr>
      </tbody>
    </table>

    <p><br />
Although not part of this study, I can say that the <a href="http://www.nature.com/nature/journal/v423/n6942/full/nature01722.html">27 genes on the Y chromosome</a> can generally be divided into those that function in many tissues, and those that now primarily function in the testes. The ampliconic genes are only turned on in the testes. Both these groups appear to have major roles in sperm motility and function.<br />
<br />
<strong>What can your research tell us about our ancestor’s history. How does it change the picture?</strong><br />
<br />
When we think about the population of our common ancestors, what this tells us is that, although there has certainly been variation in male reproductive success, and that this varies across populations, the genes on the Y chromosome continue to be preserved because they serve an important function. It also tells us that estimates of the time to the most recent common ancestor of the Y chromosome might be underestimates if purifying selection has been a constant force, reducing diversity on the Y chromosome.<br />
<br /></p>

    <table>
      <tbody>
        <tr>
          <td><a href="http://upload.wikimedia.org/wikipedia/commons/4/4e/Genghis_Khan.jpg"><img src="http://upload.wikimedia.org/wikipedia/commons/4/4e/Genghis_Khan.jpg" alt="" width="215" height="320" /></a></td>
        </tr>
        <tr>
          <td>Portrait of Genghis Khan</td>
        </tr>
      </tbody>
    </table>

    <p>Certainly there is variation across human populations. In some populations, variation in male reproductive success will have a stronger impact. For example, I look forward to extending these results to see whether the Y chromosomes of Central Asian populations, with the genetic legacy of Genghis Khan, might show more of an effect of variance in male reproductive success.<br />
<br />
<strong>What makes the sex chromosome history so fascinating for you?</strong><br />
<br />
Everything! I am amazed at how the X and Y used to be nearly identical, just like any of our non-sex-chromosomes, and yet today the Y has lost 90% of the ancestral gene content, and both X and Y have accumulated sex-specific elements. I want to understand why these chromosome became so different in some lineages but not others, how this process is shaped by population history, what unique role these chromosomes play in health, and where we can expect them to go in the future. See other posts on sex chromosomes in <a href="http://mathbionerd.blogspot.com/2013/04/emu-large-bird-with-surprisingly-intact.html">emus</a>, <a href="http://mathbionerd.blogspot.com/2013/04/accessible-research-neo-sex-chromosomes.html">spiny rats</a>, and in the plant, <a href="http://mathbionerd.blogspot.com/2013/07/young-sex-chromosomes-are-very-active.html"><em>Silene latifolia</em></a>.<br />
<br />
<strong>Can you give me some examples of mammals that already lost their Y chromosome?</strong><br />
<br />
The Okinawa spiny rat. However, it is unfair to say it completely lost its Y chromosome. What we think happened is that a few of the genes on the Y chromosome (or at least the sex-determining gene) jumped to a non-sex chromosome pair. In some Okinawa spiny rats, it was recently discovered that the degraded Y chromosome <a href="http://www.ncbi.nlm.nih.gov/pubmed/22198613">fused to a non-sex chromosome pair</a>. This new location, although we haven’t identified it yet, will now evolve into the next sex chromosome pair.</p>
  </div>

  <div>
    <p><br /></p>
  </div>

  <div>
    <p><strong>In what ways do these results present a step forward in the debate on whether or not the Y chromosome is more or less dispensable?</strong></p>
  </div>

  <div>
    <p><br /></p>
  </div>

  <div>
    <p>Our findings suggest that natural selection is acting strongly on the human Y chromosome, and it is unlikely to disappear any time soon - you can call me on this in 5 million years if I’m wrong. </p>
  </div>

  <div>
    <p><br /></p>
  </div>
</div>

<div>
  <p><strong>Why is there still so little coverage of the Y chromosome in genome sequencing?</strong><br />
<br />
It isn’t so much that there is so little coverage of the Y chromosome, but that most projects do not assemble the reads they have from the Y chromosome, and there’s a good reason for this - most of it is a mess! With the current technology, the length of pieces of DNA (usually 100-200 nucleotides) to look at from genome sequencing are too short to get a good handle on many of the highly repetitive regions of the Y chromosome. With new advances in technology, or with a lot of time and diligence using more involved techniques (such as <a href="http://www.nature.com/ng/journal/v45/n9/full/ng.2705.html">single haplotype iterative mapping and sequencing, SHIMS</a>), we will learn more about these difficult to study regions of the Y chromosome.<br />
<br />
<strong>How do your findings fit into <a href="http://www.ncbi.nlm.nih.gov/pubmed/24263135">recent research in mice that only two genes on the Y chromosome are necessary to produce offspring</a>? Even though the Y might not disappear it seems to be superfluous soon.</strong><br /></p>

  <table>
    <tbody>
      <tr>
        <td><a href="http://en.wikipedia.org/wiki/File:MRI_head_side.jpg"><img src="http://upload.wikimedia.org/wikipedia/commons/3/36/MRI_head_side.jpg" alt="" /></a></td>
      </tr>
      <tr>
        <td>MRI of human head</td>
      </tr>
    </tbody>
  </table>

  <p>The mouse Y chromosome work showed that by using twoY-specific genes, SRY, and Eif2S3Y, mice would develop with male characteristics and would produce immature sperm (it isn’t clear if they could develop fully functional sperm). In this case, I can say that the trajectory of the primate, specifically the human, Y chromosome and the mouse Y chromosome are very different. The similarity with humans is that SRY is also one of the main regulators of pathways that turn on testes formation, but a big difference is that the second gene, Eif2S3Y, is broken in humans. Furthermore, the mouse Y chromosome is pretty wimpy, in gene content, compared to the human Y. The human Y has retained many more genes from the ancestral mammalian sex chromosome, and many of these have been shown to function in sperm formation and motility, so it seems like it will be much for difficult for fertile, sperm-producing humans to develop without a Y chromosome. Furthermore, while many of the human Y-linked genes are involved in male-specific functions, there are <a href="http://mathbionerd.blogspot.com/2014/01/notes-on-fresh-look-at-male-specific.html">several that are turned on in other tissues</a>, such as the brain, and liver, so it seems unlikely there are more genes that are involved in form and function on the human Y than the mouse Y. That said, there are humans who have a single X chromosome and no partner X or Y (this is called <a href="http://turnersyndrome.org/learn-about-ts/what-ts">Turner syndrome</a>), suggesting that fertilization is a resilient process.<br />
<br />
<strong>What do you think might be the most important aspect of your research? How can it be used in the future? What are your next steps?</strong><br />
<br />
One of the most important aspects will be understanding whether positive selection has also played a strong role in shaping human Y chromosome diversity. We did not have the power to detect a difference between purifying selection (which acts to remove harmful mutations) and positive selection (which acts to increase a beneficial mutation in frequency), but plan to look at this in the future as more Y chromosomes are sequenced.<br />
<br />
<strong>Is there anything you might want to add or stress concerning your studies?</strong></p>
</div>

<div>
  <p><br />
We find complementary roles between variance in male reproductive success, and selection, in shaping the pattern of variation on the human Y chromosomes. It will be fascinating to assess this in more human populations, to see how local environments have uniquely shaped the evolution of the Y chromosome.<br />
<br /></p>
</div>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2014/01/accessible-rese-2.html';
this.page.identifier = '/archives/2014/01/accessible-rese-2';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


