<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Icons of ID: Meyer and the case of the missing references</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Icons of ID: Meyer and the case of the missing references" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2004/10/icons-of-id-mey.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="PvM" />
  <meta property="article:published_time" content="2004-10-29T16:07:01-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2004/10/icons-of-id-mey.html">Icons of ID: Meyer and the case of the missing references</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By PvM</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2004-10-29T16:07:01-07:00">October 29, 2004 16:07 MST</time></span>
	  </div>
   </header>
  <p>In his 2004 paper “<a href="http://www.discovery.org/scripts/viewDB/index.php?command=view&amp;id=2177&amp;program=CSC%20-%20Scientific%20Research%20and%20Scholarship%20-%20Science">The Origin of Biological Information and the Higher Taxonomic Categories</a>” Meyer introduces the reader to the concept of Shannon information. Despite the fact that the paper is presented as an ‘extensive review paper’ which ‘ argues that no current materialistic theory of evolution can account for the origin of the information necessary to build novel animal forms’ Meyer forgets to add the relevant scientific research showing how variation and selection can increase Shannon information in the genome.</p>

<p>What I find surprising is the number of references omitted in this review paper directly relevant to the origin of biological information.</p>

<h2 id="the-case-of-the-missing-references">The case of the missing references</h2>

<p>Tom Schneider, <a href="http://www.lecb.ncifcrf.gov/~toms/paper/ev/">Evolution of Biological Information</a> Nucleic Acids Res 28:14, p. 2794-2799, 2000</p>

<blockquote>
  <p><strong>Abstract</strong>: How do genetic systems gain information by evolutionary processes? Answering this question precisely requires a robust, quantitative measure of information. Fortunately, fifty years ago Claude Shannon defined information as a decrease in the uncertainty of a receiver. For molecular systems, uncertainty is closely related to entropy and hence has clear connections to the Second Law of Thermodynamics. These aspects of information theory have allowed the development of a straightforward and practical method of measuring information in genetic control systems. Here this method is used to observe information gain in the binding sites for an artificial ‘protein’ in a computer simulation of evolution. The simulation begins with zero information and, as in naturally occurring genetic systems, the information measured in the fully evolved binding sites is close to that needed to locate the sites in the genome. The transition is rapid, demonstrating that information gain can occur by punctuated equilibrium.</p>

  <p>Tom Schneider, <a href="http://www.lecb.ncifcrf.gov/~toms/paper/ev/">Evolution of Biological Information</a> Nucleic Acids Res 28)14_, p. 2794-2799, 2000</p>
</blockquote>

<p>Schneider shows how under selection the <a href="http://www.lecb.ncifcrf.gov/~toms/paper/ev/latex/node6.html#fig.curves">information increases</a></p>

<p>That Meyer overlooked Schneider is surprising given the fact that Dembski commented on Schneider’s paper on <a href="http://www.metanexus.net/metanexus_online/show_article.asp?3294">Metanexus: America’s Obsession with Design</a> and in his book “No Free Lunch”.</p>

<p>Schneider’s work was extended and generalized by Kim et al in:</p>

<p>Kim, Martinetz, Polani <a href="http://www.inb.uni-luebeck.de/publications/pubfiles/kim/jtb.pdf">Bioinformatic principles underlying the information content of transcription factor binding sites</a> J Theor Biol. 2003 Feb 21;220(4):529-44.</p>

<blockquote>
  <p><strong>Abstract</strong>: Empirically, it has been observed in several cases that the information content of transcription factor binding site sequences (R(sequence)) approximately equals the information content of binding site positions (R(frequency)). A general framework for formal models of transcription factors and binding sites is developed to address this issue. Measures for information content in transcription factor binding sites are revisited and theoretic analyses are compared on this basis. These analyses do not lead to consistent results. A comparative review reveals that these inconsistent approaches do not include a transcription factor state space. Therefore, a state space for mathematically representing transcription factors with respect to their binding site recognition properties is introduced into the modelling framework. Analysis of the resulting comprehensive model shows that the structure of genome state space favours equality of R(sequence) and R(frequency) indeed, but the relation between the two information quantities also depends on the structure of the transcription factor state space. This might lead to significant deviations between R(sequence) and R(frequency). However, further investigation and biological arguments show that the effects of the structure of the transcription factor state space on the relation of R(sequence) and R(frequency) are strongly limited for systems which are autonomous in the sense that all DNA-binding proteins operating on the genome are encoded in the genome itself. This provides a theoretical explanation for the empirically observed equality.</p>

  <p>Kim, Martinetz, Polani <a href="http://www.inb.uni-luebeck.de/publications/pubfiles/kim/jtb.pdf">Bioinformatic principles underlying the information content of transcription factor binding sites</a> J Theor Biol. 2003 Feb 21;220(4):529-44.</p>
</blockquote>

<p>The Effects of the Transcription Factor on Binding Site Information Are Constrained by Genetic Autonomy Jan T. Kim, Thomas Martinetz, Daniel Polani <a href="http://www.inb.uni-luebeck.de/research/bsad/pdfs/rseq_poster_a4.pdf">Poster</a></p>

<p>Lenski, R. E., C. Ofria, R. T. Pennock, and C. Adami. 2003. <a href="http://myxo.css.msu.edu/lenski/pdf/2003,%20Nature,%20Lenski%20et%20al.pdf">The evolutionary origin of complex features</a>. Nature 423:139-144</p>

<blockquote>
  <p><strong>Abstract</strong>: A long-standing challenge to evolutionary theory has been whether it can explain the origin of complex organismal features. We examined this issue using digital organisms–computer programs that self-replicate, mutate, compete and evolve. Populations of digital organisms often evolved the ability to perform complex logic functions requiring the coordinated execution of many genomic instructions. Complex functions evolved by building on simpler functions that had evolved earlier, provided that these were also selectively favoured. However, no particular intermediate stage was essential for evolving complex functions. The first genotypes able to perform complex functions differed from their non-performing parents by only one or two mutations, but differed from the ancestor by many mutations that were also crucial to the new functions. In some cases, mutations that were deleterious when they appeared served as stepping-stones in the evolution of complex features. These findings show how complex functions can originate by random mutation and natural selection.</p>

  <p>Lenski, R. E., C. Ofria, R. T. Pennock, and C. Adami. 2003. <a href="http://myxo.css.msu.edu/lenski/pdf/2003,%20Nature,%20Lenski%20et%20al.pdf">The evolutionary origin of complex features</a>. Nature 423:139-144</p>
</blockquote>

<p>Adami and <a href="http://www.krl.caltech.edu/~adami/cas.html">Evolutionary biology and biocomplexity</a></p>

<p>Adami Evolutionary biology and biocomplexity <a href="http://www.krl.caltech.edu/~adami/1995-2000.html">1995-2000 papers</a></p>

<p>Especially <a href="http://xxx.lanl.gov/PS_cache/physics/pdf/0005/0005074.pdf">Evolution of biological complexity</a> by Adami, Ofria and Collier, Proc. Nat. Acad. Sci. USA 97 (2000) 4463-4468</p>

<blockquote>
  <p><strong>Abstract</strong>:In order to make a case for or against a trend in the evolution of complexity in biological evolution, complexity needs to be both rigorously defined and measurable. A recent information-theoretic (but intuitively evident) definition identifies genomic complexity with the amount of information a sequence stores about its environment. We investigate the evolution of genomic complexity in populations of digital organisms and monitor in detail the evolutionary transitions that increase complexity. We show that because natural selection forces genomes to behave as a natural “Maxwell Demon”, within a fixed environment genomic complexity is forced to increase.</p>

  <p><a href="http://xxx.lanl.gov/PS_cache/physics/pdf/0005/0005074.pdf">Evolution of biological complexity</a> by Adami, Ofria and Collier, Proc. Nat. Acad. Sci. USA 97 (2000) 4463-4468</p>
</blockquote>

<p>Adami et al show how the total entropy decreases while the fitness increases.</p>

<blockquote>
  <p><img src="http://www.pnas.org/content/vol97/issue9/images/small/pq0805620003.gif" alt="" style="float:left;" /><strong>Fig. 3.</strong> (A) Total entropy per program as a function of evolutionary time. (B) Fitness of the most abundant genotype as a function of time. Evolutionary transitions are identified with short periods in which the entropy drops sharply, and fitness jumps. Vertical dashed lines indicate the moments at which the genomes in Fig. 1 A and B were dominant. <a href="http://www.pnas.org/cgi/content-nw/full/97/9/4463/F3">Click to enlarge</a></p>
</blockquote>

<p>and what about the complexity?</p>

<blockquote>
  <p><img src="http://www.pnas.org/content/vol97/issue9/images/small/pq0805620004.gif" alt="" style="float:left;" /><strong>Fig. 4.</strong> Complexity as a function of time, calculated according to Eq. 4. Vertical dashed lines are as in Fig. 3. <a href="http://www.pnas.org/cgi/content-nw/full/97/9/4463/F4">Click to enlarge</a></p>
</blockquote>

<p>Creationists have raised a variety of objections against these experiments, arguing that specified complexity is injected by the fitness function, ignoring the logical conclusion that natural selection (the correlation between the genome and the environment) thus can fulfil the requirements of the explanatory filter in that it can generate complex specified information, leading to the inevitable conclusion that complex specified information, like irreducible complexity is not much of a reliable indicator of intelligent design.</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2004/10/icons-of-id-mey.html';
this.page.identifier = '/archives/2004/10/icons-of-id-mey';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


