<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Testing fundamental evolutionary hypotheses</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Testing fundamental evolutionary hypotheses" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2004/10/testing-fundame.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="PvM" />
  <meta property="article:published_time" content="2004-10-04T07:52:53-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2004/10/testing-fundame.html">Testing fundamental evolutionary hypotheses</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By PvM</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2004-10-04T07:52:53-07:00">October 4, 2004 07:52 MST</time></span>
	  </div>
   </header>
  <p><a href="http://awcmee.massey.ac.nz/people/dpenny/pdf/Penny_et_al_2003.pdf">Testing fundamental evolutionary hypotheses</a> by David Penny, Michael Hendy and Anthony Pool was published in <em>Journal of Theoretical Biology</em> volume 223, pages  377-385 in 2003.
Penny et al show that ‘Intelligent Design’ can be formulated as a testable hypothesis but this requires us to formulate motivation(s), means and/or opportunity to restrain the explanatory power of an ‘intelligent designer’. Additionally, they show why various potential ID hypotheses can be rejected based on the experimental evidence. Until ID proposes other hypotheses, common descent seems to remain the best hypothesis available. Since the Geoscience Research Institute (GRISDA) proposes <a href="http://www.grisda.org/origins/55034.pdf">an alternative theory of ID</a> (multiple independent origins) I will explore this hypothesis and show that again the data do not bode well for ID.</p>

<blockquote>
  <p>Sober and Steel (J. Theor. Biol. 218, 395-408) give important limits on the use of current models with sequence data for studying ancient aspects of evolution; but they go too far in suggesting that several fundamental aspects of evolutionary theory cannot be tested in a normal scientific manner. To the contrary, we show examples of how some alternatives to the theory of descent can be formulated in such a way that they lead to predictions that can be evaluated (and rejected). The critical factor is a logical formulation of the alternatives, even though not all possible alternatives can be tested simultaneously. Similarly, some of the limits using DNA sequence data can be overcome by other types of sequence derived characters. The uniqueness (or not) of the origin of life, though still difficult, is similarly amenable to the testing of alternative hypotheses.</p>

  <p><a href="http://awcmee.massey.ac.nz/people/dpenny/pdf/Penny_et_al_2003.pdf">Testing fundamental evolutionary hypotheses</a> by David Penny, Michael Hendy and Anthony Pool</p>
</blockquote>

<p>Penny et al discusses various hypotheses</p>

<p><strong>1. Viruses from space</strong></p>

<p>This is a test of Hoyle and Wickramasinghe’s claim that influenza viruses arrive via comets. Penny et al show that this claim has to be rejected.</p>

<p><strong>2. Intelligent design</strong></p>

<blockquote>
  <p>We can test the theory of descent versus a theory of individual creation of species–with each species being intelligently designed for its environment. Consider photosynthetic enzymes from plants living in a hot, dry desert (a cactus and a desert grass) with those from a moist-temperate grass. A wise creator might design similar photosynthetic enzymes for leaves functioning under hot dry conditions (the cactus and a desert grass).
This version of intelligent design would predict the following rooted tree for these enzymes: ((cactus, desert grass), temperate grass)–see Fig. 3A. This brings together enzymes from similar physical environments; under stress from high temperatures and strong water deﬁcits. In contrast, the theory of descent predicts that the grass enzymes would be more similar: (cactus, (desert grass, temperate grass))–see Fig. 3B.
This unites sequences sharing a more recent common ancestor, irrespective of their current physical environment. In practice, common ancestry gives the correct prediction for photosynthetic enzymes.
Many similar tests can be designed. The logic is identical for comparing protein sequences in the hairs of polar bears and snow rabbits with, say, those of a rabbit in a warm environment. Under intelligent design, the proteins in the two species living under Arctic conditions could be created to give maximum insulation under freezing conditions. Thus, hair proteins from species living in the Arctic would be similar for functional reasons. This test may not have been done, but the point is that the theory of descent leads to testable predictions. It is possible for Intelligent Design to fudge predictions to make them identical to the theory of descent, but this is unsatisfactory. It provides no mechanism that leads to the observed data, and it leads to a creator appearing to be the ‘‘Great Deceiver” who deliberately misleads rational humans.</p>

  <p><a href="http://awcmee.massey.ac.nz/people/dpenny/pdf/Penny_et_al_2003.pdf">Testing fundamental evolutionary hypotheses</a> by David Penny, Michael Hendy and Anthony Pool</p>
</blockquote>

<p>On <a href="http://www.grisda.org/origins/55034.pdf">GRISDA</a> we find the following comments on this paper:</p>

<blockquote>
  <p><strong>Comment</strong>. This paper illustrates the difficulty all of us have in responding to criticisms made by persons with presuppositions that clash with our own. I will discuss three problems with the arguments presented.
First, Penny et al. argue that the best method for tree construction is the method that gives the most congruent results from different data sets. However, this is true only if the species actually do share a common ancestor. If they have separate origins, the best method might be the one that shows the greatest conflict in the different trees. Conflict among evolutionary trees based on different data sets is so widespread and common that one may prefer the conclusion that common ancestry has been falsified.</p>

  <p><a href="http://www.grisda.org/origins/55034.pdf">GRISDA</a> comments</p>
</blockquote>

<p>That conflicts between different datasets are so widespread and common is a <a href="http://www.talkorigins.org/faqs/comdesc/phylo.html">common fallacy</a>, lacking as usual in any meaningful substance.</p>

<p>Theobald shows that</p>

<blockquote>
  <p>In real-life phylogenetic analyses, shared derived characters may be in conflict with other derived characters. Thus, objective methods are required for resolving this character conflict (Kitching et al. 1998, Ch. 1; Maddison and Maddison 1992, p. 49).</p>

  <p>…</p>

  <p>In the past 40 years, several algorithmic methods have been devised to resolve such instances of character conflict and to infer correct phylogenetic trees (Felsenstein 2004, Ch. 10). The following sections outline some of the most successful of these methods. Each method attempts to infer a phylogeny from existing data, and each has its respective strengths and weaknesses. Years of empirical testing and simulation have shown that, in general, these different algorithms, each with very different underlying assumptions, converge on trees that are highly similar when judged statistically (Li 1997, Chs 5 and 6; Nei and Kumar 2000, Chs 6, 7, and 8).</p>

  <p><a href="http://www.talkorigins.org/faqs/comdesc/phylo.html">29+ Evidences for Macroevolution</a> by Douglas Theobald on Talkorigins</p>
</blockquote>

<blockquote>
  <p>Second, there is an inconsistency in one of their arguments. In discussing how to test for common ancestry, Penny et al. make the statement that “a minimal-length Steiner tree can be calculated for any data. . .”. This statement is followed later in the same paragraph by a defense of their ability to test the theory of descent for mammals
because it “allows a comparison against a null alternative (that there was no treelike information in the data).” A null hypothesis (there is no treelike information in the data) that must be rejected in every case (because a tree can be constructed for every data set) can hardly serve as a test of a hypothesis.</p>

  <p><a href="http://www.grisda.org/origins/55034.pdf">GRISDA</a> comments</p>
</blockquote>

<p>This assertion about Steiner trees misses the point that Penny et al are making totally. What Penny et al argue is that the hypothesis of a rooted Steiner tree can be compared with a hypothesis of no rooted tree to see which one gives the best match. Just like a correlation coefficient can be calculated for any dataset larger than one sample point, this does not mean that correlation coefficients cannot serve to determine the best match given a particular hypothesis.
Darwin’s theory provides for a theoretical foundation why one would expect a rooted Steiner tree, making the observation that a rooted Steiner tree is a better hypothesis than no tree, one in support of Darwinian theory. If ID proposes an alternative structure then let them present the hypotheses and analysis to support their claims.</p>

<p>Penny et al explain this in their paper</p>

<blockquote>
  <p>Our prediction from the theory of descent was that orthologous genes in mammals should lead to similar trees–they are expected to share the same evolutionary history. We found minimal-length trees from five protein datasets, and showed that the trees were much more similar than expected by chance.</p>

  <p><a href="http://awcmee.massey.ac.nz/people/dpenny/pdf/Penny_et_al_2003.pdf">Testing fundamental evolutionary hypotheses</a> by David Penny, Michael Hendy and Anthony Pool</p>
</blockquote>

<p>Nothing contradictory. If there were no rooted tree structure one would expect the rooted Steiner trees from various genes to be no  more similar than expected from chance.</p>

<blockquote>
  <p>Third, the alternative hypotheses for which tests are proposed seem more like straw men than real competing hypotheses. One alternative hypothesis is that influenza viruses have repeatedly come from outer space, rather than descending from a common ancestor. The other alternative hypothesis was that every species was created individually, optimally designed for its present environment. Neither of these hypotheses is taken seriously by those who are skeptical of common ancestry.</p>

  <p><a href="http://www.grisda.org/origins/55034.pdf">GRISDA</a> comments</p>
</blockquote>

<p>The first hypothesis addresses claims of panspermia by Hoyle and Wickramasinghe, the second hypothesis shows how we need to define means. motives, opportunities to actually constrain ID. Until ID proposes testable hypotheses, one cannot blame scientists to formulate plausible hypotheses of ID. The hypothesis that species were designed optimally for its present environment seems hardly that illogical.</p>

<blockquote>
  <p>What would be more interesting would be a test of a hypothesis that there exist multiple independent lineages, each of which has diversified into numerous species. A hypothesis of this type seems to fit the data better than any competitor, notably including the hypothesis of a single common ancestor.</p>

  <p><a href="http://www.grisda.org/origins/55034.pdf">GRISDA</a> comments</p>
</blockquote>

<p>As is often typical with ID ‘hypotheses’ it fails to be supported by any scientific analysis beyond the claim ‘a hypothesis of this kind seems to fit the data better than any competitor’. Conveniently this ignores the plausibility of common descent from <a href="http://www.icb.ufmg.br/~lbem/aulas/grad/tge/evol/pdf/penny-genevol-99.pdf">multiple ancestors</a>, such as Woese proposes. Darwin  already speculated about this possibility and thus parsimony would reject the requirement of ‘a designer’ to explain these data. While the ID strawman of the Cambrian Period may give one the impression that multiple independent lineages arose and diversified, the actual data do not really suggest support for this creationist strawman.</p>

<p>Ironically, Penny et al do address the hypothesis of multiple common ancestors in their paper.</p>

<hr />

<h2 id="useful-references">Useful references</h2>

<ul>
  <li><a href="http://awcmee.massey.ac.nz/people/dpenny/">David Penny</a> homepage</li>
</ul>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2004/10/testing-fundame.html';
this.page.identifier = '/archives/2004/10/testing-fundame';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


