<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Haldane's non-dilemma</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Haldane's non-dilemma" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2007/07/haldanes-nondil.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Ian Musgrave" />
  <meta property="article:published_time" content="2007-07-01T10:56:36-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2007/07/haldanes-nondil.html">Haldane's non-dilemma</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Ian Musgrave</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2007-07-01T10:56:36-07:00">July 1, 2007 10:56 MST</time></span>
	  </div>
   </header>
  <p>Quick, before I start the post proper, guess how many beneficial mutations separate us from the last common ancestor of humans and chimpanzees. Write your guess on a bit of paper, then read on.</p>

<p>Over at Uncommon Descent, Dave Scott opines</p>

<blockquote>
  <p>“Coyne and his chance worshipping peers are stuck between a rock and a hard place. The rock is gradualism and the hard place is Haldane’s  Dilemma (<a href="http://en.wikipedia.org/wiki/Haldane&apos;s_dilemma">http://en.wikipedia.org/wiki/Haldane’s_dilemma</a>) . As gradualism gets more gradual Haldane’s Dilemma gets more difficult to overcome – there’s a limit to the number of mutations that can become fixed. As gradualism gets less gradual then the improbability of simultaneous beneficial mutations becomes more difficult to overcome. A truly classic example of being stuck between a rock and a hard place!”</p>
</blockquote>

<p>The “simultaneous beneficial mutations” argument is a relatively new (or at least rejigged) argument that is dealt with <a href="http://scienceblogs.com/goodmath/2007/05/behes_dreadful_new_book_a_revi_1.php">elsewhere</a> (see also <a href="/archives/2004/10/theory-is-as-th.html">here</a>). However, <a href="http://www.gate.net/~rwms/haldane.html">Haldane’s dilemma</a> has been a favoured argument in anti-evolution circles for a long time. Unfortunately for the anti-evolutionists, Haldane’s dilemma has never been a barrier to evolution, despite their misrepresentations. Recent work from the Human, Chimpanzee and Macaque genome projects underlines the fact that Haldane’s dilemma does not prevent evolution, and it is worthwhile revisiting one of the core anti-evolution arguments relating to it in the light of these results.</p>

<p>Firstly, some background to the issue. In 1957, the evolutionary biologist JSB Haldane published a paper that calculated, based on a series of assumptions, that on average it took about 300 generations for a beneficial <a href="http://en.wikipedia.org/wiki/Allele">allele</a> to go from initial appearance to being present in all members of a population (the allele is “<a href="http://en.wikipedia.org/wiki/Fixation_(population_genetics)">fixed</a>” in the population). This figure was pretty well constant over a range of selection intensities [1]. Anti-evolutionist Walter ReMine has latched onto this paper, claiming that it presents severe problems for evolution. His key claim is that Haldane’s dilemma makes it impossible to fix more than 1,667 beneficial mutations since the last common ancestor of humans and chimps (ReMine, “<a href="http://www.amazon.co.uk/Biotic-Message-Evolution-Versus-Theory/dp/0963799908/ref=sr_1_1/202-6306760-1314231?ie=UTF8&amp;s=books&amp;qid=1183279403&amp;sr=8-1">The Biotic Message</a>”, page 217). ReMine claims that 1,667 beneficial mutations are too few to make a poet-philosopher from an ape, therefore Haldane’s dilemma shows evolution cannot account for humans. The recent genome results directly address this argument, but before I tackle this, I’d like to cover a few misrepresentations.</p>

<p><strong>The misrepresentation starts</strong> in ReMines presentation of Haldanes paper.</p>

<blockquote>
  <p>“In the 1950’s the evolutionary geneticist JBS Haldane, calculated the maximum rate of genetic change due to differential survival. He <em>reluctantly concluded</em> that there is a serious problem here, now known as Haldane’s Dilemma.” ReMine, pg 208, first para. Emphasis added.</p>
</blockquote>

<p>Contrast this with what Haldane actually wrote (this is the entire summary from the paper).</p>

<blockquote>
  <p>“Unless selection is very intense the number of deaths needed to secure the substitution by natural selection, of one gene for another at a locus, is independent of the intensity of selection. It is often about 30 times the number of organisms in a generation. <em>It is suggested that in horoletic evolution, the mean time taken for each gene substitution is about 300 generations. This accords with the observed slowness of evolution</em>” (page 524 Haldane JBS. (1957). <a href="http://www.blackwellpublishing.com/ridley/classictexts/haldane2.pdf">The cost of natural selection.</a> J Genet, 55, 511-524) Emphasis added.</p>
</blockquote>

<p>Haldane several times points out his calculations accord with observed rates of evolution. In the entire paper (nor in his later 1961 paper), there is <strong>NO</strong> mention of any serious problem.</p>

<p>ReMine further misrepresents Haldane and the significance of his work:</p>

<blockquote>
  <p>“His calculations show that many higher vertebrate species could not plausibly evolve in the available time” (ReMine op cit).</p>
</blockquote>

<p>In fact, Haldane gives two examples where the evolutionary rates accord with his calculations (average rate of speciation in the carnivora, and mammalia on page 522, his conclusion: “the agreement with the theory developed here is satisfactory”). Haldane also gave examples where evolution could fix substitutions faster than under his assumptions (see page 523, where he discusses radiation of species into environments with few or no competitors, and the introduction, where he discusses intense selection). Haldane also explicitly acknowledged that these were preliminary approaches to developing a mathematical treatment of selection. In 1961 produced a paper where he revised his approach, and found at least one more circumstance where evolution could proceed faster than with his original assumptions.</p>

<p><strong>What the real problem is:</strong> One of the consequences of Haldane’s calculation is that it sets an upper limit to the amount of allelic variation (heterozygosity) in the genome. Under Haldanes’s assumptions, if different alleles of genes represent deleterious variants being selected against, too much variation means that the organisms fitness fall below survivable levels. When the variation in the genomes of several organisms was measured, it was way <a href="http://darwin.eeb.uconn.edu/eeb348/lecture-notes/molevol-neutral/node2.html">above the limits that would be survivable</a> if Haldane’s assumptions held. The problem is not that evolution is too slow; the problem is that it is <em>much faster</em> than Haldane’s limit.</p>

<p>Lets restate that, the amount of <em>measured</em> variation in the genome meant that if Haldane’s assumptions were right, all vertebrates would be dead. So we know that Haldane was wrong. Exactly where he was wrong occupied many pages of journal articles in the 60’s and 70’s. <a href="http://en.wikipedia.org/wiki/Neutral_theory_of_molecular_evolution">Kimura</a> (Kimura, 1968) used the heterozygosity problem to advance the <a href="http://darwin.eeb.uconn.edu/eeb348/lecture-notes/molevol-neutral/molevol-neutral.html">neutral theory</a>. In <a href="http://en.wikipedia.org/wiki/Neutral_theory_of_molecular_evolution">neutral theory</a>, most mutations are neutral with respect to fitness, and neutral alleles are fixed by <a href="http://en.wikipedia.org/wiki/Genetic_drift">drift</a>. Since the alleles have no effect on fitness, a very large number of allelic variants can be in the population and not reduce its fitness, thus solving the heterozygosity problem.</p>

<p>Several others proposed selectionist explanations using different assumptions to Haldane’s that could drive more substitutions. The technical details need not concern us here, suffice it to say there were a number of models which could exceed Haldane’s “speed limit” (<a href="http://www.gate.net/~rwms/haldane2.html">soft selection</a>, truncation selection and gene hitchhiking for example. All of which have some experimental and observation evidence, see <a href="http://www.gate.net/~rwms/ewens.html">Ewens, 1969</a>, Grant and Flake 1974, Smith, 1968 and many others in the reference list). The discussions over Haldane’s dilemma rapidly got subsumed into the larger <a href="http://en.wikipedia.org/wiki/Neutral_theory_of_molecular_evolution#The_.22neutralist-selectionist.22_debate">neutralist <em>vs</em> adaptionist debate</a>. In the end, the evidence came down on the side of the neutralists, and it is accepted that the majority of variation in genomes is due to neutral mutations [2].</p>

<p><strong>How many benefical mutations?</strong> While the majority of variation is neutral, the question remains <em>exactly</em> how much variation is due to selection, and does it break Haldane’s “speed limit”. Recent comparisons of Human and Chimp genomes, using the Macaque as an out group, have given us a good idea of how many genes have been fixed since the last common ancestor of chimps and humans (<a href="http://www.pnas.org/cgi/content/abstract/0701705104v1">Bakewell, 2007</a>).</p>

<p><strong>154</strong></p>

<p>Actually, that’s 154 of 13,888 genes. Given that we have around 22,000 genes [3] in our genome (<a href="http://www.ensembl.org/Homo_sapiens/index.html">http://www.ensembl.org/Homo_sapiens/index.html</a>), then if the same percentage of beneficial mutations holds for the rest of the genome, no more than 238 fixed beneficial mutations is what separates us from the last common ancestor of chimps and humans.</p>

<p>You are probably sitting there astonished that we are around 240 genes away from our last common ancestor with the chimp and saying “this can’t be right”[4] (how much did the guess you wrote down differ from the real thing?).  However, this result agrees with previous estimates of the number of positively selected genes (Arbiza, 2006, Yu 2006). You can argue until the cows come home about whether you can get around Haldane’s assumptions using truncation selection, soft selection or whatever, the plain fact is that humans and the last common ancestor of humans and chimps are separated by far fewer fixed beneficial mutations than even Haldane’s limit allows.</p>

<p>Now, it’s likely that the above values is an underestimate, and the some weakly selected genes have been missed, but it is in accord with previous studies using smaller gene sets (Arbiza, 2006, Yu 2006). Even if you say we missed half of the genes that underwent selection (very unlikely), the number of beneficial genes fixed by natural selection would be around 480, and the real number is certainly less (Arbiza, 2006).</p>

<p>The above study only covered protein coding genes, not regulatory sequences, and most biologists expect that changes in regulatory sequences played an important role in evolution. Getting at the number of beneficial mutations in regulatory genes that have been fixed by natural selection is a lot harder, but it seems like around 100 regulatory genes may have been selected (Donaldson &amp; Gottgens 2006, Kehrer-Sawatzki &amp; Cooper 2007). Again, even if we set the number of regulatory genes that have been selected as the same number as the most wildly optimistic estimate of protein coding genes fixed by natural selection, then we end up with 960 fixed beneficial mutations, below ReMine’s calculation of Haldane’s limit [5]. This means Haldane’s dilemma is irrelevant to human evolution.</p>

<p><strong>Conclusion:</strong> Haldane’s dilemma has never been a problem for evolution, but the technical nature of the arguments involved made it difficult to clearly demonstrate anti-evolutionists misuse of the “dilemma”. Also, the difficulty in getting the original papers meant that the distortion of Haldane’s work by anti-evolutionists was not obvious.</p>

<p>Now Walter ReMine’s claim that 1667 beneficial mutations is too few to generate a philosopher poet from the common ancestor of chimps and humans is shown to be trivially false from comparison of the human and chimp gemone. As this claim was the keystone of ReMine’s argument, Haldane’s dilemma should disappear as an anti-evolutionist claim.</p>

<p><strong>Notes:</strong>
[1] The actual “dilemma” of Haldane’s Dilemma, is that, under a number of limiting assumptions at modest selection intensities, you cannot speed up the rate of substitutions by simultaneously selecting multiple beneficial mutations. If you increase the number of mutations you select, you have to decrease selection intensity to stop the population going extinct. Haldane himself never used the term “Dilemma”, and it isn’t used all that often in the technical literature. For a fuller discussion of Haldane’s calculations see <a href="http://www.gate.net/~rwms/hald_cost_nontech.html">Robert William’s explanation</a>.</p>

<p>[2] Well, technically, the <a href="http://en.wikipedia.org/wiki/Nearly_neutral_theory_of_molecular_evolution">Nearly Neutral</a> model won. Also, biology being what it is, in some organisms (like the fruit fly <em>Drosophilia</em>) there is a slight excess of benefical <em>vs</em> neutral mutations. But generally, neutral or nearly neutral mutations rule.</p>

<p>[3] In the light of the human genome project, it is amusing to consider this paragraph from Remine, in his 1993 book (page 249). “The evolutionary scenario, as presently told, requires that the expressed portion of the genome must be less than one part in 164. That is only 0.6%, since the typical gene is 1000 nucleotides, that could encode about 22,000 genes. That is not enough to encode all the things that make humans.” Current evidence is that around 1.2% of the genome codes for protein, about the same amount for structural RNA and another 5% for regulatory sequences.</p>

<p>[4] While we are around 240 genes away from the LCA, we are around 594 genes way from the chimp, they have fixed about 50% more genes since the LCA than we have. Most of the genes substituted are for immune and reproductive system genes, and only a handful seem to have anything to do directly with brain function.</p>

<p>[5] ReMine calculated his substitution number with a chimp human split of 10 Million years, if we used the currently accepted 6 million years for the split, the figure is 1,000, still above the optimistic estimate for gene differences.</p>

<p>For a good (mostly) non-technical discussion of Haldane’s dilemma, from a slightly different perspective to the one I present here, see <a href="http://www.gate.net/~rwms/haldane.html">Robert William’s Haldane pages</a>. For a more technical paper with simulations showing that selection can exceed Haldane’s limit see <a href="http://www.sekj.org/PDF/anz40-free/anz40-185.pdf">Nunney 2003</a>.</p>

<p><strong>References</strong> relevant to Haldane’s dilemma, see particularly Flake and Grant.</p>

<ul>
  <li>Arbiza L, Dopazo J, Dopazo H. (2006) <a href="http://compbiol.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371/journal.pcbi.0020038">Positive selection, relaxation, and acceleration in the evolution of the human and chimp genome.</a> PLoS Comput Biol. Apr;2(4):e38.</li>
  <li>Bakewell MA, Shi P, Zhang J. <a href="http://www.pnas.org/cgi/content/abstract/0701705104v1">More genes underwent positive selection in chimpanzee</a> evolution than in human evolution. (2007) <a href="http://www.kiz.ac.cn/uploadfile/200751184025391.pdf">Proc Natl Acad Sci U S A. May 1;104(18):7489-94.</a></li>
  <li>Christiansen FB. (1990 Feb). Simplified models for viability selection at multiple loci. Theor Popul Biol , 37, 39-54.</li>
  <li>Donaldson IJ, Gottgens B. (2006) <a href="http://genomebiology.com/2006/7/6/R52">Evolution of candidate transcriptional regulatory motifs since the human-chimpanzee divergence.</a> Genome Biol. 2006;7(6):R52.</li>
  <li>Ewens WJ. (1969 Mar 15). <a href="http://www.nature.com/nature/journal/v221/n5185/abs/2211076a0.html">Mean fitness increases when fitnesses are additive</a>. Nature, 221, 1076.</li>
  <li>Felsenstein J, (1971) <a href="http://links.jstor.org/sici?sici=0003-0147(197101%2F02)105%3A941%3C1%3AOTBSOT%3E2.0.CO%3B2-L">On the biological significance of the cost of gene substitution.</a> The American Naturalist 105, 1-11.</li>
  <li>Flake RH, and Grant V. (1974 Sep). <a href="http://www.pnas.org/cgi/reprint/71/9/3716.pdf">An analysis of the cost-of-selection concept</a>. Proc Natl Acad Sci U S A , 71, 3716-20.</li>
  <li>Grant V, and Flake RH. (1974 Oct). <a href="http://www.pnas.org/cgi/reprint/71/10/3863.pdf">Solutions to the cost-of-selection dilemma</a>. Proc Natl Acad Sci U S A , 71, 3863-5.</li>
  <li>Haldane JBS. (1957). <a href="http://www.blackwellpublishing.com/ridley/classictexts/haldane2.pdf">The cost of natural selection.</a> J Genet, 55, 511-524</li>
  <li>Kehrer-Sawatzki H, Cooper DN (2007). Understanding the recent evolution of the human genome: insights from human-chimpanzee genome comparisons. Hum Mutat. Feb;28(2):99-130.</li>
  <li>Kimura M. (1968 Feb 17). Evolutionary rate at the molecular level. Nature, 217, 624-626.</li>
  <li>Mukai T, Schaffer HE, and Cockerham CC. (1972 Dec). Genetic consequences of truncation selection at the phenotypic level in Drosophila melanogaster. Genetics , 72, 763-9.</li>
  <li>Nunney, L (2003) “<a href="http://www.sekj.org/PDF/anz40-free/anz40-185.pdf">The cost of natural selection revisited</a>”, Ann. Zool. Fennici. 40:185-194.</li>
  <li>O’Donald P. (1969 Mar 1). “Haldane’s dilemma” and the rate of natural selection. Nature , 221, 815-7.</li>
  <li>ReMine WJ, (1993) <a href="http://www.amazon.co.uk/Biotic-Message-Evolution-Versus-Theory/dp/0963799908/ref=sr_1_1/202-6306760-1314231?ie=UTF8&amp;s=books&amp;qid=1183279403&amp;sr=8-1">The Biotic Message</a>, St Paul Science.</li>
  <li>Smith JM. (1968 Sep 14). “Haldane’s dilemma” and the rate of evolution. Nature , 219, 1114-6.</li>
  <li>Sved JA, Reed TE, and Bodmer WF. (1967 Mar). <a href="http://www.genetics.org/cgi/reprint/55/3/469">The number of balanced polymorphisms that can be maintained in a natural population</a>. Genetics, 55, 469-81.</li>
  <li>Taddei F, Radman M, Maynard-Smith J, Toupance B, Gouyon PH, and Godelle B. (1997 Jun 12). Role of mutator alleles in adaptive evolution. Nature , 387, 700-2.</li>
  <li>Yu XJ, Zheng HK, Wang J, Wang W, Su B.( 2006) <a href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6WG1-4KF7860-1&amp;_user=10&amp;_coverDate=12%2F31%2F2006&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_acct=C000050221&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=03ba8049d68dae59724e974564095e06">Detecting lineage-specific adaptive evolution of brain-expressed genes in human using rhesus macaque as outgroup</a>. Genomics. 88(6):745-51.</li>
</ul>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2007/07/haldanes-nondil.html';
this.page.identifier = '/archives/2007/07/haldanes-nondil';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


