<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Reality 1, Behe 0</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Reality 1, Behe 0" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2007/07/reality-1-behe.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Arthur Hunt" />
  <meta property="article:published_time" content="2007-07-22T21:17:00-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2007/07/reality-1-behe.html">Reality 1, Behe 0</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Arthur Hunt</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2007-07-22T21:17:00-07:00">July 22, 2007 21:17 MST</time></span>
	  </div>
   </header>
  <p>Or …</p>

<p><strong><em>T-urf13 redux</em></strong></p>

<p>A few months ago I posted <a href="http://www.pandasthumb.org/archives/2007/05/on_the_evolutio_1.html#more">an essay</a> about a remarkable example of the evolution of Irreducible Complexity from scratch, via natural, unguided mechanisms.  While the reaction to this essay has been pretty muted (precious little to take note of, save for one well-hidden reference on Uncommon Descent to “No Free Lunch”, citing pages that make arguments clearly refuted in the PT essay), I had no idea that a much bigger response, or target, would emerge from the Halls of ID.  This would, of course, be Mike Behe’s recently-released follow-up to “Darwin’s Black Box”, entitled “The Edge of Evolution”.</p>

<p>More follows beneath the fold, but here’s the short version for those of us with miniscule attention spans -<a href="http://www.pandasthumb.org/archives/2007/05/on_the_evolutio_1.html#more">the previous essay</a> refutes (or, better said, refuted) the EoE in no uncertain terms, showing that the centerpiece of the EoE, the value Behe assigns to the “probability” of occurrence of a protein interaction or binding site, does not agree with what we can and do know about the history of a bonafide multiple simultaneous mutation.</p>

<p>To start off with, it helps to capsulize Behe’s book.  The theme of the book, indeed the very Edge of Evolution itself, is a number that Behe assigns to the probability that a protein interaction site can evolve.  This number is 1 in 10^20.  Behe “derives” this number from the following line of reasoning (the details have been floating around the blogosphere for a few weeks, so I’ll be brief).  First, in Chapter 3, Behe discusses at some length the evolution of resistance to chloroquine in the malaria parasite Plasmodium falciparum.  It has been reported that resistance to this drug involves the appearance of at least two amino acid changes.  Behe argues that these need to be essentially simultaneous in appearance, and coins a term (the Chloroquine Complexity Cluster) to stand for events that involve leaps of two simultaneous mutational changes.  Behe also cites a review by Nicholas White that provides an estimate of the frequency of occurrence of a “CCC” – 1 in 10^20 trials.  In Chapter 7, Behe then discusses the matter of the evolution of protein binding sites.  Behe reasons that the evolution of a new protein binding site will involve a modest number of amino acid changes, and that some of these changes must occur in concert; for this reason, he asserts that a single protein interaction domain is analogous in “complexity” to a CCC, and thus should be expected to occur about once in 10^20 trials.  Finally, he argues that networks of more than two proteins require at least two different protein binding sites, and that selection will not act unless these changes occur simultaneously.  Thus, for a trimeric complex, two sites are needed, and would be expected to occur once in 10^40 trials.  In light of some general numbers (age of the universe, numbers of organisms on earth, etc.), this number would seem to lie on the other (unaccessible) side of the Edge of Evolution.  This is Behe’s argument in a nutshell.</p>

<p>Let’s turn now to the <a href="http://www.pandasthumb.org/archives/2007/05/on_the_evolutio_1.html#more">previous PT essay</a>.  In it, we learned that the protein dubbed T-urf13 had evolved, in one fell swoop by random shuffling of the maize mitochondrial genome.  We also learned some fascinating things about T-urf13.  Summarizing the salient points:</p>

<p>T-urf13 forms heteromeric complexes in the membrane.  This means that different subunits bind at least two other subunits.  This means that each subunit must have at least two different protein binding sites.  That’s at least two “CCC’s”.</p>

<p>T-urf doesn’t just form heteromers in membranes.  It forms a gated ion channel.  This means that each heteromeric complex must be capable of adopting at least two different conformations, of changing from one to another, and of shepherding ions through the phospholipid bilayer.  This all means that the heteromers are not some non-descript clumps of polypeptide; they possess all the functionality of other, more well-known gated ion channels.</p>

<p>One more thing – the ion channel is gated.  It binds a polyketide toxin, and the consequence is an opening of the channel.  This is a third binding site.  This is not another protein binding site, and I rather suppose that Behe would argue that this isn’t relevant to the Edge of Evolution.  But the notion of a “CCC” derives from consideration of changes in a transporter (PfCRT) that alter the interaction with chloroquine; toxin binding by T-urf13 is quite analogous to the interaction between PfCRT and chloroquine.  Thus, this third function of T-urf13 is akin to yet another “CCC”.</p>

<p>The bottom line – T-urf13 consists of at least three “CCCs”.  Running some numbers, we can guesstimate that T-urf13 would need about 10^60 events of some sort in order to occur.  Recalling Behe’s summation from p. 146:</p>

<blockquote>
  <p>“So let’s accept my earlier conservative estimation, and spell out some implications. The immediate, most important implication is that complexes with more than two different binding sites—ones that require three or more different kinds of proteins—are beyond the edge of evolution, past what is biologically reasonable to expect Darwinian evolution to have accomplished in all of life in all of the billion-year history of the world. The reasoning is straightforward. The odds of getting two independent things right are the multiple of the odds of getting each right by itself. So, other things being equal, the likelihood of developing two binding sites in a protein complex would be the square of the probability for getting one: a double CCC, 10^20 times 10^20, which is 10^40. There have likely been fewer than 10^40 cells in the world in the past four billion years, so the odds are against a single event of this variety in the history of life. It is biologically unreasonable.”</p>
</blockquote>

<p>That makes it pretty clear – T-urf13 is a mathematical impossibility.  It simply cannot exist.</p>

<p>Well, this is of course absurd (although the farmers whose livelihoods were devastated by Southern corn leaf blight dearly wish that Behe was correct).  Indeed, it is apparent that Behe’s “line in the sand” is badly out of touch with reality.  How different is it from what actually occurs? We can run some numbers of our own and compare.  The amount of arable land on the earth is on the order of 200 million hectares, or about 500 million acres (I’ll do some liberal rounding to keep things simple).  A really good farmer can cram 50,000 corn plants into an acre (we won’t go into what the yields would be…).  A typical corn tassel may make 50 million pollen grains (OK, a superman plant, maybe, but we’re being generous here).  Finally, maize has been under cultivation for less than 10,000 years.  What does this add up to?  If each and every arable acre on earth had been used for corn production, at the unseemingly high planting rate of 50,000 per acre, and if somewhow we could cram four plantings in per year, then this would translate to some 10^26 pollen grains in the entire history of corn production.  Of course, cmsT was probably developed over a 20 year period, and probably involved a few thousand acres and 2 or 3 generations per year. Let’s say 100,000 acres and 3 generations per year, to give us a more realistic (but still generous) estimate of 10^19 pollen grains.</p>

<p>Now, this isn’t the end of things.  The number of interest is the number of mitochondrial genome rearrangements – the number of shufflings that occurred to give us T-urf13.  Let’s stuff 1000 mitochondria into each pollen grain, and let each of them undergo a million recombinations (people who work with corn pollen may commence to groan).  This translates to about 10^28 events.</p>

<p>Now, recall that we are talking about, not one, but a minimum of three CCC’s.  Behe says 1 in 10^60, what actually happened occurred in a total event size of less that 10^30.  Obviously, Behe has badly mis-estimated the “Edge of Evolution”.  <strong>Briefly stated, his “Edge of Evolution” is wrong.</strong></p>

<p>I’ll close this essay by noting one source of error on Behe’s part.  As I have discussed, Behe asserts that the probability associated with a “CCC” is 1 in 10^20. Where does this number come from?  From footnote 16 in the first excerpt given above - White, N. J. 2004. Antimalarial drug resistance. J. Clin. Invest. 113:1084-92.  Here is the actual passage from the review by White that mentions the number 10^20:</p>

<blockquote>
  <p>“Chloroquine resistance in P. falciparum may be multigenic and is initially conferred by mutations in a gene encoding a transporter (PfCRT) (13). In the presence of PfCRT mutations, mutations in a second transporter (PfMDR1) modulate the level of resistance in vitro, but the role of PfMDR1 mutations in determining the therapeutic response following chloroquine treatment remains unclear (13). At least one other as-yet unidentified gene is thought to be involved. Resistance to chloroquine in P. falciparum has arisen spontaneously less than ten times in the past fifty years (14). This suggests that the per-parasite probability of developing resistance de novo is on the order of 1 in 10^20 parasite multiplications. “</p>
</blockquote>

<p>Recall that Behe equated one CCC with a double mutation, presumably based on other work showing that two point mutations in the PfCRT gene are associated with durable resistance in the parasite.  But White is not talking about double mutations in PfCRT when he tosses out the number 10^20.  Rather, he is speculating about the frequency of occurrence of a multigenic trait that involves two or three genes, and more (perhaps many more) than two mutations.  In other words, Behe’s use of this citation to argue that the natural frequency of occurrence of a double mutation in PfCRT is 10^20 is inappropriate.  This is one reason (not the only reason, but one) why Behe’s claims are so out of touch with reality.</p>

<p>Of course, there is more, much more.  Lots and lots of combinatorial chemistry and protein structural research refutes Behe.  As does the fact that every trait that Behe imagines as demanding multiple simultaneous mutations actually do not.  Not only has Behe miscalculated the Edge of Evolution, he is actually pointing in entirely the wrong direction in looking for this edge.  But these are issues for other essays.  The example that is recalled here shows clearly that Behe’s claims are wrong.</p>

<p>As a postscript, two points of interest.  It may have occurred to readers that the pre-emption of the Edge of Evolution by my essay is more than coincidence.  But trust me – I’m not the Sal Bonpensiero of this on-going soap opera, who had secret access to Behe’s book ahead of time.  (That having been said, I don’t think I’ll be taking boat rides with any ID bigwigs.)</p>

<p>No, what this all means is simpler.  The earlier PT essay was a belated discussion related to Darwin’s Black Box, a pulling together of ideas that had been bounced around the internet for more than ten years.  The fact that <a href="http://www.pandasthumb.org/archives/2007/05/on_the_evolutio_1.html#more">the previous essay</a> is also a one-stop shopping mall of items that refutes the Edge of Evolution simply pushes home the message that the EoE is little more than a re-working of DBB.  What is supposed to be revolutionary is actually just more of the same tired antievolutionary arguments.</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2007/07/reality-1-behe.html';
this.page.identifier = '/archives/2007/07/reality-1-behe';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


