<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>3 recent reports use evolution to study mechanisms of antibody diversification</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="3 recent reports use evolution to study mechanisms of antibody diversification" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2006/07/3-recent-report.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Matt Inlay" />
  <meta property="article:published_time" content="2006-07-20T13:16:31-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2006/07/3-recent-report.html">3 recent reports use evolution to study mechanisms of antibody diversification</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Matt Inlay</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2006-07-20T13:16:31-07:00">July 20, 2006 13:16 MST</time></span>
	  </div>
   </header>
  In Chapter 6 of Darwin's Black Box, Michael Behe listed several immune subsystems that he considered irreducibly complex (IC), and therefore (according to him but no one else) unevolvable.  One incredibly complex immune subsystem that Behe neglected to mention was the system that genetically modifies antibody genes during the course of an immune response.  This system is largely responsible for our ability to generate stronger and faster resistance to subsequent infections, and is integral to why vaccines work.  3 recent papers used concepts in evolution to help characterize one of the most interesting and novel immunological genes discovered since the RAGs in the late 80s, a gene called activation induced deaminase (AID, pronounced as initials), a gene pivotal to antibody modification.  Not only did these papers reveal interesting functional insights into AID, but also helped solidify a model for the origin and evolution of this system.  Two of the articles come from labs instantly recognizable to most molecular immunolgists, but are totally unknown in ID/evolution circles.  The third comes from a lab that most of the regulars here at the Panda's Thumb would immediately recognize, PT's own Andrea Bottaro.

<a class="anchor" id="more"></a>

<b>Background: post-activation modifications of antibody genes</b>
Every antibody-producing B cell expresses an antibody of a different specificity.  That specificity is determined at two distinct stages of development.  During early B cell development, antibody genes are assembled through the somatic rearrangement of gene segments in a process known as V(D)J recombination.  The Recombination Activating Genes, RAG-1 and RAG-2, are the only B cell specific genes essential for this reaction.  V(D)J recombination was originally identified by Behe as one of his IC systems in Darwin's Black Box.  Its mechanics and putative evolution are discussed <a href=" http://www.talkdesign.org/faqs/Evolving_Immunity.html">here</a>.  After a B cell is activated by binding to a foreign agent, the B cell proliferates and can further diversify its antibody genes through three processes.  In <b>Somatic Hypermutation</b> (SHM), the antibody genes undergo point mutations within their binding regions, and through an undeniably Darwinian process, B cells that produce antibodies with beneficial mutations are selected for and expand.  In <b>Class Switch Recombination</b> (CSR), the exons encoding the tail end of the antibody (called the constant region) are swapped with different constant region exons, altering which immune cells can interact with the antibody and changing the way the immune system utilizes it.  In <b>Gene Conversion</b> (GC), the binding region of the antibody genes are swapped with regions contained within antibody pseudogenes.  This process is superficially similar to CSR, but because it occurs in the binding region, it has the same functional effect as SHM.  As GC does not occur in mice or man, it will not be further discussed here.  Through a combination of these three processes, antibody genes are optimized for each specific infection.

<b>AID enters the picture</b>
Mechanistically, SHM, CSR, and GC are quite different, and until recently, were considered unique subsystems.  However, at the turn of the millennium, a gene called activation induced deaminase (AID) was discovered to be essential for all three processes.  While much is unknown about AID's function, the current consensus is that AID deaminates cytosine nucleotides, converting them into uracils within the antibody genes.  Depending on where the deamination occurs and the DNA repair pathway employed to fix it, SHM, CSR, or GC results.  SHM has been observed in all organisms which possess an antibody rearranging immune system, from cartilaginous fish to mammals.  However, CSR is present only in tetrapods (e.g. amphibians, reptiles, birds, and mammals).  Both cartilaginous and bony fish possess a functional AID gene, but not the other components necessary for CSR, namely the additional constant regions and other regions called switch regions, where AID targets.  Each constant region has a switch region immediately in front of it, and when the switch region of both the original constant region, and a second constant region undergo AID-mediated cytosine deamination, the two switch regions recombine and the constant regions are swapped.  Each switch region has its own promoter.  As AID preferentially targets transcriptionally active loci, the choice of which switch region is targeted is determined primarily by their transcriptional activity.

<img alt="AID.png" src="/uploads/2006/AID.png" width="720" height="540" />

<quote><b>Figure 1</b>
<b>A.</b>  Genetic organization of a rearranged antibody heavy chain gene.  The binding region, which is the rearranged VDJ exon is shown in red/yellow/blue.  The 4 exons of the (mu) constant region are in green and the (gamma) constant region is in purple.  The switch regions are represented by circles.  <b>B.</b>  In a putative model of somatic hypermutation, AID binds to the VDJ exon of the antibody and converts random cytosines to uracils (represented by asterisks).  When the mismatch repair machinery repairs the mutated DNA, it utilizes an error-prone polymerase, often leaving a mutation near the site of conversion.  <b>C.</b>  In class switch recombination (CSR), AID targets the switch regions and converts cytosines to uracils there.  The DNA repair machinery uses homologous recombination to repair the damage, causing the region between the two switch regions to be deleted, and changing the constant region exons.</quote>

<b>Evolution of CSR</b>
Like the V(D)J recombination system discussed here, it is not difficult to imagine how a three part irreducibly complex system like CSR could have evolved from another system.  AID was already present in early tetropods, as SHM was and still is active in these organisms.  The appearance of additional constant regions would only require a duplication of the original constant region, an entirely plausible event given that the bony fish that do not undergo CSR often contain multiple constant regions.  In these organisms, the choice of which constant region is used is determined at the level of alterative splicing.  Therefore, the only unique feature of the CSR system is the switch regions.  Switch regions are extremely heterogenous in sequence, and have very little similarity to one another.  The only feature they share in common is a propensity to form loop-like structures when in a single strand (such as when they're undergoing transcription).  In fact, in some species of frog, switch regions are merely a region of repetitive AGCT tetramers, which don't even form loops.  It isn't difficult to imagine a scenario by which AID could be co-opted from an SHM system for use in a primitive CSR system, swapping the original constant region with a duplicated one.  But what about the switch regions?  How will AID target the constant regions if no switch regions are present?  The binding regions of the antibody genes don't contain sequences that resemble switch regions, yet they are ample targets for AID-mediated cytosine deamination, indicating that switch regions wouldn't even be necessary for a primitive CSR system.  It's entirely possible that the switch regions evolved after CSR, providing better targets for AID.

Because they don't have a research program of their own, IDists would be quick to play the incredulous card, doubting the feasibility of the scenario and demanding an unreasonable burden of proof to prevent them from dismissing the evolvability of the system altogether.  While there are certainly gaps to be filled, no one denies that, the point of science is that the gaps <i>can</i> be filled.  This is unlike ID, where no research is ever proposed and certainly never, ever conducted.  

<b>What they did</b>
One issue with the evolution of CSR from SHM is whether the original AID in the early tetropods was capable of mediating a CSR-type reaction.  The C-terminal region of AID is required for CSR, but not SHM, suggesting that additional components may be necessary to go from an SHM-capable AID protein to a CSR-capable one.  In a recent issue of the <i>Journal of Experimental Medicine</i>, Vasco Barreto and colleagues from Michel Nussenzweig's lab reported that the AID gene from zebrafish was capable of activating CSR when transfected into mouse B cells that lacked their own AID gene.  In this simple experiment, Barreto et al. demonstrated that even though bony fish do not possess a CSR system, their AID was fully able to catalyze CSR should the other components of the CSR system appear.  They also examined pufferfish AID and found extremely low levels of CSR.  In a similar study, published in the journal <i>International Immunology</i>, Wakae and collegues in the lab of Masamichi Muramatsu (who discovered AID while a post-doc in Tasuko Honjo's lab), tested both zebrafish AID as well as catfish AID, and found similar results.  With these two studies, there could be no doubt that that particular step in the proposed evolution of the CSR system was entirely feasible.  Additionally, it demonstrated that AID's catalytic mechanism for SHM was similar enough to CSR that it would be preserved in bony fish even after 400 million years of differential selection.  

Travis Ichikawa and colleagues, in the lab of PT's own Andrea Bottaro reported a more detailed analysis of pufferfish and frog AID in the <i>Journal of Immunology</i>.  While frog AID was fully capable of mediating CSR in mouse AID-deficient B cells, they found essentially no CSR when pufferfish AID was transfected (remember, frogs have CSR, but pufferfish don't).  Ichikawa et al. discovered that the catalytic domain of pufferfish AID was CSR competent, but the non-catalytic domain was inhibiting the reaction.  As the C-terminal non-catalytic domain contains a nuclear export signal (NES) required for CSR activity (but not SHM) in mice, it was hypothesized that the pufferfish NES was somehow incompatible with mouse CSR.  However, Ichikawa et al. disproved that notion by first observing that pufferfish AID had no difficulties in nuclear export/import in mouse B cells, and second that when replacing the NES sequence on human AID, the pufferfish NES did not negatively impact CSR. 

<b>Why is this significant?</b>
While the details of class-switching and somatic hypermutation are of interest to only a handful of scientists, this research is significant to the ID/evolution debate in several ways.  First of all, we now have clear-cut evidence of how a three part IC system could have evolved.  No modifications would be necessary to convert an SHM-catalyzing AID gene into a CSR-catalyzing one.  This provides yet another crystal clear example of co-option that can be widely cited for years to come.  On another level, this research shows that more and more researchers are utilizing concepts in evolution to provide functional insight into their own areas of interest.  The standard approach to characterizing a gene is to make mutations in important domains and see how that affects the function of the protein.  However, by swapping pieces with another organism, this can identify important functional domains that no one might have thought to make mutations in.  Furthermore, it helps to form a model for the evolution of the system, and this can lead to more functional predictions.  Function informs evolution, and evolution informs function.  Contrast this with ID.  Because ID, as currently formulated by its chief advocates, places no constraints on the abilities or tendencies of the designer, all data is equally probable and equally consistent.  Therefore, no functional predictions can be made from ID, and no functional data could ever provide evidence for or against ID.  This is the primary reason why ID's most vociferous advocates don't do any research on ID.  This brings up another reason for the significance of these reports.  You may have heard Behe or Wells propose some experiments on ID.  Aside from the fact that those experiments have nothing to do with ID, you've never heard either of them actually <i>do</i> the experiments they proposed.  It's one thing to sit around and come up with ideas, it's another thing entirely to act on them.  It costs nothing to propose an experiment, but to conduct one takes money, resources, and a few years of blood and sweat.  <i>That</i> is the real B.S. detector for scientists.  If their experiments are so great, why aren't they doing them?  Here we have a terrific example of someone who posts regularly at the Panda's Thumb throwing his hat into the ring and doing an evolution-inspired experiment.  Let's see Behe or Dembski or Wells do that.

<b>References</b>
1.  Barreto VM, Pan-Hammarstrom Q, Zhao Y, Hammarstrom L, Misulovin Z, Nussenzweig MC. 
AID from bony fish catalyzes class switch recombination.
J Exp Med. 2005 Sep 19;202(6):733-8.

2.  Wakae K, Magor BG, Saunders H, Nagaoka H, Kawamura A, Kinoshita K, Honjo T, Muramatsu M. 
Evolution of class switch recombination function in fish activation-induced cytidine deaminase, AID.
Int Immunol. 2006 Jan;18(1):41-7.

3.  Ichikawa HT, Sowden MP, Torelli AT, Bachl J, Huang P, Dance GS, Marr SH, Robert J, Wedekind JE, Smith HC, Bottaro A. 
Structural phylogenetic analysis of activation-induced deaminase function.
J Immunol. 2006 Jul 1;177(1):355-61.
<a href="http://www.urmc.rochester.edu/smd/mbi/xenopus/XlAID-JI06.pdf">Link to PDF</a>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2006/07/3-recent-report.html';
this.page.identifier = '/archives/2006/07/3-recent-report';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a gray vertical bar to the left of the commenter's avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


