<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Continuous geographic structure is real, "discrete races" aren't</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Continuous geographic structure is real, "discrete races" aren't" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2012/02/continuous-geog.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Nick Matzke" />
  <meta property="article:published_time" content="2012-02-29T14:38:41-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2012/02/continuous-geog.html">Continuous geographic structure is real, "discrete races" aren't</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Nick Matzke</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2012-02-29T14:38:41-07:00">February 29, 2012 14:38 MST</time></span>
	  </div>
   </header>
  <p><em>Note: This topic is outside of my specialty, so it may be that I have missed some important points.  I think I’ve got the basics correct, but this is a very complex topic.  I will be interested in critical but constructive posts in the comments.</em></p>

<p><strong>Update:</strong> required reading, which basically confirms my points I think:</p>

<p>Weiss, K. M. and J. C. Long (2009). “<a href="http://dx.doi.org/10.1101/gr.076539.108">Non-Darwinian estimation: My ancestors, my genes’ ancestors</a>.” <em>Genome Research</em> 19(5), 703-710.</p>

<p>Nievergelt, C. M., O. Libiger and N. J. Schork (2007). “<a href="http://dx.doi.org/10.1371/journal.pgen.0030051">Generalized Analysis of Molecular Variance</a>.” <em>PLoS Genetics</em> 3(4), e51.</p>

<p>On Monday, Jerry Coyne at <a href="http://whyevolutionistrue.wordpress.com/">Why Evolution is True</a> posted on “<a href="http://whyevolutionistrue.wordpress.com/2012/02/28/are-there-human-races/">Are there human races?</a>”  While acknowledging the very bad history of the race concept in human history, and noting some of the problems with applying the concept to humans, Coyne concluded, basically, that the answer was yes, there are human races.  While reviewing <a href="http://www.americanscientist.org/bookshelf/pub/race-finished">Jan Sapp’s piece</a> which concluded that human races did not objectively exist, Coyne wrote:</p>

<blockquote>
  <p>As Sapp notes, and supports his conclusion throughout the review:</p>

  <blockquote>
    <p>Although biologists and cultural anthropologists long supposed that human races–genetically distinct populations within the same species–have a true existence in nature, many social scientists and geneticists maintain today that there simply is no valid biological basis for the concept. The consensus among Western researchers today is that human races are sociocultural constructs.</p>
  </blockquote>

  <p>Well, if that’s the consensus, I am an outlier.  I do think that human races exist in the sense that biologists apply the term to animals, though I don’t think the genetic differences between those races are profound, nor do I think there is a finite and easily delimitable number of human races.  Let me give my view as responses to a series of questions.</p>
</blockquote>

<p>Unfortunately, while Coyne’s position is admirably clearly expressed, his reasons for accepting race don’t hold up.  Some of his commentators have pointed out some of the problems in the comments.  My own critique of his position is informed by a symposium I attended in San Francisco last year, “<a href="https://sites.google.com/site/babsspringconference2011/">Is There Space for Race in Evolutionary Biology?</a>”, and particularly by the arguments that speakers Massimo Pigliucci and Alan Templeton made there.</p>

<p>Here’s the conference information:</p>

<blockquote>
  <p><a href="https://sites.google.com/site/babsspringconference2011/">Is There Space for Race in Evolutionary Biology?</a></p>

  <p>The 2011 Spring Conference for the <a href="http://www.biosystematists.org/">Bay Area Biosystematists</a></p>

  <p>Are human races geographical subspecies?  How about distinct evolutionary lineages?  What about ecotypes?  Does it even make sense to divide humans into subspecies?  Can there be biological races in other species even though there aren’t any in humans?  This conference aims to answer these questions and more as it explores the question of whether race can be validly defined in the context of evolutionary biology.  Come join us at the 2011 Spring Conference for the Bay Area Biosystematists!  Please peruse this site to learn more about the conference.</p>

  <p>Date: May 14, 2011
Time: 12:45-7pm
Location: McLaren Conference Center, Room 250, University of San Francisco</p>
</blockquote>

<p>Coyne asks a series of questions and gives answers:</p>

<p><strong>1. What are races?</strong>  Coyne answers:</p>

<blockquote>
  <p>“races of animals (also called “subspecies” or “ecotypes”) are morphologically distinguishable populations that live in allopatry (i.e. are geographically separated)”</p>
</blockquote>

<p><strong>2. Under that criterion, are there human races?</strong> Coyne answers:</p>

<blockquote>
  <p>Yes.  As we all know, there are morphologically different groups of people who live in different areas, though those differences are blurring due to recent innovations in transportation that have led to more admixture between human groups.</p>
</blockquote>

<p><strong>3. How many human races are there?</strong>  Answer:</p>

<blockquote>
  <p>That’s pretty much unanswerable, because human variation is nested in groups, for their ancestry, which is based on evolutionary differences, is nested in groups.</p>
</blockquote>

<p><strong>4. How different are the races genetically?</strong>  Answer:</p>

<blockquote>
  <p>Not very different. […]</p>
</blockquote>

<p>…but he goes on to cite the evidence of the results of clustering analyses that are often able use genetic data to place humans quite accurately into one population or another.  I.e., the argument is basically that if there is enough signal in the genetics to place humans into one population or another, then races exist.</p>

<p><strong>5. Why do these differences exist?</strong> Answer:</p>

<blockquote>
  <p>The short answer is, of course, evolution.  The groups exist because human populations have an evolutionary history, and, like different species themselves, that ancestry leads to clustering and branching, though humans have a lot of genetic interchange between the branches!</p>
</blockquote>

<p>He goes on to discuss the possible influence of natural and sexual selection on a few traits, and genetic drift and neutral evolution on the rest.</p>

<p><strong>6. What are the implications of these differences?</strong>  Coyne writes:</p>

<blockquote>
  <p>Not much.  There are some medical implications.</p>
</blockquote>

<p>…and goes on to (appropriately) dismiss certain pernicious ideas like the hard-to-kill one that IQ is genetically different between different races.  However, he does leave open the possibility that a trait like intelligence could vary genetically between races, although it seems unlikely to him, he says there is no evidence supporting this, and in any event even a statistically-detectable difference between populations would be swamped by the within-population variability in such a trait. (Therefore, even in the event that such a population-level difference was found,  one could not safely extrapolate from a race identification to a prediction about an individual.)</p>

<p><strong>The critique</strong></p>

<p>The key problems with the point of view that Coyne presents were made clear, at least to me, by Alan Templeton’s talk on this subject.  I went to the talk in a quite skeptical mood, since Templeton is in my judgement on the wrong side of certain issues in phylogeography*.  But Templeton’s points on race are simple and pretty convincing.  See also:</p>

<blockquote>
  <p>Templeton, A. R. (1998). “Human Races: A Genetic and Evolutionary Perspective.” American Anthropologist 100(3), 632-650.</p>

  <p><em>Race</em> is generally used as a synonym for <em>subspecies</em>, which traditionally is a geographically circumscribed, genetically differentiated population. Sometimes traits show independent patterns of geographical variation such that some combination will distinguish most populations from all others. To avoid making “race” the equivalent of a local population, minimal thresholds of differentiation are imposed. Human “races” are below the thresholds used in other species, so valid traditional subspecies do not exist in humans. A “subspecies” can also be defined as a distinct evolutionary lineage within a species. Genetic surveys and the analyses of DNA haplotype trees show that human “races” are not distinct lineages, and that this is not due to recent admixture; human “races” are not and never were “pure.” Instead, human evolution has been and is characterized by many locally differentiated populations coexisting at any given time, but with sufficient genetic contact to make all of humanity a single lineage sharing a common evolutionary fate.</p>
</blockquote>

<p>Taking Coyne’s points in order, I will summarize what I think Templeton would say, and add my own points.</p>

<p><strong>1. What are races?  Are human races “allopatric”?</strong> Templeton would deny that human populations “live in allopatry”.  Humans are not, in fact, living in discrete, disconnected populations.  Instead, humans are pretty continuously distributed from Africa all the way around the globe, and have been since glacial times (the tiny possible exceptions to this statement involve the recent colonization of remote islands e.g. by Polynesians, but this occurred only in the last few hundred-2000 years, a geological eyeblink).  And genetic contact, i.e. interbreeding, seems to have been occurring between adjacent “populations” the whole time.</p>

<p>The overwhelming signal in human genetics seems to be isolation-by-distance – the larger the geographical distance between two samples, the greater the average genetic difference.  But it is <em>crucial</em> to understand that this is <em>not</em> a confirmation of the idea of discrete races.  If races were discrete, if you sampled along a transect, you would sample individuals from race A, A, A, A, and then suddenly switch to race B, B, B, B.</p>

<p>But what we typically see instead, if you sample along a transect, is that humans measured by some index of genetic difference along a transect would produce values with something like 0, 1, 2, 3, 4, 5, 6, 7.  (I am making these numbers up, but imagine that they are a measurement of genetic difference from the first sample, i.e. the difference is 0 at the first sampling location, 1 at a sample 1000 km away, 2 at a sample 2000 km away, etc.)</p>

<p>This is a *crucial* distinction to get before we proceed, so I encourage readers to go back and re-read the previous paragraphs to get the idea into their head.  Key concept: if you sample humans along a transect, the typical pattern is that genetic difference will increase in a continuous fashion with greater distance between samples, rather than in large discrete “steps” as you would expect with well-separated allopatric populations, or discrete “races”.</p>

<p>Here’s is Templeton’s depiction of this phenomenon:</p>

<p><img src="/uploads/2012/Templeton_1999_AA_Fig5_geographic_distance_in_miles_vs_genetic_distance.png" alt="Templeton_1999_AA_Fig5_geographic_distance_in_miles_vs_genetic_distance.png" width="533" height="605" style="text-align: center; display: block; margin: 0 auto 20px;" class="mt-image-center" /></p>

<p><strong>2. Under that criterion, are there human races?</strong> Coyne answers “yes”, because we can see obvious differences between humans in different locations.  But that, by itself, does not establish that the differences are discrete and categorical, which is what would have to be the case for ideas like “counting the number of races” and “distinguishing one race from the next” to make sense.  Quite often, geneticists like Coyne seem to be being mislead by the fact that <em>sampling is usually discrete</em>.  Usually, scientists go and get some samples from one region in Africa, some samples from one region in Europe, some from a region or two in Asia, etc.  Before there was genetic sampling, there were similar issues in skull measurements, photographs, anthropologists’ monographs, etc.</p>

<p>When this form of sampling is done, of course it *looks like* the races are discrete, but what was actually discrete was your sampling.  Templeton’s assertion is that if you had sampled on a grid, instead, and sampled all regions equally, you wouldn’t be able to draw discrete lines between races.  The pattern would be that closer samples are more related, but this similarity would decline continuously with distance, and not in big discrete steps.</p>

<p><strong>3. How many human races are there?</strong> Coyne says the question is “pretty much unanswerable”, and notes that the answers have varied from 3 to over 30.  Coyne is correct to say the question is unanswerable, and the reason this is so is easily explained on Templeton’s view.  But the unanswerability is quite a puzzle – or ought to be – if one is trying to argue that races are real.  If you are going to assert that there is a rigorous and objective and scientific concept called “race”, but you can’t even explain how to objectively identify the races and thus count them, there’s a big problem somewhere.</p>

<p>For example, even though there are all kinds of definitions of “species”, and thus some variation in the count in the number of species, at least (a) the various species concepts are more-or-less objective, and (b) it happens that different concepts will often, although not always, give the same or similar answers.  But, it is well-known that if “defining a species” is somewhat difficult and controversial, it is much worse for defining a subspecies, and it is close-to-hopeless for the ubiquitous concept of “population”.  There are <a href="http://scholar.google.com/scholar?q=%22species+concepts%22&amp;hl=en&amp;btnG=Search&amp;as_sdt=1%2C5&amp;as_sdtp=on">hundreds of papers on species concepts</a>, but there are almost none published on “population concepts” (<a href="http://http://www.mitpressjournals.org/doi/abs/10.1162/biot.2009.4.3.267">this, Millstein 2009,</a>  is one of the only ones; see also <a href="http://scholar.google.com/scholar?cites=14980497598332371470&amp;as_sdt=2005&amp;sciodt=0,5&amp;hl=en">discussion of the paper</a>).  In practice, “populations” of organisms are basically either merely convenient geographic groupings determined subjectively by biologists for convenience or sampling purposes, very often geographically defined; or, they are convenient abstract entities useful for mathematical and computational modeling.  But whether or not the “populations” you create for sampling and analysis purposes represent real discrete groups out there in the real world is an empirical question, and it looks like, in the case of humans, there is little basis for assuming discrete populations.</p>

<p>Finally, Coyne asserts at one point in his blogpost that races exist in humans much as subspecies exist in other species.  But in terms of genetic divergence, at least, this is difficult to support.  Templeton points out that the genetic divergence among living humans is much less than that typically observed between subspecies.  This is not controversial or poorly known so I won’t argue this in detail.  Templeton (1999) provides a graphical version:</p>

<p><a href="http://pandasthumb.org/assets_c/2012/02/Templeton_1999_AA_Fig1_Fst_for_humans_mammals_compared-952.html"><img src="/uploads/2012/Templeton_1999_AA_Fig1_Fst_for_humans_mammals_compared-thumb-700x619-952.png" alt="Templeton_1999_AA_Fig1_Fst_for_humans_mammals_compared.png" width="700" height="619" style="text-align: center; display: block; margin: 0 auto 20px;" class="mt-image-center" /></a></p>

<p>(Note: see <a href="http://pandasthumb.org/archives/2012/02/continuous-geog.html#comment-279881">further discussion of criticism of Templeton (1998)’s Figure 1 at this comment</a>.)</p>

<p>Or, we can compare the branch lengths of a tree of built of the aligned sequences from the mitochondria of humans and our ape relatives:</p>

<p><img src="/uploads/2012/Gagneux_etal_1999_PNAS_mtDNA_African_hominids_Fig1_phylogeny.png" alt="Gagneux_etal_1999_PNAS_mtDNA_African_hominids_Fig1_phylogeny.png" width="405" height="602" style="text-align: center; display: block; margin: 0 auto 20px;" class="mt-image-center" /></p>

<p>(From: <a href="http://www.pnas.org/content/96/9/5077.full">Gagneux et al. 1999</a>, <em><a href="http://www.pnas.org/">PNAS</a></em>; <a href="http://www.pnas.org/content/96/9/5077/F1.large.jpg">full-resolution figure</a>) (Hat tip: <a href="http://dererumnatura.us/archives/2005/09/chimps-are-laug.html">Reed Cartwright</a>)</p>

<p>Now, it is important to remember that the mitochondrial DNA sequence is really just one nonrecombining genetic marker, and it is possible for the mtDNA phylogeny to differ from the true species phylogeny (due to introgression) or to not reflect the pattern of diversity in the nuclear markers.  Nuclear alleles coalesce more slowly, and recombination means that nuclear DNA will not have a simple branching phylogenetic history that can be easily represented by a tree, at least not within an interbreeding population.  However, my sense of it is that in the great ape group (including humans), the mtDNA accurately represents the overall picture of phylogeny and genetic diversity.</p>

<p><strong>5. Why do these differences exist?</strong> Coyne writes:</p>

<blockquote>
  <p>The short answer is, of course, evolution.  The groups exist because human populations have an evolutionary history, and, like different species themselves, that ancestry leads to clustering and branching, though humans have a lot of genetic interchange between the branches!</p>
</blockquote>

<p>Obviously we agree that selection, drift, etc. explain the evolution of observed diversity in the human species.  But there are important questions here that Coyne doesn’t address.  (1) How do we know that we’ve got discrete or even somewhat discrete “populations” which then can be said to have a distinct evolutionary history.  Maybe we’ve just got one big interconnected population.  This was addressed above.  (2) How much interchange do you have to have to have before claims of clustering and branching break down?  The evidence seems to indicate that the genetic differences between human “populations” in different regions are primarily a matter of differences in allele frequencies, but where most non-tiny populations have most of the alleles observed in the global population, just with varying frequencies.  If this observation is true, this causes a problem for Coyne’s statement, because another way to say “the differences are differences in allele frequencies” is to say that “the differences are not fixed between populations”, and another way to say that is “coalescence has not occurred in these populations for these markers, either because of the lack of time or the presence of gene flow due to migration or both”.  Yet another way to say this is to say that “for most loci, these populations are not reciprocally monophyletic”, and if you are a phylogeneticist, you known that this statements is equivalent to saying “your data are not appropriately represented by a single branching tree, instead you have a number of trees representing the individual histories of individual loci in the population”.  Yet one more way of saying it is “you don’t have discrete populations, you are sampling within one genetically connected population.”</p>

<p>Coyne mentions clustering results, which have indeed impressed some people, and led to a minor resurgence in the race concept among geneticists in the last decade.  It is true that, if you sample a region where people have mostly lived in or near the villages where they grew up for dozens of generations (e.g. parts of Europe, and e.g. not North America), you can take an individual’s genetic data and place their ancestry on the map with a fair degree of accuracy.</p>

<p>And it is also true that a computer algorithm, if given a number of samples from e.g. Africa, Europe, and China, could accurately place the samples into those 3 clusters, if told the number of clusters ahead of time.  It is even true that newer algorithms can figure out the number of clusters in the data on their own, e.g. <a href="http://cteg.berkeley.edu/~structurama/authors.html">Structurama</a> (full disclosure: authored by my advisor).</p>

<p>These sorts of observations have led a number of geneticists to speak of “Lewontin’s Fallacy” (e.g. <a href="http://dx.doi.org/10.1002/bies.10315">Edwards 2003</a>).  The geneticist Richard Lewontin famously argued in 1972 that “about 85% of the total genetical variation is due to individual differences within populations and only 15% to differences between populations or ethnic groups” (Edwards 2003).  Lewontin argued that this fact made the idea of racial classification irrelevant, since most variation was within groups instead of between them.  You can see his point in an intuitive way by imagining that you have a sample of two people.  Assuming the 85%/15% distribution of variation, for any one locus that varies between the two individuals, it is much more probable that the difference is just due to within-population variability than due to between-population differences.</p>

<p>Lewontin’s paper and argument have been cited thousands of times – for example, it was cited in just this fashion in the review by Sapp which Coyne is critiquing.</p>

<p>Edwards points out that the problem with Lewontin’s statement is that it is only true if you have only one locus to look at, or if you can safely assume that the variation at all loci is independent.  It turns out that if you have, say, dozens or hundreds or thousands of loci to look at, and there is some population structure in the data, for example, population-level differences in allele frequencies, then by looking at all those loci, it becomes quite easy to identify the clusters and to accurately place individuals into them.  Edwards describes how this works:</p>

<blockquote>
  <p>Consider two haploid populations each of size n. In population 1 the frequency of a gene, say “+” as opposed to “-“, at a single diallelic locus is p and in population 2 it is q, where p + q = 1. (The symmetry is deliberate.) Each population manifests simple binomial variability, and the overall variability is augmented by the difference in the means. The natural way to analyse this variability is the analysis of variance, from which it will be found that the ratio of the within-population sum of squares to the total sum of squares is simply 4pq. Taking p = 0.3 and q = 0.7, this ratio is 0.84; 84% of the variability is within-population, corresponding closely to Lewontin’s figure. The probability of misclassifying an individual based on his gene is p, in this case 0.3. The genes at a single locus are hardly informative about the population to which their bearer belongs.</p>

  <p>Now suppose there are k similar loci, all with gene frequency p in population 1 and q in population 2. The ratio of the within-to-total variability is still 84% at each locus. The total number of “+” genes in an individual will be binomial with mean kp in population 1 and kq in population 2, with variance kpq in both cases. Continuing with the former gene frequencies and taking k=100 loci (say), the mean numbers are 30 and 70 respectively, with variances 21 and thus standard deviations of 4.58. With a difference between the means of 40 and a common standard deviation of less than 4.6, there is virtually no overlap between the distributions, and the probability of misclassification is infinitesimal, simply on the basis of counting the number of “+” genes. Fig. 1 shows how the probability falls off for up to 20 loci.</p>

  <p><img src="/uploads/2012/Edwards_2003_BioEssays_Lewontins_Fallacy_Fig1_probability_of_misclassification.png" alt="Edwards_2003_BioEssays_Lewontins_Fallacy_Fig1_probability_of_misclassification.png" width="437" height="371" style="text-align: center; display: block; margin: 0 auto 20px;" class="mt-image-center" /></p>
</blockquote>

<p>This sort of information (obviously the between-population allele frequencies always differ between loci in real life, but this doesn’t matter much for the classification) is what lets the clustering algorithms find their clusters, and place people into them. Edwards concludes:</p>

<blockquote>
  <p>There is nothing wrong with Lewontin’s statistical analysis of variation, only with the belief that it is relevant to classification. It is not true that “racial classification is… of virtually no genetic or taxonomic significance”. It is not true, as <em>Nature</em> claimed, that “two random individuals from any one group are almost as different as any two random individuals from the entire world”, and it is not true, as the <em>New Scientist</em> claimed, that “two individuals are different because they are individuals, not because they belong to different races” and that “you can’t predict someone’s race by their genes”. Such statements might only be true if all the characters studied were independent, which they are not.</p>

  <p>Lewontin used his analysis of variation to mount an unjustified assault on classification, which he deplored for social reasons. It was he who wrote “Indeed the whole history of the problem of genetic variation is a vivid illustration of the role that deeply embedded ideological assumptions play in determining scientific ‘truth’ and the direction of scientific inquiry”.(5) In a 1970 article <em>Race and intelligence</em>(11) he had earlier written “I shall try, in this article, to display Professor Jensen’s argument, to show how the structure of his argument is designed to make his point and to reveal what appear to be deeply-embedded assumptions derived from a particular world view, leading him to erroneous conclusions.”</p>

  <p>A proper analysis of human data reveals a substantial amount of information about genetic differences. What use, if any, one makes of it is quite another matter. But it is a dangerous mistake to premise the moral equality of human beings on biological similarity because dissimilarity, once revealed, then becomes an argument for moral inequality. One is reminded of Fisher’s remark in <em>Statistical Methods and Scientific Inference</em>(12) “that the best causes tend to attract to their support the worst arguments, which seems to be equally true in the intellectual and in the moral sense.”</p>
</blockquote>

<p>Edwards’s statistical argument is correct, but unfortunately – and ironically for someone criticizing Lewontin for not recognizing a mistake caused by a “deeply-embedded assumption” – Edwards is at least potentially being misled by his own deeply embedded assumption, as are others who argue along the lines of “clustering of humans based on genetics works, therefore different races exist.”  Just because an analysis shows that the humans you sample fall into clusters <em>does not necessarily mean that the clusters are out there in the real world in the human population</em>.  If the human population basically has continuous genetic variation, with genetic similarity decaying smoothly with distance, then, <em>if your sampling was geographically clustered, your samples will show that structure</em>.</p>

<p>I am not an expert in this area, but my sense of the matter from Templeton’s comments is that it is much more true to say that, at least for humans living basically in their ancestral locations, genetic difference between “populations” is essentially a smooth function of geographic distance.  This is definitely “geographic structure” in the human population, but it is not evidence for discrete populations or races.  And genetic clustering that is due to clustered sampling (done for convenience, funding reasons, political access reasons, and/or prior assumptions that localized sampling of a region represents that region, because you “already know” that a particular race inhabits that region) cannot be taken as evidence of real discrete structure in the human population.</p>

<p>It is possible that there is some discrete structure – due to language barriers or large migration events or whatever – but I would want to see this explicitly tested for rather than just assumed.  And if it exists, it will be important to ask how much of genetic variation this explains, compared to the already major isolation-by-distance effect that we already know about and already know is quite strong.</p>

<p><em>A little more on clustering</em></p>

<p>One other point about clustering, this time with hierarchical clustering.  Hierarchical clustering algorithms produce groups within groups, which are commonly represented by a tree structure very similar (superficially) to a phylogeny.  You will sometimes see the argument that one can take the genetic distances between a bunch of samples of humans, throw them into a hierarchical clustering algorithm, and get a tree out, and therefore we can safely think of human races as subspecies within our species.  This matches up superficially with a lot of our background conceptual model of how species can be placed as groups within groups on a phylogeny, and thus tends to reinforce the “races are real” idea in some peoples’ heads.</p>

<p>However, this argumentation is at least dubious.  First, clustering methods taken as a phylogenetic method are a form of “distance method”, and distance methods are well-known in phylogenetics to be the crudest sort of phylogenetic method and the one most prone to errors; unless some pretty specific assumptions are met (e.g. molecular clock) they can produce very bad errors.  Basically, distance methods rely on the pairwise Euclidean distances between samples in your data space.  You can imagine this as the “as the bird flies” distance.  However, the distance that we are really interested when we produce a phylogeny is not the “as the bird flies”, Euclidean distance between our samples; instead, we are most interested in the <em>path distance</em>, i.e., the actual mutational steps (or character-change steps) that led from a common ancestor, up each branch, to the tips that we observe.  This form of distance is known as a “Manhattan distance”, named after the square street grid in downtown New York.  In New York City, you can’t get from A to B by going “as the crow flies”, you have to drive along the street grid, avoiding one-way streets, construction, traffic jams, etc.  Only in idea situations will the actual traveled path distance equal or closely approximate or correlate linearly with the as-the-crow-flies distance.  Phylogenetic methods are thus really about finding the best paths (branching patterns) between the observed samples, according to some criterion (e.g. parsimony, maximum likelihood, posterior probability).</p>

<p>Therefore, clustering methods are not phylogenetic methods, except in special situations.</p>

<p>Second, a hierarchical clustering method will produce hierarchical clustering from almost any but the most artificially-constructed distance data.  If there is true hierarchical structure, the method can find it, but some sort of clustering will be produced, even if it’s poorly supported and changes dramatically under e.g. resampling of the data.  And there can be many sources of any clustering that is observed.  I bet that you could:</p>

<p>* lay down a latitude/longitude grid on the earth,</p>

<p>* take a “sample point” at every intersection point between latitude and longitude lines (done at, say, 1 degree or 5 degree intervals)</p>

<p>* exclude points that fell in the ocean</p>

<p>* calculate the great circle geographic distance between all points</p>

<p>* input this into a hierarchical clustering algorithm</p>

<p>…and you would get out a nice treelike diagram that would show each of the major continents as clusters, probably with linked continents forming groups of groups.</p>

<p>This would be successful hierarchical clustering, for sure, but it obviously wouldn’t be telling you about the evolutionary connectedness of the continents or anything like that.  It’s a summary of the geographical distance matrix between the samples.</p>

<p>The interesting question is – what if you had the samples to calculate the genetic distances between the (indigenous) humans sampled at each of these points.  Would you get a clustering scheme all that much different than the one you would get by clustering pure geographic distance?  Probably there would be some differences, since we know that humans started in Africa and genetic diversity is highest there – but would it be a big effect?  Or would the “clustering” that is observed in the genetic distances primarily just be the effect of isolation by distance and the fact that humans live on land, which is unevenly distributed around the globe?  This might verify “race” in a very vague way, but it would be a long way from the traditional concepts that most people have in their head.</p>

<p>Lastly, we reach point 6.</p>

<p><strong>6. What are the implications of these differences?</strong>  Coyne writes:</p>

<blockquote>
  <p>Not much.  There are some medical implications. […]</p>
</blockquote>

<p>Here, interestingly, Templeton might actually agree, at least for the United States.  Although it is illegitimate to do discrete sampling of continuous variation, and then conclude that discrete variation exists in reality, it is important to note that in some cases, “discrete sampling” in history has played a role in producing modern populations.  For example, the population of the U.S. is not a random sample of the global human population.  For historical reasons, in the formation of the U.S. population there really were major contributions from discrete parts of the globe – Europe, West Africa, East Asia, etc.  For this reason, if you were to examine a few hundred randomly sampled Americans, you would indeed find detectable discrete genetic variation!  In this (very limited) sense, “race” is physically real, but that physical reality was completely constructed by contingent cultural events in recent history!</p>

<p>In certain cases, this might even be medically relevant, for example if certain genotypes are common in a certain population, and these genotypes react especially well or especially poorly to a particular drug.  Certainly you would want to know this if you were a doctor prescribing medication to a patient, and government agencies like the FDA will want to know it when they are approving drugs for sale or for insurance coverage.  And certainly, it seems arbitrary and unfair to do drug tests on one group (say, white males) and then assume that the results will apply with equal validity to everyone else.</p>

<p>Finally, the fact that there is some geographical structure (whether discrete or continuous) in the human population, or in the weird sample of the human population that we have in the U.S., is important to take into account for statistical reasons.  Many studies are attempting to correlate genetic traits with various diseases and conditions.  But these studies can be badly misled if there is correlation in the genetic data that is due to, say, geographic ancestry that is unrecognized.  This is a vast field, and pretty much represents a huge job market for evolutionary biologists and population geneticists who decades ago would have puttered around putting equations on chalkboard, but who are now crucial components of any well-done study involving genetic data from populations.</p>

<p>Those are some of the arguments favoring taking “race” into account in medicine and medical research.  However, as you can imagine, there are all kinds of difficulties and dangers.  A big one is that some statistical effect that is true in one situation – say, people who identify as African American have a higher risk of condition X – will be illegitimately extrapolated to other people who fall into doctors’ and the public’s <em>cultural</em> definition of African American.  What if one’s ancestry is from southern Africa, but the genetic trait that correlates with the condition derives from West Africa?  The statistical generalization made in some study may not apply to you.</p>

<p>This post is long enough, but hopefully readers can see that the topic of race in the human species is a complicated one, fraught with danger even from a purely scientific point of view, even before we get to the even more hazardous arenas of culture and politics.  I think that it should be clear that one cannot give the question “Do races exist?” any kind of simple “yes” answer in humans.  One can say “yes” to the question, “Is there geographical structure in human genetics?”, but hopefully I have shown that this is a long way from establishing that there is any kind of <em>discrete</em> genetic structure in the human population at large, i.e. discrete “racial groups” which could be identified and counted.  It is possible that with ever more data, some shadow of this idea may be statistically supportable, but it seems unlikely that it will compare in strength to the strong overall pattern of continuous genetic variation following an isolation-by-distance model.</p>

<p><strong>Conclusion</strong></p>

<p>I’ll let the conclusion of Templeton (1998) speak for itself:</p>

<blockquote>
  <p>Conclusions</p>

  <p>The genetic data are consistently and strongly informative about human races. Humans show only modest levels of differentiation among populations when compared to other large-bodied mammals, and this level of differentiation is well below the usual threshold used to identify subspecies (races) in nonhuman species. Hence, human races do not exist under the traditional concept of a subspecies as being a geographically circumscribed population showing sharp genetic differentiation. A more modem definition of race is that of a distinct evolutionary lineage within a species. The genetic evidence strongly rejects the existence of distinct evolutionary lineages within humans. The widespread representation of human “races” as branches on an intraspecific population tree is genetically indefensible and biologically misleading, even when the ancestral node is presented as being at 100,000 years ago.</p>

  <p>Attempts to salvage the idea of human “races” as evolutionary lineages by invoking greater racial purity in the past followed by admixture events are unsuccessful and falsified by multilocus comparisons of geographical concordance and by haplotype analyses. Instead, all of the genetic evidence shows that there never was a split or separation of the “races’” or between Africans and Eurasians. Recent human evolution has been characterized by both population range expansions (with perhaps some local replacements but no global replacement within the last 100,000 years) and recurrent genetic interchange. The 100,000 years ago “divergence time” between Eurasians and Africans that is commonly found in the recent literature is really only an “effective divergence time” <em>in sensu</em> Nei and Roychoudhury (1974, 1982). Since no split occurred between Africans and Eurasians, it is meaningless to assign a date to an “event” that never happened. Instead, the effective divergence time measures the amount of restricted gene flow among the populations (Slatkin 1991).</p>

  <p>Because of the extensive evidence for genetic interchange through population movements and recurrent gene flow going back at least hundreds of thousands of years ago, there is only one evolutionary lineage of humanity and there are no subspecies or races under either the traditional or phylogenetic definitions. Human evolution and population structure have been and are characterized by many locally differentiated populations coexisting at any given time, but with sufficient genetic contact to make all of humanity a single lineage sharing a common, long-term evolutionary fate.</p>
</blockquote>

<p>My recommendation would be to ditch the discussion of “human races” in science wherever possible (it may be unavoidable in certain fields, e.g. medicine, due to its cultural reality), particularly in genetics, and instead go with “geographic ancestry” instead.  Some of us humans will trace back to a small region somewhere, others of us will be fusions from two or five or six continents.  Coyne’s post argued that races exist, but were becoming less distinct due to interbreeding.  I would say that continuous geographic structure exists, but that it is becoming weaker as the human population becomes more and more <a href="http://en.wikipedia.org/wiki/Panmixia">panmictic</a>.</p>

<p><strong>Note:</strong> After I wrote the above, I noticed that <a href="http://whyevolutionistrue.wordpress.com/2012/02/29/more-on-genes-and-geography-diagnosing-your-home-from-your-dna/">Coyne has a followup post</a> up that cites the famous “map of Europe reflected in genetics” (paraphrasing) figure from <a href="http://www.nature.com/nature/journal/v456/n7218/full/nature07331.html">Novembre et al. (2008)</a>.  He notes the evidence for the isolation-by-distance effect, and concludes that there are no races in Europe, at least:</p>

<blockquote>
  <p>In other words, genetically closer populations are more genetically similar, as expected if individuals tend to mate with other individuals from the same country, and close by.  This is an “isolation by distance” model: genetic similarity falls off gradually with distance.  As the authors note, this does not support the existence of “discrete, well-differentiated populations,” i.e., there are no races. None are expected in such a small area, particularly because biological “races” are those populations that (at least at one time) were geographically isolated and genetically differentiated. That geographical isolation never happened in Europe.</p>

  <p>[…]</p>

  <p>As I said, this doesn’t show that there are discrete “races” in Europe, and I don’t think there are obviously discrete “races” anywhere these days, though there is large-scale genetic differentiation among worldwide population suggesting that such races once existed as relatively discrete and geographically isolated populations.  The discreteness that once existed, or so I think, is now blurring out as transportation and migration are beginning to mix the discrete groups into not a melting pot, but sort of a lumpy pudding of humanity.</p>

  <p>What is clear is that, with considerable accuracy, you can diagnose an individual’s geographic origin from his genes.  Nearly everyone’s DNA contains reliable information about their recent and ancient past.  We are not all genetically alike. If we were, you couldn’t do studies like the one of Novembre et al.  But neither are we radically different genetically, for if we were, you wouldn’t need hundreds of thousands of genes for such accurate predictions.</p>
</blockquote>

<p>So Coyne is actually approaching the Templeton view of human genetic diversity pretty rapidly, although he is still buying into “relatively discrete” races at a larger scale (what are they then?), which I would disagree with.</p>

<p><strong>References</strong></p>

<p>Edwards, A. W. F. (2003). “<a href="http://dx.doi.org/10.1002/bies.10315">Human genetic diversity: Lewontin’s fallacy</a>.” <em>BioEssays</em> 25(8), 798-801.</p>

<p>Gagneux, P., C. Wills, U. Gerloff, D. Tautz, P. A. Morin, C. Boesch, B. Fruth, G. Hohmann, O. A. Ryder and D. S. Woodruff (1999). “<a href="http://www.pnas.org/content/96/9/5077.full">Mitochondrial sequences show diverse evolutionary histories of African hominoids</a>.” <em>Proceedings of the National Academy of Sciences</em> 96(9), 5077-5082.</p>

<p>Lewontin RC. The apportionment of human diversity. In: Dobzhansky T, Hecht MK, Steere WC, editors. <em>Evolutionary Biology</em> 6. New York: Appleton-Century-Crofts. 1972. p 381-398.</p>

<p>Millstein, R. L. (2009). “<a href="http://http://www.mitpressjournals.org/doi/abs/10.1162/biot.2009.4.3.267">Populations as Individuals</a>.” <em>Biological Theory</em> 4(3), 267-273.</p>

<p>Templeton, A. R. (1998). “<a href="http://dx.doi.org/10.1525/aa.1998.100.3.632">Human Races: A Genetic and Evolutionary Perspective</a>.” <em>American Anthropologist</em> 100(3), 632-650.</p>

<p><strong>Notes</strong></p>

<p>* For example (1) he promotes a method that seems to be subjective and to not work (it’s called Nested Clade Analysis, it’s not about nested clades in the traditional phylogenetic/macroevolutionary senses of those words, and it would take awhile to explain…but see “<a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2008.00481.x/full">Why does a method that fails continue to be used?</a>” by Lacy Knowles for the basics, and <a href="http://scholar.google.com/scholar?hl=en&amp;q=nested+clade+analysis+false+positive&amp;btnG=Search&amp;as_sdt=0%2C5&amp;as_ylo=&amp;as_vis=0">Google Scholar</a> for the ongoing discussion.) and (2) he doesn’t seem to get Bayesian methods and his one-man war against ABC (Approximate Bayesian Computation) consists primarily of misunderstandings of Bayesian logic (although ABC does have various limitations and problems, they aren’t primarily the ones he identifies; see e.g. “<a href="http://www.sciencedirect.com/science/article/pii/S0169534710001503">Invalid arguments against ABC: Reply to AR Templeton</a>” by Csillery et al.).</p>

<p>However, NCA is Templeton’s pet method and was popular for a long time (thousands of citations), and Bayesian logic can be difficult even for very smart professionals, especially if they have been raised thinking of frequentism as the only way to think about statistics, and Templeton has vast experience in human population genetics and popgen applied to medicine, so I don’t think his opinions on broader, simpler issues can be dismissed because of the fight over NCA/ABC.</p>

<p>I might as well add that my specialty is evolutionary biogeography at the macroevolutionary level of large number of species related by a phylogeny, not phylogeography, which operates primarily at the level of population genetics of one species or a few closely-related species/populations, so my opinions in this arena are still developing as I learn more about it.</p>

<p><strong>** Another note:</strong>re-reading Templeton (1998) after some time just now, I found rather more Nested Clade Analysis in there than I remembered.  I don’t think this affects the parts of his conclusions that I quoted, except perhaps for his skepticism of the 100,000 year “divergence time” for Africans vs. non-Africans and his endorsement of an alternative model (derived from his “trellis model”).</p>

<p>Templeton in 1998 is in part lobbying for the “trellis” model, where all humans are connected by gene flow and always have been, all the way back to <em>Homo erectus</em>.  He makes the modern isolation-by-distance observation a part of the evidence for this, but I don’t see any reason to endorse this.  We could have a fairly standard Recent Out-of-Africa model (adding minor interbreeding with Neandertals etc.) and still get the isolation-by-distance effect as long as humans have the behavior of breeding with nearby tribes and thus having high gene flow but mostly with adjacent groups.</p>

<p><strong>Other links of interest:</strong></p>

<p><a href="http://en.wikipedia.org/wiki/%22Human_genetic_diversity:_Lewontin%27s_fallacy%22_%28scientific_paper%29">http://en.wikipedia.org/wiki/%22Human_genetic_diversity:<em>Lewontin%27s_fallacy%22</em>%28scientific_paper%29</a></p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2012/02/continuous-geog.html';
this.page.identifier = '/archives/2012/02/continuous-geog';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


