<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Scurvy, Guinea Pigs, and You</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Scurvy, Guinea Pigs, and You" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2004/09/scurvy-guinea-p.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Reed A. Cartwright" />
  <meta property="article:published_time" content="2004-09-05T03:36:09-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2004/09/scurvy-guinea-p.html">Scurvy, Guinea Pigs, and You</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Reed A. Cartwright</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2004-09-05T03:36:09-07:00">September 5, 2004 03:36 MST</time></span>
	  </div>
   </header>
  <p>In my last post, <a href="http://pandasthumb.org/archives/2004/08/common-design-e.html">Common Design Errors</a>, I proposed a problem for biblical creation.  I received one response from a creationist, who cited Inai et al. (2003).  This paper compared the largest set of homologous exons between humans, guinea pigs, and rats.  You see, guinea pigs, like most primates and a few other taxa, lack L-guluno-gamma-lactone oxidase.  Two sections were quoted to me.</p>

<blockquote>
  <p>When the human and guinea pig sequences (647 nucleotides in total) of the regions of exons 4, 7, 9, 10, and 12 were compared, we found 129 and 96 substitutions in humans and guinea pigs, respectively, when compared with the rat sequences (Fig. 2).  The same substitutions from rats to both human and guinea pigs occurred at 47 nucleotide positions among the 129 positions where substitutions occurred in the human sequences.  A high percentage of the same substitutions in the total substitutions (36%) indicates that there were many hot spots for nucleotide substitution throughout the sequences examined.</p>

  <p>p. 316</p>
</blockquote>

<blockquote>
  <p>Assuming an equal chance of substitution throughout the sequences, the probability of the same substitutions in both humans and guinea pigs occurring at the observed number of positions and more was calculated to be 1.84 x 10-12.  This extremely small probability indicates the presence of many mutational hot spots in the sequences.</p>

  <p>p. 317</p>
</blockquote>

<p>This was to support the following response by the reader</p>

<blockquote>
  <p>Obviously if the same substitutions are occurring independently in the primate and guinea pig lineages, they could have arisen independently within the primates as well.</p>
</blockquote>

<p>which was followed up by</p>

<blockquote>
  <p>If single sustitutions can occur independently, how do we know that larger deletions couldn’t as well?</p>
</blockquote>

<p>However, the sections quoted from Inai et al. (2003) suffer from a major methodological error; they failed to consider that substitutions could have occured in the rat lineage after the splits from the other two.  The researchers actually clustered substitutions that are specific to the rat lineage with separate substitutions shared by guinea pigs and humans.  To illustrate this point, I am going to use sequences homologous to rat exon 10, for which we have the most data from other species.</p>

<p>The following figures list a section of rat exon 10 and the sections from ten species homologous to it.  The sequences were retreived from NCBI and aligned using ClustalX.  In the figure, periods represent nucleotides that are the same as  rat nucleotides.</p>

<p><code class="language-plaintext highlighter-rouge">                                                                                                                 1
                     1         2          3         4          5         6          7         8          9         0          1         2          3         4          5         6     
            12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345
Rat         GGAGAAGACCAAGGAGGCCC TACTGGAGCTAAAGGCCATG CTGGAGGCCCACCCCAAAGT GGTAGCCCACTACCCCGTAG AGGTGCGCTTCACCCGAGGC GATGACATTCTGCTGAGCCC CTGCTTCCAGAGGGACAGCT GCTACATGAACATCATTATG TACAG
Guinea Pig  A................... .G........G...AG.... .....A..T........G.. ..C............T..G. G...............G..G ..C.....C........... ..C................. ..............TGC..A .....
Human       AA.........C........ .G........G......G.. ..............TG.G.. ...GT.........TG..G. G...A.........T.GA.G -.......C..A........ ..........T........C .....C.........ACC.. .....</code></p>

<p>If I performed the same analysis as Inai et al. (2003), I would conclude that there are ten positions where humans and guinea pigs experienced separate substitutions of the same nucleotide, otherwise known as shared, derived traits.  These positions are 1, 22, 31, 58, 79, 81, 97, 100, 109, 157.  However, most of these are shown to be substitutions in the rat lineage when we look at larger samples of species.</p>

<p><code class="language-plaintext highlighter-rouge">                                                                                                                 1
                     1         2          3         4          5         6          7         8          9         0          1         2          3         4          5         6     
            12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345
Rat         GGAGAAGACCAAGGAGGCCC TACTGGAGCTAAAGGCCATG CTGGAGGCCCACCCCAAAGT GGTAGCCCACTACCCCGTAG AGGTGCGCTTCACCCGAGGC GATGACATTCTGCTGAGCCC CTGCTTCCAGAGGGACAGCT GCTACATGAACATCATTATG TACAG
Mouse       .................... .G.................. .................G.. ..................G. ...................T ........C........... G................... .................... .....
Guinea Pig  A................... .G........G...AG.... .....A..T........G.. ..C............T..G. G...............G..G ..C.....C........... ..C................. ..............TGC..A .....
Human       AA.........C........ .G........G......G.. ..............TG.G.. ...GT.........TG..G. G...A.........T.GA.G -.......C..A........ ..........T........C .....C.........ACC.. .....
Chimp       AA.........C........ .G........G......... ...............G.G.. ...GT.........TG..G. G.C.A.........T.GA.G -.......C..A........ ..........C........C .....C.........ACC.. .....
Oragutan    AA.........C........ .G........G......... ..............TG.G.. ...GT..........G..G. G..............AGA.G -.....G.C..A........ ..........CA.......C ....TC.........ACC.. .....
Macaque     AA.........CA.G..... .G......A.G......... ..............TG.G.. ...GT.......A..G..G. G..............A...G -.......CA.A........ ..........CA........ .....C..G......ACC.. .....
Cow         A...........A....... .G........G......... ........GAG......G.. A..G..............G. ....A........T..C..G ..C.....C........... ..........C.A....... ...........G....C... .....
Pig         A................... .C........G......... .................G.. ...G..............G. .............T..G.CG ..C.....C........... .................... ................C... .....
Chicken     T........A.....A..A. .G........G.....TGCC ......AA.A.......GA. ...G...........T..G. ..........TG.T....CG .....G..CTG......... .................... ................C... .....
Tiger Shark TA..C....T.GA.CA..T. .GGA.C....G....ATTG. .....CAA.A.T..T..T.. .CG...A..T.TT..T..C. ....T..G..TGTT..T.CA ..C..T........C..... ...T.A.AGACA........ .......T........C... .....</code></p>

<p>When we look at this larger data table, only one position of the ten, 81, stands out as a possible case of a shared derived trait, one position, 97, is inconclusive, and the other eight positions are more than likely shared ancestral sites.   With this additional phylogenetic information, I have shown that the “hot spots” Inai et al. (2003) found are not well supported.  Therefore, the explaination given to me by the creationist who responded does not work.</p>

<ul>
  <li>Inai Y et al. (2003), “The Whole Structure of the Human Non-Functional L-Guluno-gamma-Lactone Oxidase Gene - the Gene Responsible for Scurvy - and the Evolution of Repetitive Sequences Thereon,” J Nutr Sci Vitaminology
49:315-319.</li>
</ul>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2004/09/scurvy-guinea-p.html';
this.page.identifier = '/archives/2004/09/scurvy-guinea-p';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


