<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Meyer on Medved: the blind leading the blind</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="Meyer on Medved: the blind leading the blind" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2013/10/meyer-on-medved.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Nick Matzke" />
  <meta property="article:published_time" content="2013-10-23T12:59:24-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2013/10/meyer-on-medved.html">Meyer on Medved: the blind leading the blind</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Nick Matzke</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2013-10-23T12:59:24-07:00">October 23, 2013 12:59 MST</time></span>
	  </div>
   </header>
  <p>So, <a href="http://www.evolutionnews.org/2013/10/on_the_medved_s_3078241.html">Stephen Meyer is allegedly going to “respond” to critics of <em>Darwin’s Doubt</em></a> this afternoon on the Medved show (Medved is a DI fellow, this particular show is being broadcast from inside the Discovery Institute). They are even inviting critics to call in – it’s your very own chance to rebut Meyer’s dozens of overlapping errors and admissions with a single question or statement, and with the Discovery Institute controlling the microphone!  What a deal!</p>

<p>Over here in real science, we mostly try to address complex scientific topics through writing and analysis.  And the critics have done quite a bit of that, although you’d never know it from reading the Discovery Institute blog, where David “spin til it hurts” Klinghoffer has pushed the rhetoric-to-truth ratio to heights not seen since the <em>Kitzmiller</em> days.</p>

<p>Klinghoffer’s constant refrain is that the critics aren’t addressing Meyer’s arguments. Well, let’s make a little list, shall we?  It will be convenient as a checklist for anyone who listens to the Medved show today.</p>

<p><strong>Major criticisms of Darwin’s Doubt by informed critics:</strong></p>

<p><strong>1.</strong> Meyer’s book, which is supposed to be about the Cambrian Explosion, gets the Cambrian Explosion fossil record wrong.  The <a href="http://pandasthumb.org/archives/2013/06/meyers-hopeless-2.html">real fossil record is</a>:</p>

<blockquote>
  <ol>
    <li>
      <p>Ediacaran fauna in the Precambrian</p>
    </li>
    <li>
      <p>Before the beginning of the Cambrian, we see then bilaterian worm tracks that gradually increase in diversity</p>
    </li>
    <li>
      <p>In the latest few million years of the Precambrian, we observe <em>Cloudina</em> and other minute and very simple “small shelly” fossils</p>
    </li>
    <li>
      <p>For about 20 million years, throughout the earliest part of the Cambrian, we see the “small shelly” fossil faunas gradually increase in diversity</p>
    </li>
    <li>
      <p>Only after all of this do we reach the “main pulse” of the Cambrian Explosion. But even here, the representatives of the “phyla” mostly stem taxa, meaning that they lack some of the key characters found in the extant crown groups, and many of them exhibiting character suites “transitional” between the crown groups</p>
    </li>
    <li>
      <p>Throughout the later Cambrian and on into the Ordovician, we have gradual takeover of faunas by the earliest members of what end up as the living crown groups, and further evolution of new bodyplans in the Ordovician for some groups</p>
    </li>
  </ol>
</blockquote>

<p>As Graham Budd writes in <a href="http://icb.oxfordjournals.org/content/43/1/157.long">Budd 2003</a>:</p>

<blockquote>
  <p><strong>CONCLUSIONS</strong></p>

  <p>The combination of important refinements in the treatment of the systematics of Cambrian fossils, and in our understanding of Cambrian stratigraphy is leading to a more precise view of the Cambrian explosion. Phyla do not appear in a sudden jumble, implying an appearance in the fossil record induced by some external influence (e.g., a rise in atmospheric oxygen levels) that allowed a standing diversity already present to be manifested in the record. Rather, the impression rather is of a rapid, but nevertheless resolvable and orderly appearance, starting with the earliest skeletal forms such as <em>Cloudina</em> that are reasonably assignable to a diploblast grade (i.e., stem- or crown-group cnidarians or basal stem-group bilaterians). These are followed by taxa that lie in basal positions within bilaterian clades, and (in general) considerably later by representatives of the crown-groups of phyla. Revisions to the Cambrian time-scale allow a moderately long period of time, some tens of millions of years, between the first likely bilaterian trace fossils, and the general appearance of crown-group members of the phyla.</p>
</blockquote>

<p>However, however, because of poor research, or a tendency to ignore inconvenient facts, or both, all Meyer and his defenders give their readers is:</p>

<blockquote>
  <ol>
    <li>
      <p>Ediacaran fauna</p>
    </li>
    <li>
      <p>Poof! Almost all of the the phyla appear at once, fully formed, with no transitional forms!</p>
    </li>
  </ol>
</blockquote>

<p>As I’ve said before, this doesn’t respect the science or the data.</p>

<p>As I’m writing this, I see that Stephen Meyer just (Finally! 4 months after I made the criticism!) <a href="http://www.evolutionnews.org/2013/10/more_on_small_s078251.html">put up a response on the small shellies question</a>. And…wow, Meyer made the briefest mention of small shellies in endnote 39 of chapter 4, in an endnode on the definition of the base of the Cambrian, therefore he’s OK!</p>

<p>(never mind that small shellies technically predate the base of the Cambrian, so Meyer got even that part wrong)</p>

<p>He also confuses the question of whether the small shellies can be identified with specific groups which is debatable in many cases (but there have been several successes in doing this), with the question of whether or not they are ancestral at all.  I don’t think there is anyone now who would assert they are some completely independent group.  Gould half-suggested this in <em>Wonderful Life</em> in the tiny bit of text that he puts into the small shellies, but that was nearly 25 years ago, and the study and dating of small shellies has advanced dramatically since then. And having critters with chainmail armor just before you find fossils with full plate armor is surely not a coincidence.</p>

<p>Regarding dating the “Explosion”, surely it is more important to get correct the timing and sequence of events relevant to the origin of the bilaterians, than it is to quote authorities about when “the Explosion” was.  At the very least, skeletons and skeletonization were evolving during the small shelly period, millions of years before the “Explosion”.  How can this not be important?  How could Meyer have left this out for his readers, none of whom would have checked the endnote, nor realized that a huge piece of the puzzle that was being elided there!</p>

<p>.</p>

<p>.</p>

<p><strong>2.</strong> Meyer says that transitional fossils for the Cambrian groups don’t exist, but fossils with morphology transitional between the crown phyla do exist, oodles of them.  Meyer just doesn’t have the knowledge, either of fossils or methodology, to know a transitional fossil even if it is right there in front of him.  <a href="http://pandasthumb.org/archives/2013/06/meyers-hopeless-2.html">I made the basic argument previously</a>, but it can be done even more clearly with the amazing new study by Legg et al., which coded and analyzed hundreds of characters of hundreds of Cambrian arthropods and arthropod relatives.</p>

<blockquote>
  <p><img src="/uploads/2013/Legg_etal_2013_NatComm_panarthropod_phylogeny.png" alt="Legg_etal_2013_NatComm_panarthropod_phylogeny.png" width="578" height="872" style="text-align: center; display: block; margin: 0 auto 20px;" class="mt-image-center" /></p>
</blockquote>

<p>…and, if we zoom in on the base of the tree:</p>

<blockquote>
  <p><img src="/uploads/2013/Legg_etal_2013_NatComm_Figure4ab_Lobopodia_etc_bigger.png" alt="Legg_etal_2013_NatComm_Figure4ab_Lobopodia_etc_bigger.png" style="text-align: center; display: block; margin: 0 auto 20px;" class="mt-image-center" /></p>

  <p>Figure 4: Phylogeny of panarthropoda (just parts a &amp; b).</p>
</blockquote>

<p>Bam!</p>

<p>See “EUARTHROPODA” down at the lower right?  That’s the node representing the last common ancestor of all living arthropods, i.e. crown group arthropods.  Bumblebees, king crabs, hummingbird moths, millipedes, shrimp, horseshoe crabs – they’re all just subgroups of that one little subgroup of these fossils.  Everything branching  below EUARTHROPODA on the tree lacks one of more characters that unite all living arthropods.  See ONYCHOPHORA, in the middle of (a)?  That’s the closest living phylum to arthropods.  Between two modern crown group phyla (arthropods and onychophorans), there are <em>dozens</em> of species with intermediate character suites.  And that’s just the species currently known to science well enough to describe and put in a character matrix.  This is why I was <a href="http://pandasthumb.org/archives/2013/06/meyers-hopeless-2.html">ranting and raving about lobopods</a> in my previous posts.</p>

<p>No transitional fossils?  Take a hike, Stephen Meyer and David Klinghoffer!</p>

<p>.</p>

<p>.</p>

<p><strong>3.</strong> Meyer claims that phylogenetic methods are worthless, but doesn’t know that phylogenetic hypotheses are statistical statements and are statistically testable through standard methods – methods which themselves are testable and well-tested.  Virtually any time such methods are applied to real data, including in Cambrian divergences, there is a strong statistical tree signal, and strong congruence between different datasets, and this remains true whether or not different datasets give <em>exactly</em> the same phylogenetic results on any particular phylogenetic question.  The most divergent results typically come from the smallest datasets – small numbers of taxa or small numbers of characters, or both (with the number of taxa being more important than the number of characters), which is exactly what any scientist would expect from any statistical inference procedure.</p>

<p>For actual serious information on this topic, which Meyer mentions only to then ignore, see <a href="http://www.talkorigins.org/faqs/comdesc/">Doug Theobald’s FAQ</a>, <a href="http://www.talkorigins.org/faqs/comdesc/section1.html#independent_convergence">particularly the section on phylogenetic tree statistics</a>.</p>

<p>.</p>

<p>.</p>

<p><strong>4.</strong> Meyer claims that the evolution of new genetic information is virtually impossible, a claim he is able to sustain mostly because he doesn’t understand the phylogenetic methods (see above) that are necessary for inferring the history of the origin of genes. Apparently he wants us scientists to think that gene sequences share similarities that fall into nice tree patterns – including, often, direct genomic remnants of duplication, transposition, and other mutational processes – just because the intelligent designer put those signals there just to make it look like evolution happened. Even some of Meyer’s colleagues at the Discovery Institute agree that Meyer is wrong here – <a href="http://www.evolutionnews.org/2013/06/a_new_gene_fits073151.html">Michael Behe</a> and <a href="http://www.evolutionnews.org/2013/08/hopeless_matzke075631.html">David Berlinski</a> accept that natural evolutionary processes can produce new genes and thus new genetic information – they are apparently just too craven to admit the contradiction forthrightly.</p>

<p>Since Meyer’s main argument is about how intelligent design is the unique and only known cause of new information, he really is sunk even if one duplicate gene with a few mutations and little bit of new information is accepted as having occurred naturally.  If it can happen one time in one species, what can happen over hundreds of millions of years across millions of species?</p>

<p>.</p>

<p>.</p>

<p><strong>5.</strong> Meyer’s claim that “massive amounts” of new genetic information was required for the Cambrian Explosion is belied by the fact that gene number, and most of the key developmental patterning genes are shared broadly across the phyla, and even outside of bilaterians, rather than looking like they originated in the midst of the divergence of the phyla.**</p>

<p>Meyer obscures this with wild assertions about genomics and tons of “information” in the noncoding DNA.  This leads us to:</p>

<p><strong>6.</strong> Meyer claims that the “junk DNA” hypothesis has been refuted, and that therefore the 90+ percent of large animal genomes that doesn’t code for genes or gene regulation is actually a massive additional amount of new information, but Meyer doesn’t rebut or even acknowledge the massive, basic, evidential problems with this idea. (And, for that matter, neither does the ENCODE project or the various journalists who were snookered by this triumph of wishful thinking over hard evidnece.)  The problems are:</p>

<p><strong>6a.</strong> <a href="http://sandwalk.blogspot.com/2013/10/non-darwinian-evolution-in-1969-case.html">Simple calculations show</a> that if most of the DNA in large-genomed organisms like humans was essential, given known mutation rates, we would die from fatal mutations each generation. This was Ohno’s original argument for junk DNA, and it has not been rebutted by ENCODE, the creationists, or anyone else.</p>

<p><strong>6b.</strong> Genome sizes in complex animals and plants <a href="http://en.wikipedia.org/wiki/Genome_size#Variation_in_genome_size_and_gene_content">vary by orders of magnitude</a> within many specific groups (tetrapods, <a href="http://www.genomicron.evolverzone.com/2007/04/onion-test/">onions</a>, ferns, salamanders, arthropods, whatever), but despite this, within each group they all have about the same number of genes, and approximately the same organs and developmental complexity.  The obvious and straightforward inference is that most of the DNA which is sometimes present, and sometimes absent, isn’t necessary, and probably isn’t doing much even when it is present.</p>

<p><strong>6c.</strong> When you <a href="http://www.polypompholyx.com/2012/09/a-brief-history-of-rubbish/">actually look</a> at the sequences in the variable fraction of the genome, most of it <a href="http://www.polypompholyx.com/wp-content/uploads/2012/09/genome_contents.png">looks like</a> the product of mutational processes with no selective filtering – transposon remnants, fossil viruses, duplications and other mutational errors, etc.  Furthermore, unlike genes and important regulatory regions, which are well-conserved between closely related species, the apparent junk DNA looks like it has no constraint.</p>

<p>Arguments 6a, 6b and 6c have not been rebutted by ENCODE, the creationists, or anyone else.  They’ve hardly even been acknowledged, let alone addressed in a serious way, let alone effectively rebutted.</p>

<p>In the face of all of this, it staggers belief for Meyer and his defenders to think that anyone who knows the above facts will be impressed with Meyer’s book, or Klinghoffer’s ceaseless din of spin. I suspect that Klinghoffer himself sustains his energy on little more than the faith that his chosen “experts” know what they are talking about, and that they are brave rebels against the dogmatic establishment.</p>

<p>Earth to David: look into these topics for yourself.  Pick any one of them!  Read the papers, look at the raw data for yourself, and take the trouble to learn enough about the methods and data to actually do a simple analysis (a graph or phylogeny) yourself.  Then start asking Meyer or other ID authorities what their explanation is, and why the evolutionary analysis is wrong.  Don’t be afraid to follow the evidence where it leads – just make sure you are actually aware of the actual evidence, instead of relying on the mishmash of quote-mines and creationist misunderstandings that are tossed together in Meyer’s book.</p>

<p>.</p>

<p>.</p>

<p><strong>Notes and References</strong></p>

<p>* Graham Budd is the highest form of expert here, in that he a paleontologist whose speciality is specifically Cambrian organisms.</p>

<p>** Ditto for protein domains, etc., which are also widely distributed, and are not specific to the bilaterian phyla, so did not have to originate in the Cambrian.  If a creationist/ID fan wants to talk about more remote origins, or the origin of life, that’s fine, but they should realize that by switching topics in an attempt to avoid admitting error, they have effectively admitted defeat on the original topic, which was the Cambrian Explosion.</p>

<p>Legg et al. (2013). “Arthropod fossil data increase congruence of morphological and molecular phylogenies.” <em>Nature Communications</em> 4, doi:10.1038/ncomms3485</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2013/10/meyer-on-medved.html';
this.page.identifier = '/archives/2013/10/meyer-on-medved';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


