<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>On the evolution of Irreducible Complexity</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="On the evolution of Irreducible Complexity" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2007/05/on-the-evolutio-1.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="Arthur Hunt" />
  <meta property="article:published_time" content="2007-05-01T21:55:48-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2007/05/on-the-evolutio-1.html">On the evolution of Irreducible Complexity</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Arthur Hunt</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2007-05-01T21:55:48-07:00">May 1, 2007 21:55 MST</time></span>
	  </div>
   </header>
  <p>There has been a spate of interest in the blogosphere recently in the matter of protein evolution, and in particular the proposition that new protein function can evolve.  Nick Matzke summarized a review (reference 1) on the subject <a href="http://scienceblogs.com/pharyngula/2007/02/dr_michael_egnor_challenges_ev.php#comment-353197">here</a>. Briefly, the various mechanisms discussed in the review include exon shuffling, gene duplication, retroposition, recruitment of mobile element sequences, lateral gene transfer, gene fusion, and de novo origination.  Of all of these, the mechanism that received the least attention was the last – the de novo appearance of new protein-coding genes basically “from scratch”.  A few examples are mentioned (such as antifreeze proteins, or AFGPs), and long-time followers of ev/cre discussions will recognize the players.  However, what I would argue is the most impressive of such examples is not mentioned by Long et al. (1).  Below the fold, I will describe an example of de novo appearance of a new protein-coding gene that should open one’s eyes as to the reach of evolutionary processes.  To get readers to actually read below the fold, I’ll summarize – what we will learn of is a protein that is not merely a “simple” binding protein, or one with some novel physicochemical properties (like the AFGPs), but rather a gated ion channel.  Specifically, a multimeric complex that: 1. permits passage of ions through membranes;  2.  and binds a “trigger” that causes the gate to open (from what is otherwise a “closed” state).  Recalling that Behe, in Darwin’s Black Box, explicitly calls gated ion channels IC systems, what the following amounts to is an example of the de novo appearance of a multifunctional, IC system.</p>

<p><strong>An IC System From Scratch</strong></p>

<p>The subject of this essay is a fascinating protein that I will call T-urf13 (see the first footnote).  This protein, and its gene, was discovered in the mitochondrial genome of certain maize varieties that had been bred to be male-sterile (see footnote 2).  There are many different cytoplasmic male sterility (or cms) “traits” in maize, and T-urf13 is found only in so-called cmsT maize (T stands for Texas).  This protein was discovered, and studied, owing to an unfortunate circumstance in the 1969 and 1970 (see reference 2 for a review).  Briefly, cmsT maize fields were devastated by the appearance of a fungal pathogen (Cochliobolus heterostrophus race T) that seemingly had a unique “taste” for cmsT plants.  (The disease is known as Southern corn leaf blight.)  After many years of genetic, molecular, and ultimately biochemical studies, it was found that disease was due to a unique susceptibility of cmsT maize to a toxin produced by the fungus (see reference 3 for a recent reviewon the biosynthesis of this toxin).  It was also found that disease susceptibility, as well as male sterility, were due to rearrangements of the maize mitochondrial genome, resulting in a novel cmsT-specific locus (see the following paragraph; 4).  Finally, it was found that this novel locus encoded a small protein, T-urf13, whose production correlated exactly with both male sterility and disease susceptibility (5).</p>

<p>(I apologize if the pathogen’s name is not familiar or current – I long ago gave up trying to keep pace with the taxonomy of fungi.)</p>

<h3 id="the-origins-of-the-cmst-locus-are-fascinating--several-years-ago-this-locus-was-cloned-and-sequenced-and-the-sequence-compared-with-normal-maize-mitochondrial-dna--this-analysis-revealed-that-the-t-urf13-locus-was-the-product-of-numerous-recombination-events-as-many-as-seven-different-ones-along-with-reference-4-reference-6-has-a-nice-overview-of-this--the-consequence-of-these-recombination-events-was-the-cobbling-together-of-a-number-of-disparate-mitochondrial-dna-mtdna-segments-i-have-tried-to-illustrate-their-origins-in-figure-1-to-yield-a-novel-dna-segment-in-the-cmst-mitochondrial-genome">The origins of the cmsT locus are fascinating.  Several years ago, this locus was cloned and sequenced, and the sequence compared with normal maize mitochondrial DNA.  This analysis revealed that the T-urf13 locus was the product of numerous recombination events (as many as seven different ones; along with reference 4, reference 6 has a nice overview of this).  The consequence of these recombination events was the cobbling together of a number of disparate mitochondrial DNA (mtDNA) segments (I have tried to illustrate their origins in Figure 1) to yield a novel DNA segment in the cmsT mitochondrial genome.</h3>

<p><img src="/uploads/2007/Turf13F1.png" alt="Turf13F1.png" width="720" height="540" /></p>

<h3 id="figure-1--i-have-superimposed-two-sections-colored-circles-that-have-been-brought-together-to-form-the-unique-region-2h3-in-the-mitochondrial-genome-of-cmst-maize--for-this-i-have-used-the-circular-map-of-the-maize-nb-mitochondrial-genome-downloaded-from-clifton-s-w-et-al-plant-physiol-20041363486-3503-to-show-the-relative-positions-of-the-two-major-parts-of-the-2h3-region-as-well-as-the-location-in-the-genome-of-the-2h3-region-itself--note-that-these-circles-as-well-as-the-2h3-region-are-not-drawn-to-scale--also-note-that-the-nb-genome-map-is-being-used-for-convenience-the-paper-has-a-nice-figure-that-allows-me-to-make-the-basic-point--the-actual-genome-from-which-cmst-was-derived-may-differ-in-some-ways-from-the-nb-genome-although-not-in-the-general-features-shown-here--as-shown-in-figures-2-and-3-the-t-urf13-gene-lies-within-the-green-shaded-portion-of-2h3">Figure 1.  I have superimposed two sections (colored circles) that have been brought together to form the unique region (2H3) in the mitochondrial genome of cmsT maize.  For this, I have used the circular map of the maize NB mitochondrial genome (downloaded from Clifton, S. W., et al. Plant Physiol. 2004;136:3486-3503) to show the relative positions of the two major parts of the 2H3 region, as well as the location in the genome of the 2H3 region itself.  Note that these circles, as well as the 2H3 region, are not drawn to scale.  Also note that the NB genome map is being used for convenience (the paper has a nice figure that allows me to make the basic point).  The actual genome from which cmsT was derived may differ in some ways from the NB genome, although not in the general features shown here.  As shown in Figures 2 and 3, the T-urf13 gene lies within the green shaded portion of 2H3.</h3>

<p>One consequence of these rearrangements was the protein-coding region for T-urf13.  What is remarkable is that none of the T-urf13 protein coding region came from other protein-coding genes.  This is illustrated in Figures 2 and 3.  Briefly, this novel protein-coding” gene consists of a segment of the 3’-flanking region of the mitochondrial 26S ribosomal RNA gene, a much smaller (and not contiguous) part of the RNA-coding part of the same gene, and a number of bp that seemingly appeared from out of the blue (Figure 2).</p>

<p><img src="/uploads/2007/Turf13F2a.png" alt="Turf13F2a.png" width="720" height="540" /></p>

<p>Closer perusal of the sequences (Figure 3) reveals that the origination of this gene involved, not just the piecing together of two different parts of the mitochondrial genome, but several other mutational events (point mutations, small insertions and/or deletions).</p>

<p><img src="/uploads/2007/Turf13F3a.png" alt="Turf13F3a.png" width="720" height="540" /></p>

<p>The bottom line is that T-urf13 is a new protein, encoded by a gene that has no protein-coding antecedents; it is, bluntly, a new protein that arose “from scratch”, through a series of duplications, recombinations, and other mutations that occurred spontaneously in the course of the breeding process that gave rise to the cmsT line.</p>

<p>These points are already problematic for the assertion by ID proponents that new protein-coding information cannot arise by natural processes.  But T-urf13 is more than a nondescript polypeptide that happens to affect male fertility in corn.  It turns out that T-urf13 is a membrane protein, and in membranes it forms oligomeric structures (I am not sure if the stoichiometries have been firmly established, but that it is oligomeric is not in question).  This is the first biochemical trait I would ask readers to file away – this protein is capable of protein-protein interactions, between like subunits.  This means that the T-urf13 polypeptide must possess interfaces that mediate protein-protein interactions.  (Readers may recall Behe and Snokes, who argued that such interfaces are very unlikely to occur by chance.)</p>

<p>T-urf13 also binds to the toxin produced by the fungal pathogen.  But it does not just bind the toxin “passively” – upon binding, a non-selective ion channel is opened, leading to dissipation of transmembrane ion gradients, and all of the resulting events that accompany collapse of proton-motive force.  (In mitochondria, this will lead to uncoupling and crippling of mitochondrial function; this is probably why cmsT plants are so devastated by the disease.)  This is the second biochemical trait that readers should keep fresh in their minds – T-urf13 is a gated ion channel.  (This an the other interesting biochemical properties of Turf13 are reviewed in reference 7.)</p>

<p>Those who have read Darwin’s Black Box might recall Behe’s description of a gated ion channel.  On pp. 108-110, Behe describes the signal recognition particle (SRP)-mediated transport of proteins (footnote 3) as a gated transport process.  In so doing, he asserts (among other things) that “(b)ecause gated transport requires a minimum of three separate components to function, it is irreducibly complex”.  The three components he describes for SRP-mediated protein translocation are the signal peptide, SRP, and the transport channel.  The T-urf13 gated ion channel also consists of three components – the fungal toxin (footnote 4) is analogous to the signal peptide, the toxin binding site is analogous to SRP, and the ion channel is analogous to the protein channel.  In case this comparison has hidden the bottom line, it is this – T-urf13 is irreducibly complex in exactly the same way that Behe asserts for SRP-mediated protein transport.</p>

<p>The take-home message of all this is: portions of the maize mitochondrial genome that do not normally encode any protein were shuffled, extensively, so as to cobble together an expressed gene that encodes, not just any old polypeptide, but a multimeric gated ion channel.  In other words, an irreducibly complex structure arose <strong>in one fell swoop</strong>, using DNA sequences that do not encode proteins.  Basically, this is a case of IC from scratch.</p>

<p>What does all this mean?  A linchpin of ID thought is the notion that functional proteins, especially multifunctional ones, cannot arise de novo via natural processes.  This proposition is one of <a href="/archives/2006/04/an-argument-is.html">Paul Nelson’s antievolutionary arguments</a> that center on ORFans (briefly, the appearance of new function is one of Nelson’s alleged discontinuities), it is a part of the reason so many ID supporters retreat to the stage of the OOL when faced with other facts (presumably, they think that the origins of the first functioning proteins are beyond the grasp of natural mechanisms), it is ultimately where Behe’s ideas about irreducible complexity end up.  The example discussed in this essay shows that even complex multifunctional proteins are well within the “reach” of normal, natural molecular processes.</p>

<p><strong>Footnotes.</strong></p>

<ol>
  <li>
    <p>Veterans of the ARN boards will recognize this subject.  One of the two threads that discussed T-urf13 has apparently been lost, but the other <a href="http://www.arn.org/ubbthreads/showflat.php?Cat=0&amp;Number=187227&amp;an=0&amp;page=9#Post187227">can still be accessed</a>.</p>
  </li>
  <li>
    <p><a href="http://en.wikipedia.org/wiki/Cytoplasmic_male_sterility">Male sterility</a> is a trait used by plant breeders to promote outcrossing; in crops like maize, it is used in the production of hybrids, which increases yield through the phenomenon known as hybrid vigor.  The trait is called “cytoplasmic male sterility” because it is inherited in a non-Mendelian fashion.  Briefly, cms is maternally inherited, so that 100% of the progeny of a cross in which the mother is male sterile will also be cms.  In animals, maternal inheritance is a hallmark of a mitochondrial gene; the same holds in plants, but maternal inheritance also applies for chloroplast genes.</p>
  </li>
  <li>
    <p><a href="http://en.wikipedia.org/wiki/Signal_recognition_particle">SRP-mediated transport</a> is the process by which proteins destined for vesicular transport and export out of the cell are synthesized.  Briefly, proteins destined for transport possess N-terminal signal peptides that are recognized by the ribonucleoprotein Signal Recognition Particle; association of the SRP-nascent polypeptide peptide with the translating ribosome causes a pausing, which is released upon “docking” with the transport apparatus on the surface of the endoplasmic reticulum.</p>
  </li>
  <li>
    <p>The toxin made by  C. heterostrophus race T is a <a href="http://en.wikipedia.org/wiki/Polyketide">polyketide</a>. It is beyond the scope of this essay to detail this class of compound, or the fascinating enzymes that synthesize them.  Suffice to say that these enzymes add another layer of “complexity” to this subject, in that a rather complex set of activities had to evolve, along with the maize mitochondrial genome, to “assemble” the IC T-urf13 system.</p>
  </li>
</ol>

<p><strong>References</strong></p>

<ol>
  <li>
    <p>Long M, Betran B, Thornton K, Wang W. 2003.  The origin of new genes: Glimpses from the young and old.  Nature Rev Genet. 4: 865-875. (a review, cited by Nick Matzke, on the origins of genes)</p>
  </li>
  <li>
    <p>Levings CS 3rd. 1990. The Texas cytoplasm of maize:  Cytoplasmic male sterility and disease susceptibility. Science 250, 942-947. (a nice summary of the phenomenon)</p>
  </li>
  <li>
    <p>Turgeon BG, Baker SE. 2007. Genetic and genomic dissection of the Cochliobolus heterostrophus Tox1 locus controlling biosynthesis of the polyketide virulence factor T-toxin Adv Genet 57, 219-261. (a recent review of the synthesis of the T toxin)</p>
  </li>
  <li>
    <p>Dewey RE, Levings CS 3rd, Timothy DH. 1986. Novel recombinations in the maize mitochondrial genome produce a unique transcriptional unit in the Texas male-sterile cytoplasm.  Cell 44(3):439-49. (characterization of the T-urf13 locus in cmsT maize)</p>
  </li>
  <li>
    <p>Dewey RE, Timothy DH, Levings CS 3rd. 1987. A mitochondrial protein associated with cytoplasmic male sterility in the T cytoplasm of maize. Proc Natl Acad Sci U S A 84(15):5374-5378.  (links the T-urf13 protein with cms)</p>
  </li>
  <li>
    <p>Hanson MR. 1991. Plant mitochondrial mutations and male sterility. Annu Rev Genet. 25:461-86.  (A review that discusses cms in a broad context)</p>
  </li>
  <li>
    <p>Rhoads DM, Levings CS 3rd, Siedow JN. 1995. URF13, a ligand-gated, pore-forming receptor for T-toxin in the inner membrane of cms-T mitochondria. J Bioenerg Biomembr. 27(4):437-45.  (a review of the biochemical properties of T=urf13)</p>
  </li>
</ol>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2007/05/on-the-evolutio-1.html';
this.page.identifier = '/archives/2007/05/on-the-evolutio-1';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


