<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>ID, scientifically vacuous and collapsing gaps</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="ID, scientifically vacuous and collapsing gaps" />
  <meta property="og:url" content="https://pandasthumb.org/archives/2005/05/id-scientifical.html" />
  <meta property="og:image" content="https://pandasthumb.org/media/roar.jpg" />
  <meta property="article:author" content="PvM" />
  <meta property="article:published_time" content="2005-05-31T21:06:16-07:00" />


  <link rel="stylesheet" type='text/css' href="/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="/search.html" title="Searching Panda's Thumb"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/archives/2005/05/id-scientifical.html">ID, scientifically vacuous and collapsing gaps</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By PvM</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2005-05-31T21:06:16-07:00">May 31, 2005 21:06 MST</time></span>
	  </div>
   </header>
  <p>While researching the nematode work by Azevedo, I ran across the following website called ‘Design vs. Descent: A war of predictions”. The original article can be found at the <a href="http://www.ideacenter.org/contentmgr/showdetails.php/id/846">Idea Center</a></p>

<p>While the article has many problems, one in particular caught my eye</p>

<blockquote>
  <p>Finally, a study which compared many proteins in humans, nematodes, arthropods, and yeast found that 2 starkly different trees were produced, depending on which genes were used.25 This pattern of different genes yielding very different phylogenetic trees is very common in the scientific literature, and shows that molecular data fail to give a consistent picture of the alleged common descent ancestry of organisms.</p>

  <ol>
    <li>Mushegian A, Garey J, Martin J, Liu L. Large-Scale Taxonomic Profiling of Eukaryotic Model Organisms: A Comparison of Orthologous Proteins Encoded by the Human, Fly, Nematode, and Yeast Genomes. Genome Research 1998;8:590-8.</li>
  </ol>
</blockquote>

<p>To my ‘surprise’, the <a href="http://www.genome.org/cgi/content/full/8/6/590">actual article</a> reads as follows</p>

<blockquote>
  <p>In this study we evaluated the nematode C. elegans and the fruit fly D. melanogaster as model systems for studying human proteins using protein sequence comparison techniques. By applying strict and reproducible criteria for identifying orthologous proteins, we could extract numerous protein-coding genes for phylogenetic analysis. Our simultaneous analysis of multiple orthologous proteins shows that different proteins can generate different apparent phylogenetic tree topologies, strongly suggesting that historical phylogenies should not be inferred based on a single protein-coding gene. Unequal evolutionary rates are an important factor in calculating phylogenetic trees, and indeed it appears that the majority of C. elegans genes are evolving more rapidly than their Drosophila counterparts. The approaches of ortholog extraction used in this work can be used to better define data sets for phylogenetic analysis among a broader range of representative animal phyla. The available portion of the fly proteome appears to be comparatively enriched in conserved protein domains because of abundant representation of phenotypically defined genes, while missing numerous protein families. The ortholog-to-paralog ratio with regard to human proteins is very similar in the two model animals, indicating that the domain architecture in fly and nematode proteins approximates that of their human homologs to the same extent.</p>
</blockquote>

<p>In other words, the authors warn that due to differences in evolutionary rates, one has to be careful to use a single gene dataset to infer phylogenetic relationships. In other words, the authors argued WHY these findings were at odds with the vaste amount of fossil and genetic data. While such outliers indeed exist, the overall picture from independent phylogenies shows a <a href="An adaptive radiation model for the origin of new gene functions">coherent picture</a></p>

<blockquote>
  <p>“Biologists seem to seek the ‘The One Tree’ and appear not to be satisfied by a range of options. However, there is no logical difficulty in having a range of trees. There are 34,459,425 possible [unrooted] trees for 11 taxa (Penny et al. 1982), and to reduce this to the order of 10-50 trees is analogous to an accuracy of measurement of approximately one part in 10^6.” (Penny and Hendy 1986, p. 414)</p>
</blockquote>

<p>This is much better than the accuracy of some physics constants…</p>

<p>To see why common descent is strongly supported by scientific evidence, one need not look further than the excellent <a href="http://www.talkorigins.org/faqs/comdesc/section1.html">29+ Evidences for Macroevolution</a> by Douglas Theobald.</p>

<p>Okay, let’s add another example from the same page, showing the dangers of gap arguments, when knowledge expands (does this count as a falsification of ID?)</p>

<blockquote>
  <p>Schwabe and Warr found that proteins such as relaxin, insulins, adrenocorticotropic hormone, somatostatin, histocompatability antigens, neural glycoproteins and microglobulin are distributed in both animal and non-animal groups in ways which differ markedly from predictions of common descent.13</p>
</blockquote>

<ol>
  <li>Schwabe, C., Warr, G. W., “A polyphyletic View of Evolution: The Genetic Potential Hypothesis.” Perspectives in Biology and Medicine, 27, 3, Spring 1984 pg. 465-484. See also “On the validity of molecular evolution” by Christian Schwabe (TIBS 11 - July 1986 pg. 280-283).</li>
</ol>

<p><a href="http://www.talkorigins.org/faqs/comdesc/camp.html#schwabe_warr">Douglas Theobald</a> on Camp’s usage of Schwabe et al. See also <a href="http://www.skepticfiles.org/evolut/moleculr.htm">Laurence A. Moran</a> on Schwabe and relaxins. And also <a href="http://www.biomedcentral.com/1471-2148/5/14">Evolution of the relaxin-like peptide family</a> BMC Evolutionary Biology 2005, 5:14</p>

<blockquote>
  <p>We present here a phylogeny for the relaxin-like peptide family. Relaxin has long been used as an example of a gene that conflicts with the Darwinian theory of evolution [24,32-34]. However, we have shown that these can issues can be resolved when studied in the context of the rest of the relaxin-like peptide family, in particular the new, but likely ancestral relaxin, relaxin-3.</p>
</blockquote>

<p>Btw it seems that it was <a href="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=eurekah.figgrp.4537">Schwabe who might have been responsible for Denton’s erroneous interpretation</a> of the <a href="http://home.wxs.nl/~gkorthof/korthof56.htm">Cytochrome-C data</a>.</p>

<p>I am not sure why the IDEA center is rejecting common descent, something accepted by most ID proponents such as Dembski or Behe. While YECers may strongly object to common descent, relying on this kind of poor representation of scientific presents a significant risk and liability to Intelligent Design.  Not only is Intelligent Design scientifically vacuous in that it does not present any explanation as to how relaxins or other examples can be explained, but the examples show that the gaps for ID’s God to be hiding in are either collapsing or were never really there.</p>

<p>In addition to ID being considered by many to be scientifically vacuous, I see ID as being a threat to education. Is this really what we want to teach our children? That science may not have all the answers? Fine… That ID is based on gaps in our knowledge? Gaps which either collapse with additional knowledge or gaps which have been ‘created’ by ID based on a flawed understanding of the issues involved… Perhaps it is time to teach the controversy and expose ID for what it is and what it isn’t.</p>

</article>

      </div>

   
<div id="disqus_thread" class="unit whole"></div>
<script>
var disqus_config = function () {
this.page.url = 'https://pandasthumb.org/archives/2005/05/id-scientifical.html';
this.page.identifier = '/archives/2005/05/id-scientifical';
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = '//the-pandas-thumb.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>



    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post 
     and then the comments to load (may take many seconds).  The comment should have a yellow vertical bar next to the commenter avatar.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=99&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


